<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Episkin Academy Report | Insilico Medicine</title>
    <link href="/static/bower_components/bootstrap-stylus/dist/bootstrap.min.css" rel="stylesheet" type="text/css">
    <script src="//code.jquery.com/jquery-2.1.4.min.js" type="text/javascript"></script>
    <script src="/static/js/oncofinder2.js" type="text/javascript"></script>
    <script src="/static/bower_components/bootstrap-stylus/dist/bootstrap.min.js" type="text/javascript"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore.js" type="text/javascript"></script>
    <script src="/static/js/d3.v3.min.js" type="text/javascript"></script>
    <script src="https://cdn.datatables.net/1.10.10/js/jquery.dataTables.min.js" type="text/javascript"></script>
    <script src="http://code.highcharts.com/highcharts.js" type="text/javascript"></script>
    <script src="https://code.highcharts.com/highcharts-more.js"  type="text/javascript"></script>
    <script src="https://code.highcharts.com/highcharts-3d.js"  type="text/javascript"></script>
    <script src="http://code.highcharts.com/modules/exporting.js"  type="text/javascript"></script>
    <link href="/static/css/report_style.css" rel="stylesheet" type="text/css">
    <link href="https://cdn.datatables.net/1.10.10/css/dataTables.bootstrap.min.css" rel="stylesheet" type="text/css">
    <link href="https://cdn.datatables.net/buttons/1.0.3/css/buttons.dataTables.min.css" rel="stylesheet">
    
    <link rel="shortcut icon" href="/static/images/favicon.ico" type="image/x-icon">
    <link rel="icon" href="/static/images/favicon.ico" type="image/x-icon">
    
  </head>
  <body style='font-size: 14px;'>
    <div class="content h100">
      <div class="main">
        <div class="inner">
         <div>
         <span class='pull-left' style='font-size:11pt; margin:15px;'>
         <img src='/static/images/insilico.png' style="width:40px;"/>
         Insilico Medicine - Episkin Academy Report</span>
         <span class='pull-right' style='font-size:11pt; margin:15px;'>Access Level: Z-II, <br/>Confidential </span>
         </div><br style="clear:both;"/>
          <div class='text-center'>
            <h1>Using signalling pathway activation analysis to compare engineered and natural human skin</h1>
          </div>
          
          <div style="margin: 50px 0 0 50px;">
            <h4 style='font-weight:bold; font-size:12pt;'>Organization: </h4>
            <span style="font-size:11pt">Episkin Academy</span>
            <br/><br/>
            <!-- <h4 style='font-weight:bold; font-size:12pt;'>Data Quality Assessment: </h4>-->
            
            <ul class="list-unstyled" style='font-size: 11pt; margin: 50px 0 0 0;' >
            <li class="title">Table of contents: </li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#input_data">Input data</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#methods">Methods</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#gene_analysis">Gene level analysis</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#path_analysis">Pathway analysis</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#public_domain">Public domain screening</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#references">References</a></li>
          </ul>
          
          <h2 id="input_data">Input data</h2>
          <hr/>
          <h3 id="input_data">Abbreviations and gene names</h3>
          <ul>
          <li>NHE - normal human epidermis</li>
          <li>NHEK - normal human epidermal keratinocytes</li>
          <li>RhE - reconstructed human epidermis</li>
          <li>PAS - pathway activation score</li>
          <li>PCA - principal component analysis</li>
          <li>ECM - extracellular matrix</li>
          <li>MMP - matrix metalloproteinases</li>
          <li>HYAL - hyaluronoglucosaminidase </li>
          <li>HAS - hyaluronan synthases</li>
          <li>COL - collagen</li>
          <li>FN - fibronectin</li>
          <li>KRT - keratin</li>
          <li>ELN - elastin</li>
          <li>GF - growth factor</li>
          <li>FGFR - fibroblast growth factor receptor</li>
          <li>EGF - epidermal growth factor</li> 
          <li>PLK1 - Polo-Like Kinase 1</li>
          <li>GSK3B - Glycogen synthase kinase 3 beta</li>
          <li>ILK - interleukin</li>
          <li>PTEN - phosphatase and tensin homolog</li>
          <li>EGFR - epidermal growth factor receptor</li>
          <li>IGF- Insulin-like growth factor</li>
          <li>NHDF - normal human dermal fibroblasts</li>
          <li>NHS - native human skin</li>
          <li>RHS - reconstructed human skin</li>
          <li>NHD - normal human dermis</li>
          <li>FGF - fibroblast growth factor</li>
          <li>GH - growth hormone</li>
          </ul>
          
          <hr/>
          <h2 id="methods" >Methods</h2>
          <hr/>
          
          Raw gene expression data, provided by Episkin, was processed with frozen RMA method [1]. Five 
          functional groups were considered in the differential gene expression analysis: NHE, NHEK, RhE 
          
          (Type 1), RhE (Type 2) and RhE (Type 3). In all following parts of analysis NHE group was used 
          as a reference and other groups were compared to it. For the gene-level analysis, we performed 
          Studentâ€™s T-test followed by Benjamini-Hochberg FDR adjustment [2]. For the pathway level analysis 
          each preprocessed gene expression group was independently compared to NHE reference using an 
          algorithm called OncoFinder [3]. Using preprocessed gene expression data as an input allows 
          for cross-platform dataset comparison with low error rate and provides the ability to obtain 
          functional features of intracellular regulation using mathematical estimations. Positive and 
          negative PAS values indicate pathway up- and down-regulation, respectively. In total, we 
          performed the analysis of 1856 signaling and 319 metabolic pathways. 
 
 
          <hr/>          
          <h2 id='gene_analysis'>Gene level analysis</h2>
          <hr/>
          PCA demonstrated that samples correlate between groups and show small variability within groups on gene expression level. 
          All three types of reconstructed human epidermis make up one big cluster.
          <br/><br/>
          <div>
          
          <!-- GENE Nav tabs -->
         <ul class="nav nav-tabs" role="tablist">
         <li role="presentation" class="active"><a href="#allgenes_tab" aria-controls="home1" role="tab" data-toggle="tab">ALL</a></li>
         <li role="presentation" ><a href="#nhk_tab" aria-controls="home" role="tab" data-toggle="tab">NHK</a></li>
         <li role="presentation"><a href="#type1_tab" aria-controls="profile" role="tab" data-toggle="tab">TYPE1</a></li>
         <li role="presentation"><a href="#type2_tab" aria-controls="messages" role="tab" data-toggle="tab">TYPE2</a></li>
         <li role="presentation"><a href="#type3_tab" aria-controls="settings" role="tab" data-toggle="tab">TYPE3</a></li>
         </ul>
         
         <!-- GENE Tab panes -->
  <div class="tab-content">
   <div role="tabpanel" class="tab-pane fade in active" id="allgenes_tab">
   
   <div id="genes-3d" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto"></div>
   
   
   </div>
  
    <div role="tabpanel" class="tab-pane fade " id="nhk_tab">
    
      <br/> 
          
          <!-- GENES NKK -->
          <div id="genes-nhk" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  <br/><br/>
  
  <h4>Table 1. L'Oreal_preprocessed_NHK</h4>
      <table id="tbl-gene-nhk" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Gene</th>
           <th>log2FoldChange</th>
           <th>pval</th>
           <th>padj</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
      
      <div role="tabpanel" class="tab-pane fade " id="type1_tab">
      <!-- GENES TYPE 1 -->
      <div id="genes-type1" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  <br/><br/>
  
  <h4>Table 2. L'Oreal_loreal_preprocessed_RhE (Type 1)</h4>
      <table id="tbl-gene-type1" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Gene</th>
           <th>log2FoldChange</th>
           <th>pval</th>
           <th>padj</th>
          </tr>
        </thead>  
      </table> 
      <br/><br/>
      </div>
      
      <div role="tabpanel" class="tab-pane fade " id="type2_tab">
      <!-- GENES TYPE 2 -->
      <div id="genes-type2" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  <br/><br/>
  
  <h4>Table 3. L'Oreal_loreal_preprocessed_RhE (Type 2)</h4>
      <table id="tbl-gene-type2" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Gene</th>
           <th>log2FoldChange</th>
           <th>pval</th>
           <th>padj</th>
          </tr>
        </thead>  
      </table> 
       <br/><br/>
       </div>
       
       <div role="tabpanel" class="tab-pane fade " id="type3_tab">
       <!-- GENES TYPE 3 -->
      <div id="genes-type3" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  <br/><br/>
  
  <h4>Table 4. L'Oreal_loreal_preprocessed_RhE (Type 3)</h4>
      <table id="tbl-gene-type3" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Gene</th>
           <th>log2FoldChange</th>
           <th>pval</th>
           <th>padj</th>
          </tr>
        </thead>  
      </table> 
       <br/><br/>
     </div>
     
  </div> <!--  end of tabs GENE -->
<hr/>
<h3>Notable genes</h3>
<hr/>
The analysis of gene expression of the components that are involved in formation and remodeling of ECM 
shows that RhE samples contain highly expressed enzymes that are connected with the degradation of ECM 
(such as MMPs, HYALs) and low expressed enzymes that are involved in formation of ECM (HAS1, COL1, FN1). 
Some of the major GF associated genes are significantly upregulated in RhE (for example, GH1) compared to
 NHE and NHEK, while some are downregulated (FGF1, EGF). Several GF genes and genes of GF receptors may 
 serve as a marker for NHEK (FGF1, EGF, WNT1).

<div>
 <!-- GENE Nav tabs -->
         <ul class="nav nav-tabs" role="tablist">
           {%for gene in lGenes%}
           <li role="presentation" {%if forloop.counter == 1 %}class="active"{%endif%}><a href="#{{gene}}_tab" aria-controls="home1" role="tab" data-toggle="tab">{{gene}}</a></li>
           {%endfor%}
         </ul>

   <div class="tab-content">
    {%for gene in lGenes%}
     {%if forloop.counter == 1 %}
     <div role="tabpanel" class="tab-pane fade in active" id="{{gene}}_tab">
     {%else%}
     <div role="tabpanel" class="tab-pane fade " id="{{gene}}_tab">
     {%endif%}
     <div id="{{gene}}" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto"></div>
   
     </div>
    {%endfor%}
   
   </div> <!--  end of tab panel -->
</div> <!-- end of Notable genes -->
<!-- ---------------------------------PATHWAY--------------------------------------------- -->
 <hr/> 
<h2 id='path_analysis'>Pathway Analysis</h2>
<hr/>


<h3>Signaling Pathways</h3>

NHE samples were selected as a norm for differential pathway analysis.<br/>
Among all examined samples NHEK samples show more perturbed pathways 
(1826 dysregulated pathways compared to 1784, 1765, 1768 for RhE type 1, type 2, type 3, respectively) 
and so these samples are more distant from NHE. Strong upregulation of Aurora B signaling pathway [4],
 PLK1-pathway [5], GSK3B ILK regulated pathway [6], repression of PTEN pathway [7] support the evidence
  that NHEK proliferate more actively than RhE samples. Also, pathway signature shows that RhE resemble
   mature skin with downregulation of early stage of skin development and repair processes pathways such
    as WNT pathway [8], EGF and EGFR pathways [9]. Growth hormone is well known to be actively involved 
    in skin development and repair [10] and is also inhibited along with IGF.
The group of Extracellular remodeling pathways is activated in NHEK, while in RhE samples it is repressed.

 <!-- PAthway Nav tabs -->
         <ul class="nav nav-tabs" role="tablist">
         <li role="presentation" class="active"><a href="#all_tab" aria-controls="settings" role="tab" data-toggle="tab">ALL</a></li>
         <li role="presentation" ><a href="#nhk_type1_tab" aria-controls="home" role="tab" data-toggle="tab">NHK-TYPE1</a></li>
         <li role="presentation"><a href="#nhk_type2_tab" aria-controls="profile" role="tab" data-toggle="tab">NHK-TYPE2</a></li>
         <li role="presentation"><a href="#nhk_type3_tab" aria-controls="messages" role="tab" data-toggle="tab">NHK-TYPE3</a></li>
         <li role="presentation"><a href="#type1_type2_tab" aria-controls="settings" role="tab" data-toggle="tab">TYPE1-TYPE2</a></li>
         <li role="presentation"><a href="#type1_type3_tab" aria-controls="settings" role="tab" data-toggle="tab">TYPE1-TYPE3</a></li>
         <li role="presentation"><a href="#type2_type3_tab" aria-controls="settings" role="tab" data-toggle="tab">TYPE2-TYPE3</a></li>
         
         </ul>
         
         <!-- PATHWAY Tab panes -->
  <div class="tab-content">
  <!-- PATH ALL -->
    <div role="tabpanel" class="tab-pane fade in active" id="all_tab">     <br/> 
    <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_path-all" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_path-all-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_path-all-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table>        
    
  <div id="venn_path_dynamic" class="" style="max-width:1000px;"></div> <!--  div to place dynamic tables from Venn clicks -->
  
  <h4>Table 5. All</h4>
      <table id="tbl-all" class="path table table-striped table-bordered path-dynamic" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>NHK</th>
           <th>Type1</th>
           <th>Type2</th>
           <th>Type3</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
  
    <!-- PATH NHK TYPE1 -->
    <div role="tabpanel" class="tab-pane fade " id="nhk_type1_tab">
    
      <br/> 
          
          
          <div id="path-nhk-type1" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  <hr/>
  <!--  <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_path-nhk-type1" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div>
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_path-nhk-type1-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_path-nhk-type1-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table> -->
  <h4>Table 6. NHK vs Type1</h4>
      <table id="tbl-nhk-type1" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>NHK</th>
           <th>Type1</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
      
      
      <!-- PATH NHK TYPE2 -->
    <div role="tabpanel" class="tab-pane fade " id="nhk_type2_tab">
     
          
          
          <div id="path-nhk-type2" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  <hr/>
  <!--  <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_path-nhk-type2" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_path-nhk-type2-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_path-nhk-type2-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table>-->
  <h4>Table 7. NHK vs Type2</h4>
      <table id="tbl-nhk-type2" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>NHK</th>
           <th>Type2</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
      
      <!-- PATH NHK TYPE3 -->
    <div role="tabpanel" class="tab-pane fade " id="nhk_type3_tab">
    
      <br/> 
          
          
          <div id="path-nhk-type3" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  <hr/>
  
  <!--  <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_path-nhk-type3" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_path-nhk-type3-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_path-nhk-type3-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table>-->
  
  <h4>Table 8. NHK vs Type3</h4>
      <table id="tbl-nhk-type3" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>NHK</th>
           <th>Type3</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
      
      <!-- PATH TYPE1 TYPE2 -->
    <div role="tabpanel" class="tab-pane fade " id="type1_type2_tab">
    
      <br/> 
          
          
          <div id="path-type1-type2" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  <hr/>
  
  <!--  <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_path-type1-type2" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_path-type1-type2-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_path-type1-type2-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table>-->
  
  <h4>Table 9. Type1 vs Type2</h4>
      <table id="tbl-type1-type2" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>Type1</th>
           <th>Type2</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
      
      <!-- PATH TYPE1 TYPE3 -->
    <div role="tabpanel" class="tab-pane fade " id="type1_type3_tab">
    
      <br/> 
          
          
          <div id="path-type1-type3" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  <hr/>
  
  <!--  <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_path-type1-type3" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_path-type1-type3-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_path-type1-type3-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table>-->
  
  <h4>Table 10. Type1 vs Type3</h4>
      <table id="tbl-type1-type3" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>Type1</th>
           <th>Type3</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
      
      
      <!-- PATH TYPE2 TYPE3 -->
    <div role="tabpanel" class="tab-pane fade " id="type2_type3_tab">
    
      <br/> 
          
          
          <div id="path-type2-type3" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  <hr/>
  
  <!--  <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_path-type2-type3" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_path-type2-type3-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_path-type2-type3-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table>-->
  
  <h4>Table 11. Type2 vs Type3</h4>
      <table id="tbl-type2-type3" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>Type2</th>
           <th>Type3</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
      
      

</div></div>

<hr/>
<!-- ---------------------------- METABOLIC------------------------ -->
<h3>Metabolic Pathways</h3>
<hr/>

NHE demonstrated more dysregulated metabolic pathways 
(281 compared to 256, 246, 252 for RhE type 1, type 2, type 3 respectively).<br/>
Metabolic pathways screening showed that all studied samples have a suppressed biosynthesis of
 ECM skin-associated polysaccharides, such as dermatan sulfate, heparan sulfate [11]. However,
  RhE samples demonstrate an activation of all metabolic pathways of cholesterol biosynthesis 
  and fatty acids metabolism. For example, we observed the activation of ester lipids and 
  arachidonic acid (polyunsaturated omega-6 fatty acid), linoleic acid (polyunsaturated omega-6 fatty acid)
   metabolism signaling pathways in RhE samples. Since cholesterol, fatty acids and their esters are major 
   components of stratum corneum [12] and all of the above mentioned pathways are not extremely perturbed 
   compared to the reference NHE, RhE resemble the signature of a "horny layer" of skin on a pathway level. 
To sum up, the models of RhE resemble mature epidermis with "horny layer" on a pathway and gene expression 
level. RhE type 2 samples are more similar to NHE on signaling and metabolic pathway levels.
 <!-- Meabolic PAthway Nav tabs -->
         <ul class="nav nav-tabs" role="tablist">
         <li role="presentation" class="active"><a href="#meta_all_tab" aria-controls="settings" role="tab" data-toggle="tab">ALL</a></li>
         <li role="presentation" ><a href="#meta_nhk_type1_tab" aria-controls="home" role="tab" data-toggle="tab">NHK-TYPE1</a></li>
         <li role="presentation"><a href="#meta_nhk_type2_tab" aria-controls="profile" role="tab" data-toggle="tab">NHK-TYPE2</a></li>
         <li role="presentation"><a href="#meta_nhk_type3_tab" aria-controls="messages" role="tab" data-toggle="tab">NHK-TYPE3</a></li>
         <li role="presentation"><a href="#meta_type1_type2_tab" aria-controls="settings" role="tab" data-toggle="tab">TYPE1-TYPE2</a></li>
         <li role="presentation"><a href="#meta_type1_type3_tab" aria-controls="settings" role="tab" data-toggle="tab">TYPE1-TYPE3</a></li>
         <li role="presentation"><a href="#meta_type2_type3_tab" aria-controls="settings" role="tab" data-toggle="tab">TYPE2-TYPE3</a></li>
         
         </ul>
         
         <!-- Meabolic PATHWAY Tab panes -->
  <div class="tab-content">
  <!-- Meabolic PATH ALL -->
    <div role="tabpanel" class="tab-pane fade in active" id="meta_all_tab">     <br/> 
          
  <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_meta-all" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_meta-all-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_meta-all-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table>
  <div id="venn_meta_dynamic" class="" style="max-width:1000px;"></div> <!--  div to place dynamic tables from Venn clicks -->
  <h4>Table 12. All</h4>
      <table id="meta_tbl-all" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>NHK</th>
           <th>Type1</th>
           <th>Type2</th>
           <th>Type3</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
  
    <!-- Meabolic PATH NHK TYPE1 -->
    <div role="tabpanel" class="tab-pane fade " id="meta_nhk_type1_tab">
    
      <br/> 
          
          
     <div id="meta_path-nhk-type1" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  
  <!--  <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_meta-nhk-type1" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div>
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_meta-nhk-type1-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_meta-nhk-type1-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table>-->
    
  <h4>Table 13. NHK vs Type1</h4>
      <table id="meta_tbl-nhk-type1" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>NHK</th>
           <th>Type1</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
      
      
      <!-- PATH NHK TYPE2 -->
    <div role="tabpanel" class="tab-pane fade " id="meta_nhk_type2_tab">
    
      <br/> 
          
          
  <div id="meta_path-nhk-type2" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  
  <!--  <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_meta-nhk-type2" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div>
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_meta-nhk-type2-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_meta-nhk-type2-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table>-->
    
  <h4>Table 14. NHK vs Type2</h4>
      <table id="meta_tbl-nhk-type2" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>NHK</th>
           <th>Type2</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
      
      <!-- Meabolic PATH NHK TYPE3 -->
    <div role="tabpanel" class="tab-pane fade " id="meta_nhk_type3_tab">
    
      <br/> 
          
          
  <div id="meta_path-nhk-type3" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  
  <!--  <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_meta-nhk-type3" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div>
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_meta-nhk-type3-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_meta-nhk-type3-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table>-->
    
  <h4>Table 15. NHK vs Type3</h4>
      <table id="meta_tbl-nhk-type3" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>NHK</th>
           <th>Type3</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
      
      <!-- Meabolic PATH TYPE1 TYPE2 -->
    <div role="tabpanel" class="tab-pane fade " id="meta_type1_type2_tab">
    
      <br/> 
          
          
  <div id="meta_path-type1-type2" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  
  <!--  <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_meta-type1-type2" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div>
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_meta-type1-type2-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_meta-type1-type2-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table>-->
  <h4>Table 16. Type1 vs Type2</h4>
      <table id="meta_tbl-type1-type2" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>Type1</th>
           <th>Type2</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
      
      <!-- Meabolic PATH TYPE1 TYPE3 -->
    <div role="tabpanel" class="tab-pane fade " id="meta_type1_type3_tab">
    
      <br/> 
          
          
  <div id="meta_path-type1-type3" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
 
  <!--  <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_meta-type1-type3" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div>
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_meta-type1-type3-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_meta-type1-type3-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table>-->
  <h4>Table 17. Type1 vs Type3</h4>
      <table id="meta_tbl-type1-type3" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>Type1</th>
           <th>Type3</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
      
      
      <!--Meabolic  PATH TYPE2 TYPE3 -->
    <div role="tabpanel" class="tab-pane fade " id="meta_type2_type3_tab">
    
      <br/> 
          
          
  <div id="meta_path-type2-type3" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">Loading plot...</div>
  
  <!--  <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_meta-type2-type3" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div>
       Up and down regulated pathways
       </td>
       <td>
       <div id="venn_meta-type2-type3-up" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Up regulated pathways
       </td>
       <td>
       <div id="venn_meta-type2-type3-down" style="min-width: 300px; height: 300px; max-width: 300px; margin: 0 auto">Loading diagram...</div> 
       Down regulated pathways
       </td>
      </tr>
    </table>-->
  <h4>Table 18. Type2 vs Type3</h4>
      <table id="meta_tbl-type2-type3" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Pathway Name</th>
           <th>Type2</th>
           <th>Type3</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
      </div>
      
      

</div>
<hr/>
<h2 id='public_domain'>Public domain screening</h2>
<hr/>
<div dir="ltr">
<h4>Table 13. Datasets summary</h4>
    <table class='table table-bordered table-striped' style="width:95%">
        <colgroup>
            <col width="106"/>
            <col width="95"/>
            <col width="113"/>
            <col width="39"/>
            <col width="192"/>
            <col width="127"/>
        </colgroup>
        <thead>
        <tr>
                <th>
                    <p dir="ltr">
                        Name
                    </p>
                </th>
                <th>
                    <p dir="ltr">
                        Platform
                    </p>
                </th>
                <th>
                    <p dir="ltr">
                        Groups of samples
                    </p>
                </th>
                <th>
                    <p dir="ltr">
                        N
                    </p>
                </th>
                <th>
                    <p dir="ltr">
                        Cell source
                    </p>
                </th>
                <th>
                    <p dir="ltr">
                        Model
                    </p>
                </th>
            </tr>
        </thead>
        <tbody>
            
            <tr>
                <td rowspan="4">
                    <p dir="ltr">
                        GSE19330
                    </p>
                </td>
                <td rowspan="4">
                    <p dir="ltr">
                        HG-U133 Plus 2
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        Epiderm_219
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        5
                    </p>
                </td>
                <td rowspan="4">
                    <br/>
                    <p dir="ltr">
                        epidermal keratinocytes
                    </p>
                    <br/>
                    <br/>
                    <br/>
                    <br/>
                </td>
                <td rowspan="4">
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        Epiderm_254
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        5
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        Epiderm_926
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        5
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        Epiderm_1188
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        5
                    </p>
                </td>
            </tr>
            <tr>
                <td rowspan="4">
                    <br/>
                    <p dir="ltr">
                        GSE17539
                    </p>
                </td>
                <td rowspan="4">
                    <br/>
                    <p dir="ltr">
                        HG-U133 Plus 2
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        NHS
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        4
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        whole skin
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        whole skin
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        RHS_3days
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        3
                    </p>
                </td>
                <td rowspan="3">
                    <p dir="ltr">
                        epidermal keratinocytes + dermal fibroblasts
                    </p>
                </td>
                <td rowspan="3">
                    <p dir="ltr">
                        whole skin
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        RHS_7days
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        3
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        RHS_14days
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        3
                    </p>
                </td>
            </tr>
            <tr>
                <td rowspan="4">
                    <p dir="ltr">
                        GSE3204
                    </p>
                </td>
                <td rowspan="4">
                    <p dir="ltr">
                        HG-U133A
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        RHS
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        4
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermal keratinocytes + dermal fibroblasts
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        whole skin
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        NHDF
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        4
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        dermal fibroblasts
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        dermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        NHEK
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        3
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermal keratinocytes
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        NHS
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        4
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        whole skin
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        whole skin
                    </p>
                </td>
            </tr>
            <tr>
                <td rowspan="2">
                    <p dir="ltr">
                        GSE11418
                    </p>
                </td>
                <td rowspan="2">
                    <p dir="ltr">
                        HG-U133 Plus 2
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        NHDF_6pas_neo
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        2
                    </p>
                </td>
                <td rowspan="2">
                    <p dir="ltr">
                        dermal fibroblasts
                    </p>
                </td>
                <td rowspan="2">
                    <p dir="ltr">
                        dermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        NHDF_13pas_neo
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        3
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        GSE63626
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        HG-U133 Plus 2
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        NHDF
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        4
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        dermal fibroblasts
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        dermis
                    </p>
                </td>
            </tr>
            <tr>
                <td rowspan="3">
                    <p dir="ltr">
                        GSE30355
                    </p>
                </td>
                <td rowspan="3">
                    <p dir="ltr">
                        HG-U133 Plus 2
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        NHEK
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        10
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        cultured epidermal keratinocytes
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        NHEK_sort
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        10
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        flow cytometry
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        inflamed epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        NHEK_las
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        10
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        isolated by laser capture microscopy
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td rowspan="4">
                    <p dir="ltr">
                        GSE42114
                    </p>
                </td>
                <td rowspan="4">
                    <p dir="ltr">
                        HG-U133 Plus 2
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        Basal_NHE
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        3
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        Suprabasal_NHE
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        3
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        Reticular_NHD
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        3
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        dermis
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        dermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        NHE
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        10
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        GSE8056
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        HG-U133 Plus 2
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        NHS
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        3
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        whole skin
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        whole skin
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        GSE62333
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        HuGene-1_1-st
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        NHDF
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        20
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        dermal fibroblasts
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        dermis
                    </p>
                </td>
            </tr>
            <tr>
                <td rowspan="5">
                    <p dir="ltr">
                        LÃ“real
                    </p>
                </td>
                <td rowspan="5">
                    <p dir="ltr">
                        HuGene-1_0-st
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        NHE
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        4
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        NHEK
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        4
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermal keratinocytes
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        RHE_type_1
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        4
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermal keratinocytes
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        RHE_type_2
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        4
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermal keratinocytes
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        RHE_type_3
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        3
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermal keratinocytes
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        GSE67527
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        HuGene-1_0-st
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        NHDF
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        4
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        dermal fibroblasts
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        dermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        GSE25400
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        HuGene-1_0-st
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        EpiDerm
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        2
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermal keratinocytes
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        GSE48459
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        HuGene-1_0-st
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        EpiDerm
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        4
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermal keratinocytes
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        GSE48761
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        HuGene-1_0-st
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        NHDF
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        10
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        dermal fibroblasts
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        derma
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        GSE28315
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        HuGene-1_0-st
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        NHE
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        6
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        Mainly epidermis with minimal dermis
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        epidermis
                    </p>
                </td>
            </tr>
        </tbody>
    </table>
</div>

We obtained gene expression data for native skin, different skin layers and skin substitutes 
(engineered skin/engineered epidermis/ NHDF/NHEK) from public domain (GEO and ArrayExpress):

<ul>
  <li> native human skin
       <ul>
         <li>whole skin</li>
         <li>epidermis</li>
         <li>dermis</li>
       </ul>
   </li>
   <li>cultured skin substitutes
       <ul>
         <li>reconstructed epidermis</li>
         <li>reconstructed human skin</li>
         <li>epidermal skin-derived keratinocytes</li>
         <li>dermal skin-derived fibroblasts</li>
       </ul>
   </li>
</ul>
<br/>
 Data were profiled on different microarray platforms, so we performed harmonization in order 
 to correct the difference between platforms.
<ul>
  <li>NHE_LÃ“real samples show strong correlation (r=0,86) with all the three types of reconstructed epidermis (RhE type 1, type 2, type 3) 
      as well as other NHE samples on</li>
  <li>NHEK_LÃ“real samples correlate better with reconstructed epidermis than with native skin.</li>
  <li>NHDF samples show the weakest correlation with native skin on gene expression level among all samples</li>
</ul>
<hr/>
<img src='/static/report/loreal/All_genes.png' style='width:95%' />
<h4>Figure 1. Native skin compared with different kinds of skin substitutes </h4>
<hr/>
Gene expression data suffers from the "curse of dimensionality" and so requires special approaches 
to work with. One possible way to deal with dimensionality is to select a subset of features 
(genes in our case) that is related to the process. We selected 192 genes that are associated with
 ECM formation and remodeling and 89 core genes that are GFs or involved in regulation of GFs that are 
 linked to the processes of skin development and proliferation.
<hr/>

<img src='/static/report/loreal/ECM_corplot.png' style='width:95%' />
<h4>Figure 2. ECM associated genes.</h4>
<hr/>
<ul>
  <li>RhE type 1 shows similarity to native human skin on ECM associated gene expression level.</li>
  <li>NHDF samples do not correlate with NHE samples and all reconstructed epidermis samples (RhE type 1, 
  type 2, type 3 and Epiderm samples except Epiderm GSE48459 samples)</li>
  <li>NHS samples and whole skin substitutes (such as RHS from GSE3204 and GSE17539 datasets) group with 
  NHDF samples and Reticual Dermis samples on ECM associated gene expression level</li>
</ul>
<hr/>
<img src='/static/report/loreal/GF.png' style='width:95%' />
<h4>Figure 3. GFs genes</h4>
<hr/>
<ul>
  <li>RhE type 2 shows stronger correlation (r=0,97) with NHE LÃ“real samples than RhE type 1 and type 3.</li>
</ul>
<br/>
Gene subsetting analysis demonstrated that samples could be divided into 2 groups that show "dermis" 
signature and "epidermis" signature.<br/>
Also, analysis allows to determine outliers from public domain, such as GSE48459 (EpiDerm samples) that 
strangely resemble "dermis" signature, while EpiDerm is a model of skin epidermis and is supposed to show 
"epidermis" signature.


<hr/>  
  <h2  id='references'>References</h2>
   
   <ol style="font-size:9pt;">
     <li>McCall, Matthew N., Benjamin M. Bolstad, and Rafael A. Irizarry. "Frozen robust multiarray analysis
      (fRMA)." Biostatistics 11.2 (2010): 242-253.</li>
     <li>Benjamini, Y., and Y. Hochberg. 1995. Controlling the false discovery rate: a practical and 
     powerful approach to multiple testing. Journal of the Royal Statistical Society B 57: 289-300.</li>
      <li>Buzdin AA, Zhavoronkov AA, Korzinkin MB, Venkova LS, Zenin AA, Smirnov PY, Borisov NM. 
      Oncofinder, a new method for the analysis of intracellular signaling pathway activation using 
      transcriptomic data. Front Genet. 2014; 5:55.</li>
      <li>Fu, J., Bian, M., Jiang, Q. & Zhang, C. Roles of Aurora kinases in mitosis and tumorigenesis. 
      Mol. Cancer Res. 5, 1â€“10 (2007).</li>
      <li>Vaid, R. et al. Functions of Polo-Like Kinases: A Journey From Yeast To Humans. Protein Pept.
       Lett. 23, 185â€“197 (2016).</li>
       <li>Rayasam, G. V., Tulasi, V. K., Sodhi, R., Davis, J. A. & Ray, A. Glycogen synthase kinase 3: 
       more than a namesake. Br. J. Pharmacol. 156, 885â€“898 (2009).</li>
       <li>Chalhoub, N. & Baker, S. J. PTEN and the PI3-Kinase Pathway in Cancer. Annu. Rev. Pathol. Mech. 
       Dis. 4, 127â€“150 (2009).</li>
       <li>Augustin, I. Wnt signaling in skin homeostasis and pathology. J. Dtsch. Dermatol. Ges. 13, 302â€“306 (2015).</li>
       <li> Pastore, S., Mascia, F., Mariani, V. & Girolomoni, G. The Epidermal Growth Factor Receptor System in Skin 
       Repair and Inflammation. J. Invest. Dermatol. 128, 1365â€“1374 (2008).</li>
       <li>PÃ³voa, G. & Diniz, L. M. Growth hormone system: skin interactions. An. Bras. Dermatol. 86, 
       1159â€“65 (2011).</li>
       <li>Trowbridge, J. M. & Gallo, R. L. Dermatan sulfate: new functions from an old glycosaminoglycan. 
       Glycobiology 12, 117Râ€“25R (2002).</li>
       <li>Wertz, P. W. Epidermal lipids. Semin. Dermatol. 11, 106â€“13 (1992).</li>
   
   </ol>

</div>


<div style='min-height:500px'></div>
            
          </div>
          
          
          
        </div>
      </div>
      
    </div>
    <div class="bar" style="heigh:100%">
      <div class="inner h100" style="heigh:100%">
        <div class="top"><img src="/static/images/logo.svg"></div>
        <div class="menu">
          <ul class="list-unstyled" >
            <li class="title">Table of contents: </li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#input_data">Input data</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#methods">Methods</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#gene_analysis">Gene level analysis</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#path_analysis">Pathway analysis</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#public_domain">Public domain screening</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#references">References</a></li>
          </ul>
        </div>
        <div class="bottom">
          <div class="inner"> Insilico Medicine Inc., Johns Hopkins <br/>University Eastern, ETC B301, 1101 East <br/>33rd Street, Baltimore, MD 21218 </div>
        </div>
      </div>
    </div>
    
    
    <script src="https://cdn.datatables.net/1.10.10/js/jquery.dataTables.min.js" type="text/javascript"></script>
    <script src="https://cdn.datatables.net/1.10.10/js/dataTables.bootstrap.min.js" type="text/javascript"></script>
    <script src="https://cdn.datatables.net/buttons/1.1.0/js/dataTables.buttons.min.js" type="text/javascript"></script>
    <script src="https://cdn.datatables.net/buttons/1.1.0/js/buttons.bootstrap.min.js" type="text/javascript"></script>
    <script src="//cdn.datatables.net/buttons/1.1.0/js/buttons.flash.min.js" type="text/javascript"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/jszip/2.5.0/jszip.min.js" type="text/javascript"></script>
    <script src="//cdn.rawgit.com/bpampuch/pdfmake/0.1.18/build/pdfmake.min.js" type="text/javascript"></script>
    <script src="//cdn.rawgit.com/bpampuch/pdfmake/0.1.18/build/vfs_fonts.js" type="text/javascript"></script>
    <script src="//cdn.datatables.net/buttons/1.1.0/js/buttons.html5.min.js" type="text/javascript"></script>
    <script src="//cdn.datatables.net/buttons/1.1.0/js/buttons.print.min.js" type="text/javascript"></script>
    
    <script src="/static/report/loreal/js/venn.js/venn.js" type="text/javascript"></script>
    <script src="/static/report/loreal/js/tables.js" type="text/javascript"></script>
    <script src="/static/report/loreal/js/scatter.js" type="text/javascript"></script>
    <script src="/static/report/loreal/js/boxplot.js" type="text/javascript"></script>
    
    <script src="/static/report/loreal/js/venn-diagram.js" type="text/javascript"></script>
    <script src="/static/report/loreal/js/3d.js" type="text/javascript"></script>
    
    
    
    <!-- MODAL FOR PATHS  -->
    <div id="pathmodal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
 <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;  </button>
        <h4 class="modal-title" id="myModalLabel">Modal title</h4>
      </div>
      <div class="modal-body" id='modalBody'>
        
      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
      </div>
    </div>
  </div>
</div>
    
    <!--
    script.
      $(function () {
        svgPanZoom('#pathway', {
          zoomEnabled: true,
          controlIconsEnabled: true,
          fit: true,
          center: true
        });
      });
    -->
  </body>
</html>
