<!DOCTYPE html>
<html lang="en">
  <head>
    <title> {{report.title_short}} | Insilico Medicine</title>
    <link href="/static/bower_components/bootstrap-stylus/dist/bootstrap.min.css" rel="stylesheet" type="text/css">
    <script src="//code.jquery.com/jquery-2.1.4.min.js" type="text/javascript"></script>
    <script src="/static/js/oncofinder2.js" type="text/javascript"></script>
    <script src="/static/bower_components/bootstrap-stylus/dist/bootstrap.min.js" type="text/javascript"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore.js" type="text/javascript"></script>
    <script src="/static/js/d3.v3.min.js" type="text/javascript"></script>
    <script src="https://cdn.datatables.net/1.10.10/js/jquery.dataTables.min.js" type="text/javascript"></script>
    <script src="http://code.highcharts.com/highcharts.js" type="text/javascript"></script>
    <script src="https://code.highcharts.com/highcharts-more.js"  type="text/javascript"></script>
    <script src="https://code.highcharts.com/highcharts-3d.js"  type="text/javascript"></script>
    <script src="http://code.highcharts.com/modules/exporting.js"  type="text/javascript"></script>
    <link href="/static/css/report_style.css" rel="stylesheet" type="text/css">
    <link href="https://cdn.datatables.net/1.10.10/css/dataTables.bootstrap.min.css" rel="stylesheet" type="text/css">
    <link href="https://cdn.datatables.net/buttons/1.0.3/css/buttons.dataTables.min.css" rel="stylesheet">
    
    <link rel="shortcut icon" href="/static/images/favicon.ico" type="image/x-icon">
    <link rel="icon" href="/static/images/favicon.ico" type="image/x-icon">
    
    <link href="/static/report/bt/js/opentip.css" rel="stylesheet" type="text/css">
    <link href="/static/report/bt/js/volcano.css" rel="stylesheet" type="text/css">
    
  </head>
  <body style='font-size: 14px;'>
    <div class="content h100">
      <div class="main">
        <div class="inner">
         <div>
         <span class='pull-left' style='font-size:11pt; margin:15px;'>
         <img src='/static/images/insilico.png' style="width:40px;"/>
         Insilico Medicine - {{report.title_short}}</span>
         <span class='pull-right' style='font-size:11pt; margin:15px;'>Access Level: Z-II, <br/>Confidential </span>
         </div><br style="clear:both;"/>
          <div class='text-center'>
            <h1>{{report.title}}</h1>
          </div>
          
          
          <div style="margin: 50px 0 0 50px; padding: 0 50px 0 0;">
            <h4 style='font-weight:bold; font-size:12pt;'>Organization: </h4>
            <span style="font-size:13pt">{{report.organization}}</span>
            <br/><br/>
            <!-- <h4 style='font-weight:bold; font-size:12pt;'>Data Quality Assessment: </h4>-->
 
            
            
            <ul class="list-unstyled" style='font-size: 11pt; margin: 50px 0 0 0;' >
            <li class="title">Table of contents: </li>

            
            <!--<li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#input_data">Input data</a></li>
              <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#dl_analysis">Deep Learning</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#kinetic_analysis">Kinetic pathway modeling</a></li>-->
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#gene_analysis">Gene level analysis</a></li>
            <!--  <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tf_analysis">Transcription Factor Analysis</a></li>-->
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#path_analysis">Pathway Analysis</a></li>            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#dl_analysis">Drug Repurposing</a></li>
           
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#biomap">BioMAP inference</a></li> 
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#biomap_l1000">L1000-BioMAP correlation</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#hit">Hit qualification</a></li>  
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tox">Tox groups analysis</a></li> 
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tox_analysis">Toxcast analysis</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#sim_analysis">Similarity to LINCS dataset</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#barcodes">Barcodes for L1000 data</a></li>
           
            {%if 'gsk' in report.slug %}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#conclusions">Discussion</a></li>
            {%endif%}
            <!--  
            
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#cor_analysis">Public domain screening</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#pt_analysis">Scoring of potential targets</a></li>
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#conclusions">Discussion</a></li>
            -->
          </ul>
          
          {%if report.input_data_text%}
          <h2 id="input_data">Experimental design and RNA-seq data analysis</h2>
          <hr/>
          {{report.input_data_text|safe}}

          {%endif%}
          

{%if 'dmso' in report.slug %}
<center>
<img src='/media/report-portal/gsk_prj3_dmso/prj3_dmso.png' style="width:800px"/>
</center>
{%endif%}
{%if 'prj3_l1k' in report.slug %}
<center>
<img src='/media/report-portal/gsk_prj3_l1k/prj3_l1k.png' style="width:800px"/>
</center>
<h4>PCA gene weights</h4>
<table id="tbl-top_l1k" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr >
           <th>Probeset</th>
           <th>Symbol</th>
           <th>Comp1</th>
           <th>Comp2</th>
           <th>Comp3</th>
          </tr>
        </thead>  
      </table>



{%endif%}
{%if 'prj3_mm' in report.slug %}
<center>
<img src='/media/report-portal/gsk_prj3_mm/prj3_mm.png' style="width:800px"/>
</center>
{%endif%}
{%if 'nocmp' in report.slug %}
<center>
<img src='/media/report-portal/gsk_prj3_nocmp/prj3_nocmp.png' style="width:800px"/>
</center>
{%endif%}

<!-------------------------------------  GENE level  -->          
          <hr/>
          <h2 id='gene_analysis'>Gene level analysis</h2>
          <hr/>
          {%if report.gene_text%}
          {{report.gene_text|safe}}
          <br/><br/>
          {%endif%}


<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_genes'>Show Data</button>
</center>

<div id='hide_genes' style='display:none;'> <!-- Div to hide and load on button click -->

<p>
Provided samples were distributed into comparison groups. 
Each comparison included stimulated or perturbed data and unstimulated controls. 
For gene level analysis each comparison was analyzed by limma algorithm.
</p>

           
   <!-- GENE Nav tabs -->      
   <ul class="nav nav-tabs" role="tablist">
     {%for group in report.genegroup_set.all%}
       <li role="presentation" {%if forloop.counter == 1 %}class="active"{%endif%}>
         <a href="#{{group.get_slug}}_tab" aria-controls="home1" role="tab" data-toggle="tab" id='vol_boot_{{group.get_slug}}'>
           {{group.name}} vs {{report.norm_name}}
         </a>
       </li>
     {%endfor%}
   </ul>
         
   <div class="tab-content">
    {%for group in report.genegroup_set.all%}
     {%if forloop.counter == 1 %}
     <div role="tabpanel" class="tab-pane fade in active" id="{{group.get_slug}}_tab">
     {%else%}
     <div role="tabpanel" class="tab-pane fade " id="{{group.get_slug}}_tab">
     {%endif%}
       <div id="{{group.get_slug}}_volcano" 
            style="min-width: 800px; height: 400px; max-width: 800px; margin: 5px auto">
            <span>Loading plot...</span>
       </div> <!-- Volcano plot div -->
       
       
       <h4>Table <span class='tbl_num'></span>. {{group.name}} vs {{report.norm_name}}</h4>
       <table id="tbl-{{group.get_slug}}" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Gene</th>
           <th>log2FoldChange</th>
           {%if 'nova' in report.slug %}
           <th>p-value</th>
           {%else%}
           <th>adj. p-value</th>
           {%endif%}
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
    {%endfor%}
   
   </div> <!--  end of tab-content div -->
   
<!-- -------------------- GENE VENN  -->
<br/>
<h3>Gene level comparison</h3>
<br/>

<!-- Gene level comparison Nav tabs -->
<ul class="nav nav-tabs" role="tablist">
         <li role="presentation" class="active"><a href="#gene_all_tab" aria-controls="settings" role="tab" data-toggle="tab">ALL</a></li>
         {%for perm in report.get_gene_permutations%}
         <li role="presentation" ><a href="#gene_{{perm.strip}}_tab" aria-controls="settings" role="tab" data-toggle="tab" id='venn_boot_gene_{{perm.strip}}'>{{perm}}</a></li>
         {%endfor%}
</ul>

<!-- GEne VEnn Tab panes -->
  <div class="tab-content">
  
  <div role="tabpanel" class="tab-pane fade in active " id="gene_all_tab">     <br/> 
    <h4>Table . All genes</h4>
    <table id="tbl-gene_all" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Gene</th>
           {%for group in report.get_all_gene_groups_as_list%}
            <!--  {% if forloop.counter > 1 %}<th>{{group}}</th> {%endif%}-->
            <th>{{group}}</th>
           {%endfor%}
          </tr>
        </thead>  
      </table>       
   <br/>
  
  </div>
  
  {%for perm in report.get_gene_permutations%}
  <div role="tabpanel" class="tab-pane fade " id="gene_{{perm.strip}}_tab">     <br/> 
    <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_gene-{{perm.strip}}-up" style="min-width: 450px; height: 400px; max-width: 450px; margin: 0 auto">Loading diagram...</div> 
       Significantly upregulated genes
       </td>
       <td>
       <div id="venn_gene-{{perm.strip}}-down" style="min-width: 450px; height: 400px; max-width: 450px; margin: 0 auto">Loading diagram...</div> 
       Significantly downregulated genes
       </td>
      </tr>
    </table>        
   <br/>
  <div id="venn_gene_dinamic_{{perm.strip}}" class="" style="max-width:1000px;"></div> <!--  div to place dinamic tables from Venn clicks -->
  </div>
  {%endfor%}
  
  </div>
<hr/> 
{%if report.notable_genes%}
   <h3>Notable genes</h3>
<hr/>


 <!-- Notable GENE Nav tabs -->
         <ul class="nav nav-tabs" role="tablist">
           {%for gene in report.get_notable_genes_list%}
           <li role="presentation" {%if forloop.counter == 1 %}class="active"{%endif%}><a href="#{{gene}}_tab" aria-controls="home1" role="tab" data-toggle="tab">{{gene}}</a></li>
           {%endfor%}
         </ul>

   <div class="tab-content">
    {%for gene in report.get_notable_genes_list%}
     {%if forloop.counter == 1 %}
     <div role="tabpanel" class="tab-pane fade in active" id="{{gene}}_tab">
     {%else%}
     <div role="tabpanel" class="tab-pane fade " id="{{gene}}_tab">
     {%endif%}
     <div id="{{gene}}" style="min-width: 800px; height: 400px; width: 1100px; margin: 0 auto">Loading box plot...</div>
   
     </div>
    {%endfor%}
   
   </div> <!--  end of tab panel -->
{%endif%}

</div> <!--  end of hide div -->




<!-- ---------------------------------PATHWAY--------------------------------------------- -->
 <hr/> 
<h2 id='path_analysis'>Pathway Analysis</h2>
<hr/>
{%if report.path_text%}
          {{report.path_text|safe}}
          <br/><br/>
          {%endif%}

<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_path'>Show Data</button>
</center>

<div id='hide_path' style='display:none;'> <!-- Div to hide and load on button click -->

<p>Pathway level analysis was performed on several curated pathway databases 
(KEGG, NCI, Reactome, SABiosciences) comprising 3117 signaling and metabolic pathways in total. 
To estimate pathway activation we used iPANDA algorithm.
        </p><br/>
<h3>Signaling Pathways</h3>
<hr/> 

<!-- Signaling Pathways level comparison Nav tabs -->
<ul class="nav nav-tabs" role="tablist">
         <li role="presentation" class="active"><a href="#path_all_tab" aria-controls="settings" role="tab" data-toggle="tab">ALL</a></li>
         <li role="presentation" ><a href="#path_tox_tab" aria-controls="settings" role="tab" data-toggle="tab">Tox</a></li>
         {%for perm in report.get_path_permutations%}
         <li role="presentation" ><a href="#path_{{perm.strip}}_tab" aria-controls="settings" role="tab" data-toggle="tab" id="venn_boot_path_{{perm.strip}}">{{perm}}</a></li>
         {%endfor%}
</ul>

<!-- PAth VEnn Tab panes -->
  <div class="tab-content">
  
  <div role="tabpanel" class="tab-pane fade in active " id="path_all_tab" style='overflow-x: auto;'>     <br/> 
    <h4>Table . All signaling pathways</h4>
    <table id="tbl-path_all" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th id='pathname_th'>Pathway</th>
           {%for group in report.get_all_gene_groups_as_list%}
            <!--  {% if forloop.counter > 1 %}<th>{{group}}</th> {%endif%}-->
            <th>{{group}}</th>
           {%endfor%}
          </tr>
        </thead>  
      </table>       
   <br/>
  
  </div>
  
  <div role="tabpanel" class="tab-pane fade  " id="path_tox_tab" style='overflow-x: auto;'>     <br/> 
    <h4>Table . Tox signaling pathways</h4>
    <table id="tbl-path_tox" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th id='pathname_th'>Pathway</th>
           {%for group in report.get_all_gene_groups_as_list%}
            <!--  {% if forloop.counter > 1 %}<th>{{group}}</th> {%endif%}-->
            <th>{{group}}</th>
           {%endfor%}
          </tr>
        </thead>  
      </table>       
   <br/>
  
  </div>
  
  {%for perm in report.get_path_permutations%}
  <div role="tabpanel" class="tab-pane fade " id="path_{{perm.strip}}_tab">     <br/> 
    <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_path-{{perm.strip}}-up" style="min-width: 450px; height: 400px; max-width: 450px; margin: 0 auto">Loading diagram...</div> 
       Upregulated pathways
       </td>
       <td>
       <div id="venn_path-{{perm.strip}}-down" style="min-width: 450px; height: 400px; max-width: 450px; margin: 0 auto">Loading diagram...</div> 
       Downregulated pathways
       </td>
      </tr>
    </table>        
   <br/>
  <div id="venn_path_dinamic_{{perm.strip}}" class="" style="max-width:1000px;"></div> <!--  div to place dinamic tables from Venn clicks -->
  </div>
  {%endfor%}
  
  </div>
  
  <hr/>
  <h3>BioMAP diversity data</h3>
  <br/>
  <p>
  Here we collected all BioMAP samples in complete diversity set and all mapped biomarkers to corresponding genes, 
  then for each biomarker we calculated the ratio "Log Ratio"/"Significance prediction envelope at 95%"
   aggregated", after that for genes present across several assays we took the mean of that ratio and mapped 
   the resulting values onto the pathway database.
  </p>
  <br/>
  
  <div  style='overflow-x: auto;'>
    <table id="tbl-pathbiomap" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
    </table>
  </div>
          
</div>  <!-- end of hide div -->
<script>

$(function(){
	
	$.ajax({
        type: 'GET',
        dataType: 'json',
        url: '/report-portal/report-ajaxtablecolumns/',
        data:{
   	         reportID: {{report.id}},
   	         file_name: 'biomap_diversity.pathway_scores.csv'
           },
        success: function(d) {
        	
        	let thead = '<tr>';
        	
        	for (let col of d.columns) {
        		
        		thead+='<th>'+col+'</th>';
        	}
        	
        	thead+='</tr>';
        	
        	$("#tbl-pathbiomap").find("thead").html(thead);
        	
        	var tbl_sim = $("#tbl-pathbiomap").DataTable( {
        		"paging":   true,
        	    "iDisplayLength": 20,
        	    "ordering": true,
        	    "order": [],
        	    "info":     false,
        	    "dom": 'Bfrtip',
        	    "scrollX": false,
        	    "bAutoWidth": false,
        	    "buttons": [
        	               {extend: 'csv', title: 'tbl-pathbiomap'},
        	               {extend: 'print', title: 'tbl-pathbiomap'}
        	               
        	          ],
        	    
        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
        	    	     'type': 'GET',
        	    	     'data':{
        	    	    	     reportID: {{report.id}},
        	    	    	     file_name: 'biomap_diversity.pathway_scores.csv'
        	    	            },        	      
        	    
        	},"deferRender": true });
        }
    });
	
	
});

</script>
<!------------- End of Pathway Analysis -->


<!-- ---------------------------------Deep Learning--------------------------------------------- -->


 <hr/> 
<h2 id='dl_analysis'>Drug Repurposing</h2>

<hr/>
<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_dl'>Show Data</button>
</center>

<div id='hide_dl' style='display:none;'> <!-- Div to hide and load on button click -->

<h3>Therapeutic use</h3>
<hr/>
   <!-- DL Nav tabs -->      
   <ul class="nav nav-tabs" role="tablist">
      <li role="presentation" class="active">
         <a href="#overall_dlfc_tab" aria-controls="home1" role="tab" data-toggle="tab" id='dlfc_boot_overall'>
           Overall
         </a>
       </li>
     {%for group in report.genegroup_set.all%}
       <li role="presentation" >
         <a href="#{{group.get_slug}}_dlfc_tab" aria-controls="home1" role="tab" data-toggle="tab" id='dlfc_boot_{{group.get_slug}}'>
           {{group.name}}
         </a>
       </li>
     {%endfor%}
   </ul>
         
   <div class="tab-content">
     <div role="tabpanel" class="tab-pane fade in active" id="overall_dlfc_tab">
     <br/>
     Loading...
     <br/><br/><br/>
     <!--  
      <ul class="nav nav-tabs" role="tablist">
        <li role="presentation" class="active">
          <a href="#overall_dlfc_tab10"  role="tab" data-toggle="tab" id='dlfc_boot_overall10'>
          Drug 10</a>
          </li>
          <li role="presentation" >
          <a href="#overall_dlfc_tab12"  role="tab" data-toggle="tab" id='dlfc_boot_overall12'>
          Drug 12</a>
          </li>
          <li role="presentation" >
          <a href="#overall_dlfc_tab15"  role="tab" data-toggle="tab" id='dlfc_boot_overall15'>
          Drug 15</a>
          </li>
          <li role="presentation" >
          <a href="#overall_dlfc_tab29"  role="tab" data-toggle="tab" id='dlfc_boot_overall29'>
          Drug 29</a>
          </li>
      </ul>-->
      <!--  
      <div class="tab-content">
        <div role="tabpanel" class="tab-pane fade in active" id="overall_dlfc_tab10">
          <div id="dlfc_overall10" style="min-width: 800px; height: 500px; max-width: 800px; margin: 0 auto">
            <span>Loading bar plot...</span>
          </div>
        </div>
        <div role="tabpanel" class="tab-pane fade " id="overall_dlfc_tab12">
          <div id="dlfc_overall12" style="min-width: 800px; height: 500px; max-width: 800px; margin: 0 auto">
            <span>Loading bar plot...</span>
          </div>
        </div>
        <div role="tabpanel" class="tab-pane fade " id="overall_dlfc_tab15">
          <div id="dlfc_overall15" style="min-width: 800px; height: 500px; max-width: 800px; margin: 0 auto">
            <span>Loading bar plot...</span>
          </div>
        </div>
        <div role="tabpanel" class="tab-pane fade " id="overall_dlfc_tab29">
          <div id="dlfc_overall29" style="min-width: 800px; height: 500px; max-width: 800px; margin: 0 auto">
            <span>Loading bar plot...</span>
          </div>
        </div>
      
      
      </div> --><!-- end of inner tab content -->
           
                  
     
     </div>
   
    {%for group in report.genegroup_set.all%}
     
     <div role="tabpanel" class="tab-pane fade " id="{{group.get_slug}}_dlfc_tab">     
             
       <div id="dlfc_{{group.get_slug}}" style="min-width: 800px; height: 500px; max-width: 800px; margin: 0 auto">
         <span>Loading bar plot...</span>
       </div>     
     
     </div> <!-- end of tabpanel div -->
    {%endfor%}
   
   </div> <!--  end of tab-content div -->
<br/>
<hr style="width:70%" />

<h4>Table <span class='tbl_granular'></span>Top diseases  </h4>
       <table id="tbl_granular" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr style="min-width: 900px; width: 100%;">
           <th>Perturbation</th>
           <th>Disease	</th>
           <th>Score</th>
        
          </tr>
        </thead>  
      </table>
      <br/><br/>

</div>

<script>
$(function(){
	
    $("#btn_show_dl").click(function(){
		
		$(this).hide();
		$("#hide_dl").slideDown('slow');
		
		
		//overall_dlfc_tab
		$.ajax({
	        type: 'GET',
	        dataType: 'json',
	        url: '/report-portal/report-deeplearningfarmcompjson/',
	        data:{
	   	         report_id: {{report.id}},
	   	         type: 'fc'
	           },
	        success: function(res) {	        
	        	arResponce = JSON.parse(res);
	        	
	        	var ulHTML = '<ul class="nav nav-tabs" role="tablist">';
	        	var divHTML = '<div class="tab-content">';
	        	
	        	for(let i in arResponce){
	        		let compound = arResponce[i];
	        		if(i==0)   ulHTML+='<li role="presentation" class="active">';
	        		else        ulHTML+='<li role="presentation" >';
	        		
	        		ulHTML+= '<a href="#overall_dlfc_tab'+compound+'"  role="tab" data-toggle="tab" id="dlfc_boot_overall'+compound+'">';
	        		ulHTML+= compound;
	        		ulHTML+= '</a></li>';
	        		
	        		
	        		
	        		if(i==0) divHTML+= '<div role="tabpanel" class="tab-pane fade in active" id="overall_dlfc_tab'+compound+'">';
	        		else     divHTML+= '<div role="tabpanel" class="tab-pane fade " id="overall_dlfc_tab'+compound+'">';
	        		divHTML+= '  <div id="dlfc_overall'+compound+'" style="min-width: 800px; height: 500px; max-width: 800px; margin: 0 auto">';
	        		divHTML+= '    <span>Loading bar plot...</span>';
	        		divHTML+= '  </div></div>';
	        	    }
	        	ulHTML+='</ul>';
	        	divHTML+='</div>';
	        	
	        	$("#overall_dlfc_tab").html(ulHTML+divHTML).promise().done(function(){
	        		for(let i in arResponce){
	        			let compound = arResponce[i];
	        		    drawDLColumn({{report.id}}, 'dlfc_overall'+compound,
	        	              'overall_'+compound, 'fc');
	        		}
	        		
	        	});
	        	
	        	}
	           
		});
		
		
		
		
		var tbl_granular = $("#tbl_granular").DataTable( {
			"paging":   true,
		    "iDisplayLength": 20,
		    "ordering": true,
		    "order": [],
		    "info":     false,
		    "dom": 'Bfrtip',
		    "scrollX": false,
		    "bAutoWidth": false,
		    "buttons": [
		               {extend: 'csv', title: 'tbl_granular'},
		               {extend: 'print', title: 'tbl_granular'}
		               
		          ],
		    
		    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
		    	     'type': 'GET',
		    	     'data':{
		    	    	     reportID: {{report.id}},
		    	    	     file_name: '{{report.slug}}/granular_ds.txt'
		    	            },        	      
		    
		},"deferRender": true });
	});
    
  //Group tabs
    {%for group in report.genegroup_set.all%}              
        
        
            	
        	
        		drawDLColumn({{report.id}}, 'dlfc_{{group.get_slug}}',
      		                 '{{group.get_slug}}', 'fc');
    		       
          
    {%endfor%}
	
	
	
});

</script>


<!-- ---------------------------------Drug Scoring--------------------------------------------- -->
{% if report.count_ds_groups > 0 %} 
 <hr/> 
<h2 id='ds_analysis'>Drug Scoring</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_ds'>Show Data</button>
</center>

<div id='hide_ds' style='display:none;'> <!-- Div to hide and load on button click -->

   <!-- DS Nav tabs -->      
   <ul class="nav nav-tabs" role="tablist">
     {%for group in report.drugscoregroup_set.all%}
       <li role="presentation" {%if forloop.counter == 1 %}class="active"{%endif%}>
         <a href="#{{group.get_slug}}_ds_tab" aria-controls="home1" role="tab" data-toggle="tab" id='ds_boot_{{group.get_slug}}'>
           {{group.name}}
         </a>
       </li>
     {%endfor%}
   </ul>
         
   <div class="tab-content">
    {%for group in report.drugscoregroup_set.all%}
     {%if forloop.counter == 1 %}
     <div role="tabpanel" class="tab-pane fade in active" id="{{group.get_slug}}_ds_tab">
     {%else%}
     <div role="tabpanel" class="tab-pane fade " id="{{group.get_slug}}_ds_tab" style='overflow-x: auto;'>
     {%endif%}
      
        
       <h4>Table <span class='tbl_num'></span>Individual perturbation scoring for {{group.name}} </h4>
       <table id="tbl-ds-{{group.get_slug}}" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr style="min-width: 900px; width: 100%;">
           <th>Perturbation</th>
           <th>Perturbation_id</th>
           <th>Time</th>
           <th>Concentration</th>
           <th>Cell line</th>
           <!--  <th>Condition</th>-->
           <th>p-val</th>
           <th>adj. p-val</th>
           <th>Effect</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
    {%endfor%}
   
   </div> <!--  end of tab-content div -->
   
   <h3>Top-10 small molecule hits</h3>
   <!-- DS Boxplot Nav tabs -->      
   <ul class="nav nav-tabs" role="tablist">
     {%for group in report.drugscoregroup_set.all%}
       <li role="presentation" {%if forloop.counter == 1 %}class="active"{%endif%}>
         <a href="#{{group.get_slug}}_dsbarmolecule_tab" aria-controls="home1" role="tab" data-toggle="tab" id='dsbarmolecule_boot_{{group.get_slug}}'>
           {{group.name}}
         </a>
       </li>
     {%endfor%}
   </ul>
         
   <div class="tab-content">
   
     {%for group in report.drugscoregroup_set.all%} 
     
     {%if forloop.counter == 1 %}
     <div role="tabpanel" class="tab-pane fade in active" id="{{group.get_slug}}_dsbarmolecule_tab">
     {%else%}
     <div role="tabpanel" class="tab-pane fade " id="{{group.get_slug}}_dsbarmolecule_tab">
     {%endif%}
     <h4>Top-10 hits for {{group.name}} </h4>
     <div id="{{group.get_slug}}_dsbarmolecule_plot" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">
     <span>Loading Bar Plot...</span>
     </div>
   
     </div>
    {%endfor%}
   
   </div> <!--  end of tab panel -->
   
<!--  DS GENE -->
<h3>Top-10 gene knockdowns</h3>
   <!-- DS Boxplot Nav tabs -->      
   <ul class="nav nav-tabs" role="tablist">
     {%for group in report.drugscoregroup_set.all%}
       <li role="presentation" {%if forloop.counter == 1 %}class="active"{%endif%}>
         <a href="#{{group.get_slug}}_dsbargene_tab" aria-controls="home1" role="tab" data-toggle="tab" id='dsbargene_boot_{{group.get_slug}}'>
           {{group.name}}
         </a>
       </li>
     {%endfor%}
   </ul>
         
   <div class="tab-content">
   
     {%for group in report.drugscoregroup_set.all%} 
     
     {%if forloop.counter == 1 %}
     <div role="tabpanel" class="tab-pane fade in active" id="{{group.get_slug}}_dsbargene_tab">
     {%else%}
     <div role="tabpanel" class="tab-pane fade " id="{{group.get_slug}}_dsbargene_tab">
     {%endif%}
     <h4>Top-10 hits for {{group.name}} </h4>
     <div id="{{group.get_slug}}_dsbargene_plot" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">
     <span>Loading Bar Plot...</span>
     </div>
   
     </div>
    {%endfor%}
   
   </div> <!--  end of tab panel -->
   

</div> <!--  end of hide div -->
{%endif%}
<!-- End of DS -->




<!-- ---------------------------------BioMAP inference--------------------------------------------- -->

 <hr/> 
<h2 id='biomap'>BioMAP inference</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_biomap'>Show Data</button>
</center>

<div id='hide_biomap' style='display:none;'> <!-- Div to hide and load on button click -->

<div class="row">
  <div class="col-lg-12 text-center">
    <img src='/media/report-portal/gsk_prj4_t5/g_distribution/gsk_aring.png'  />    
    <h4>Distribution of ARING property</h4>
    <hr style="width:50%"/>
    
    <img src='/media/report-portal/gsk_prj4_t5/g_distribution/gsk_clogp.png'  />    
    <h4>Distribution of CLOGP property</h4>
    <hr style="width:50%"/>
    
    <img src='/media/report-portal/gsk_prj4_t5/g_distribution/gsk_mw.png'  />    
    <h4>Distribution of MW property</h4>
    <hr style="width:50%"/>
    
    <img src='/media/report-portal/gsk_prj4_t5/g_distribution/gsk_tpsa.png'  />    
    <h4>Distribution of TPSA property</h4>
    <hr style="width:50%"/> 
  </div>
</div>


<span id="docs-internal-guid-cf75ab3e-f99b-284f-fc64-f1995dd70398">
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">In this section we present the results of the experiment aimed at inferring BioMAP cytotox and diversity biomarkers independently from L1000 dataset using drug perturbation response data. We utilised 280 different model architectures that vary by feature selection method (S_523, f_score, fisher_score, reliefF), number of input features (200, 300, 500, 978) and algorithm (ElasticNet, Linear regression, KNN, RF, ExtraTrees, SVR, MLP). As an input for a models we utilised robust z-scores for landmark genes from L1000 dataset and as an output the ratio ‘Log-Ratio/Confidence interval’ for BioMAP cytotox and diversity data. Robust z-scores were averaged for perturbation replicates. Input L1000 samples and output BioMAP samples were matched by drug and concentration. Also for L1000 dataset we experimented with addition of provided molecular properties (# of aromatic rings, clogp, molecular weight, TPSA) as extra features.</span>
</p><br/>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Since the number of common with L1000 drugs profiled in cytotox and diversity BioMAP sets is different we used two separate output sets: cytotox and diversity. We also used 4 different training set designs to stratify the L1000 set: </span>
</p><br/>
<ol style="margin-top:0pt;margin-bottom:0pt;">
	<li dir="ltr" style="list-style-type: decimal; font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline;"><span style="font-size: 11pt; background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Data set separated by time point (6 and 24 hours) with robust z-scores for landmark genes (Landmarks_by_time).</span></li>
	<li dir="ltr" style="list-style-type: decimal; font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline;"><span style="font-size: 11pt; background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Data set separated by time point (6 and 24 hours) with robust z-scores for landmark genes and values for 4 additional molecular properties (Landmarks_and_molprop_by_time).</span></li>
	<li dir="ltr" style="list-style-type: decimal; font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline;"><span style="font-size: 11pt; background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Aggregated data set with robust z-scores for landmark genes (Landmarks).</span></li>
	<li dir="ltr" style="list-style-type: decimal; font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline;"><span style="font-size: 11pt; background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Aggregated data set with robust z-scores for landmark genes and values for 4 additional molecular properties (Landmarks_and_molprop).</span></li>
</ol><br/>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">For each of the training sets feature selection methods were applied to L1000 landmark genes followed by building ML and DL models to predict the BioMAP feature. Each BioMAP biomarker was inferred independently. For each ML and DL model optimization of its hyperparameters was applied using grid search. Each model underwent 10-fold cross-validation and the resulting performance was averaged across folds. </span>
</p><br/>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">In this report we present the tables with performance (mean R-squared across all biomarkers in either diversity or cytotox sets) of the models for each training set design. After each performance table we also put in separate tabs the R-squared metrics for each biomarker of top model for cytotox and top model for diversity inference. Due to curse of dimensionality and relatively small number of matching samples DNNs lost to a more conventional methods like ElasticNet and tree-based methods.</span>
</p><br/>
<div><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"><br></span>
</div></span>


<!-- BIO Nav tabs  -->     
   <ul class="nav nav-tabs" role="tablist">
     
       <li role="presentation" class="active">
         <a href="#lbt_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_lbt'>
           Landmarks_by_time
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmc1_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmc1'>
           Top model cytotox Landmarks_by_time
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmd1_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmd1'>
           Top model diversity Landmarks_by_time
         </a>
       </li>
       <li role="presentation" >
         <a href="#lambt_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_lambt'>
           Landmarks_and_molprop_by_time
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmc2_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmc2'>
           Top model cytotox Landmarks_and_molprop_by_time
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmd2_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmd2'>
           Top model diversity Landmarks_and_molprop_by_time
         </a>
       </li>
       <li role="presentation" >
         <a href="#land_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_land'>
           Landmarks
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmc3_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmc3'>
           Top model cytotox Landmarks
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmd3_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmd3'>
           Top model diversity Landmarks
         </a>
       </li>
       <li role="presentation" >
         <a href="#lam_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_lam'>
           Landmarks_and_molprop
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmc4_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmc4'>
           Top model cytotox Landmarks_and_molprop
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmd4_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmd4'>
           Top model diversity Landmarks_and_molprop
         </a>
       </li>
    
   </ul>
         
   <div class="tab-content">
    
     <div role="tabpanel" class="tab-pane fade in active" id="lbt_tab">  
             
       <h4>Table <span class='tbl_num'></span>Landmarks_by_time </h4>
       <table id="tbl-bio-lbt" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>count</th>
           <th>key</th>
           <th>mean_r2</th>
           <th>method</th>
           <th>model</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     <div role="tabpanel" class="tab-pane fade " id="tmc1_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model cytotox Landmarks_by_time</h4>
       <table id="tbl-bio-tmc1" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
       
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     <div role="tabpanel" class="tab-pane fade " id="tmd1_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model diversity Landmarks_by_time</h4>
       <table id="tbl-bio-tmd1" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="lambt_tab">  
             
       <h4>Table <span class='tbl_num'></span>Landmarks_and_molprop_by_time </h4>
       <table id="tbl-bio-lambt" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>count</th>
           <th>key</th>
           <th>mean_r2</th>
           <th>method</th>
           <th>model</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="tmc2_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model cytotox Landmarks_and_molprop_by_time</h4>
       <table id="tbl-bio-tmc2" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
       
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     <div role="tabpanel" class="tab-pane fade " id="tmd2_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model diversity Landmarks_and_molprop_by_time</h4>
       <table id="tbl-bio-tmd2" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="land_tab">  
             
       <h4>Table <span class='tbl_num'></span>Landmarks </h4>
       <table id="tbl-bio-land" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>count</th>
           <th>key</th>
           <th>mean_r2</th>
           <th>method</th>
           <th>model</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="tmc3_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model cytotox Landmarks</h4>
       <table id="tbl-bio-tmc3" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
       
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     <div role="tabpanel" class="tab-pane fade " id="tmd3_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model diversity Landmarks</h4>
       <table id="tbl-bio-tmd3" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="lam_tab">  
             
       <h4>Table <span class='tbl_num'></span>Landmarks_and_molprop </h4>
       <table id="tbl-bio-lam" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>count</th>
           <th>key</th>
           <th>mean_r2</th>
           <th>method</th>
           <th>model</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="tmc4_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model cytotox Landmarks_and_molprop</h4>
       <table id="tbl-bio-tmc4" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
       
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     <div role="tabpanel" class="tab-pane fade " id="tmd4_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model diversity Landmarks_and_molprop</h4>
       <table id="tbl-bio-tmd4" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
 <div style="overflow-x:auto" >     
     <h4 style="padding: 10px 0;">Table <span class='tbl_num'></span>Permutation feature importance results for Cytotox data</h4>
       <table id="tbl-indef_cytotox" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
    </table>
      <br/><br/>
      
  <h4 style="padding: 10px 0;">Table <span class='tbl_num'></span>Permutation feature importance results for Diversity data</h4>
       <table id="tbl-indfer_diversity" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
    </table>
      <br/><br/>
</div>    
   
   </div> <!--  end of tab-content div -->

</div>

<script>

$(function() {
	
	
	$("#btn_show_biomap").click(function(){
		
		$(this).hide();
		$("#hide_biomap").slideDown('slow');
	});
	
	
	var Landmarks_by_time = $("#tbl-bio-lbt").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl-bio-lbt'},
	               {extend: 'print', title: 'tbl-bio-lbt'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/timewise_report_land.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var Landmarks_and_molprop_by_time = $("#tbl-bio-lambt").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'Landmarks_and_molprop_by_time'},
	               {extend: 'print', title: 'Landmarks_and_molprop_by_time'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/timewise_report_land_mol.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var Landmarks = $("#tbl-bio-land").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'Landmarks'},
	               {extend: 'print', title: 'Landmarks'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/report_land.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var Landmarks_and_molprop = $("#tbl-bio-lam").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'Landmarks_and_molprop'},
	               {extend: 'print', title: 'Landmarks_and_molprop'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/report_land_mol.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	
	var tbl_bio_tmc1 = $("#tbl-bio-tmc1").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmc1'},
	               {extend: 'print', title: 'tbl_bio_tmc1'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/Landmarks_by_time_cytotoxtop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var tbl_bio_tmd1 = $("#tbl-bio-tmd1").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmd1'},
	               {extend: 'print', title: 'tbl_bio_tmd1'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/Landmarks_by_time_diversitytop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	////////
	var tbl_bio_tmc2 = $("#tbl-bio-tmc2").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmc2'},
	               {extend: 'print', title: 'tbl_bio_tmc2'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/LandmarksMol_by_time_cytotoxtop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var tbl_bio_tmd2 = $("#tbl-bio-tmd2").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmd2'},
	               {extend: 'print', title: 'tbl_bio_tmd2'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/LandmarksMol_by_time_diversitytop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	/////////////
	
	////////
	var tbl_bio_tmc3 = $("#tbl-bio-tmc3").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmc3'},
	               {extend: 'print', title: 'tbl_bio_tmc3'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/Landmarks_cytotoxtop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var tbl_bio_tmd3 = $("#tbl-bio-tmd3").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmd3'},
	               {extend: 'print', title: 'tbl_bio_tmd3'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/Landmarks_diversitytop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	/////////////
	
	////////
	var tbl_bio_tmc4 = $("#tbl-bio-tmc4").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmc4'},
	               {extend: 'print', title: 'tbl_bio_tmc4'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/LandmarksMol_cytotoxtop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var tbl_bio_tmd4 = $("#tbl-bio-tmd4").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmd4'},
	               {extend: 'print', title: 'tbl_bio_tmd4'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/LandmarksMol_diversitytop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	/////////////
	
	$.ajax({
        type: 'GET',
        dataType: 'json',
        url: '/report-portal/report-ajaxtablecolumns/',
        data:{
   	         reportID: {{report.id}},
   	         file_name: '../gsk_prj4_t5/cytotox_pfi.csv'
           },
        success: function(d) {	        	
        	let thead = '<tr>';	        	
        	for (let col of d.columns) {	        		
        		thead+='<th>'+col+'</th>';
        	}
        	thead+='</tr>';
        	
        	$("#tbl-indef_cytotox").find("thead").html(thead);
        	
        	var tbl_sim = $("#tbl-indef_cytotox").DataTable( {
        		"paging":   true,
        	    "iDisplayLength": 20,
        	    "ordering": true,
        	    "order": [],
        	    "info":     false,
        	    "dom": 'Bfrtip',
        	    "scrollX": false,
        	    "bAutoWidth": false,
        	    "buttons": [
        	               {extend: 'csv', title: 'tbl-cytotox'},
        	               {extend: 'print', title: 'tbl-cytotox'}
        	               
        	          ],
        	    
        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
        	    	     'type': 'GET',
        	    	     'data':{
        	    	    	     reportID: {{report.id}},
        	    	    	     file_name: '../gsk_prj4_t5/cytotox_pfi.csv'
        	    	            },        	      
        	    
        	},"deferRender": true });
        }
    });
	
	///////////////////////////
	$.ajax({
        type: 'GET',
        dataType: 'json',
        url: '/report-portal/report-ajaxtablecolumns/',
        data:{
   	         reportID: {{report.id}},
   	         file_name: '../gsk_prj4_t5/diversity_pfi.csv'
           },
        success: function(d) {	        	
        	let thead = '<tr>';	        	
        	for (let col of d.columns) {	        		
        		thead+='<th>'+col+'</th>';
        	}
        	thead+='</tr>';
        	
        	$("#tbl-indfer_diversity").find("thead").html(thead);
        	
        	var tbl_sim = $("#tbl-indfer_diversity").DataTable( {
        		"paging":   true,
        	    "iDisplayLength": 20,
        	    "ordering": true,
        	    "order": [],
        	    "info":     false,
        	    "dom": 'Bfrtip',
        	    "scrollX": false,
        	    "bAutoWidth": false,
        	    "buttons": [
        	               {extend: 'csv', title: 'tbl-diversity'},
        	               {extend: 'print', title: 'tbl-diversity'}
        	               
        	          ],
        	    
        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
        	    	     'type': 'GET',
        	    	     'data':{
        	    	    	     reportID: {{report.id}},
        	    	    	     file_name: '../gsk_prj4_t5/diversity_pfi.csv'
        	    	            },        	      
        	    
        	},"deferRender": true });
        }
    });
	
	
});


</script>

<!-- ---------- end of BioMAP inference -->

<!-- ---------------------------------L1000-BioMAP correlation--------------------------------------------- -->

 <hr/> 
<h2 id='biomap_l1000'>L1000-BioMAP correlation</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_biomapcor'>Show Data</button>
</center>

<div id='hide_biomapcor' style='display:none;'> <!-- Div to hide and load on button click -->
<br/>
<p>
For each system (3C and SAg) and time point in L1000 dataset we matched corresponding samples to BioMAP cytotox 
and diversity dataset samples by drug and system. Then we calculated Pearson correlation coefficients for each 
L1000 gene- BioMAP marker pair. Here in the table below for each BioMAP marker we selected Top-100 most correlated 
(by absolute value) L1000 genes and the resulting set of Top-100 lists was aggregated.
</p>
<br/>

<!-- BIO Nav tabs  -->     
   <ul class="nav nav-tabs" role="tablist">
     
       <li role="presentation" class="active">
         <a href="#3c_6h_cytotox_tab"  role="tab" data-toggle="tab" >
           3c.6h.cytotox
         </a>
       </li>
       <li role="presentation" >
         <a href="#3c_6h_diversity_tab"  role="tab" data-toggle="tab" >
           3c.6h.diversity
         </a>
       </li>
       <li role="presentation" >
         <a href="#3c_24h_cytotox_tab" role="tab" data-toggle="tab" >
           3c.24h.cytotox
         </a>
       </li>
       <li role="presentation" >
         <a href="#3c_24h_diversity_tab"  role="tab" data-toggle="tab" >
           3c.24h.diversity
         </a>
       </li>
       <li role="presentation" >
         <a href="#sag_6h_cytotox_tab" role="tab" data-toggle="tab">
           sag.6h.cytotox
         </a>
       </li>
       <li role="presentation" >
         <a href="#sag_6h_diversity_tab"  role="tab" data-toggle="tab" >
           sag.6h.diversity
         </a>
       </li>
       <li role="presentation" >
         <a href="#sag_24h_cytotox_tab"  role="tab" data-toggle="tab" >
           sag.24h.cytotox
         </a>
       </li>
       <li role="presentation" >
         <a href="#sag_24h_diversity_tab"  role="tab" data-toggle="tab" >
           sag.24h.diversity
         </a>
       </li>
   </ul>
   
   <div class="tab-content" style="overflow-x: auto">
    
     <div role="tabpanel" class="tab-pane fade in active" id="3c_6h_cytotox_tab">  
             
       <h4 style="padding: 10px 0;">Table <span class='tbl_num'></span>3c_6h_cytotox </h4>
       <table id="tbl-3c_6h_cytotox" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
    </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade in " id="3c_6h_diversity_tab">  
            
       <h4 style="padding: 10px 0;">Table <span class='tbl_num'></span>TOP-100 3c_6h_diversity. <a href="/media/report-portal/gsk_prj4_t5/corr/corr.3c.6h.diversity.csv">Download full file</a></h4>
       <table id="tbl-3c_6h_diversity" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
    </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <!-- ------------------------------------------------------------------------- -->
     
     <div role="tabpanel" class="tab-pane fade" id="3c_24h_cytotox_tab">  
             
       <h4 style="padding: 10px 0;">Table <span class='tbl_num'></span>3c_24h_cytotox </h4>
       <table id="tbl-3c_24h_cytotox" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
    </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade in " id="3c_24h_diversity_tab">  
            
       <h4 style="padding: 10px 0;">Table <span class='tbl_num'></span>TOP-100 3c_24h_diversity. <a href="/media/report-portal/gsk_prj4_t5/corr/corr.3c.24h.diversity.csv">Download full file</a></h4>
       <table id="tbl-3c_24h_diversity" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
    </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <!-- ------------------------------------------------------------------------- -->
     
     <div role="tabpanel" class="tab-pane fade" id="sag_6h_cytotox_tab">  
             
       <h4 style="padding: 10px 0;">Table <span class='tbl_num'></span>sag_6h_cytotox </h4>
       <table id="tbl-sag_6h_cytotox" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
    </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade in " id="sag_6h_diversity_tab">  
            
       <h4 style="padding: 10px 0;">Table <span class='tbl_num'></span>TOP-100 sag_6h_diversity. <a href="/media/report-portal/gsk_prj4_t5/corr/corr.sag.6h.diversity.csv">Download full file</a></h4>
       <table id="tbl-sag_6h_diversity" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
    </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <!-- ------------------------------------------------------------------------- -->
     
     <div role="tabpanel" class="tab-pane fade" id="sag_24h_cytotox_tab">  
             
       <h4 style="padding: 10px 0;">Table <span class='tbl_num'></span>sag_24h_cytotox </h4>
       <table id="tbl-sag_24h_cytotox" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
    </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade in " id="sag_24h_diversity_tab">  
            
       <h4 style="padding: 10px 0;">Table <span class='tbl_num'></span>TOP-100 sag_24h_diversity. <a href="/media/report-portal/gsk_prj4_t5/corr/corr.sag.24h.diversity.csv">Download full file</a></h4>
       <table id="tbl-sag_24h_diversity" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
    </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
   </div><!-- end of tab-content div -->
   
   

</div> <!-- end of hide div -->

<script>
$(function(){
	
$("#btn_show_biomapcor").click(function(){
		
		$(this).hide();
		$("#hide_biomapcor").slideDown('slow');
		
		
		$.ajax({
	        type: 'GET',
	        dataType: 'json',
	        url: '/report-portal/report-ajaxtablecolumns/',
	        data:{
	   	         reportID: {{report.id}},
	   	         file_name: '../gsk_prj4_t5/corr/corr.3c.6h.cytotox_tr.csv'
	           },
	        success: function(d) {	        	
	        	let thead = '<tr>';	        	
	        	for (let col of d.columns) {	        		
	        		thead+='<th>'+col+'</th>';
	        	}
	        	thead+='</tr>';
	        	
	        	$("#tbl-3c_6h_cytotox").find("thead").html(thead);
	        	
	        	var tbl_sim = $("#tbl-3c_6h_cytotox").DataTable( {
	        		"paging":   true,
	        	    "iDisplayLength": 20,
	        	    "ordering": true,
	        	    "order": [],
	        	    "info":     false,
	        	    "dom": 'Bfrtip',
	        	    "scrollX": false,
	        	    "bAutoWidth": false,
	        	    "buttons": [
	        	               {extend: 'csv', title: 'tbl-3c_6h_cytotox'},
	        	               {extend: 'print', title: 'tbl-3c_6h_cytotox'}
	        	               
	        	          ],
	        	    
	        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
	        	    	     'type': 'GET',
	        	    	     'data':{
	        	    	    	     reportID: {{report.id}},
	        	    	    	     file_name: '../gsk_prj4_t5/corr/corr.3c.6h.cytotox_tr.csv'
	        	    	            },        	      
	        	    
	        	},"deferRender": true });
	        }
	    });
		
		/////////////////////////////////////
		$.ajax({
	        type: 'GET',
	        dataType: 'json',
	        url: '/report-portal/report-ajaxtablecolumns/',
	        data:{
	   	         reportID: {{report.id}},
	   	         file_name: '../gsk_prj4_t5/corr/corr.3c.6h.diversity_sorted.csv'
	           },
	        success: function(d) {	        	
	        	let thead = '<tr>';	        	
	        	for (let col of d.columns) {	        		
	        		thead+='<th>'+col+'</th>';
	        	}
	        	thead+='</tr>';
	        	
	        	$("#tbl-3c_6h_diversity").find("thead").html(thead);
	        	
	        	var tbl_sim = $("#tbl-3c_6h_diversity").DataTable( {
	        		"paging":   true,
	        	    "iDisplayLength": 20,
	        	    "ordering": true,
	        	    "order": [],
	        	    "info":     false,
	        	    "dom": 'Bfrtip',
	        	    "scrollX": false,
	        	    "bAutoWidth": false,
	        	    "buttons": [
	        	               {extend: 'csv', title: 'tbl-3c_6h_diversity'},
	        	               {extend: 'print', title: 'tbl-3c_6h_diversity'}
	        	               
	        	          ],
	        	    
	        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
	        	    	     'type': 'GET',
	        	    	     'data':{
	        	    	    	     reportID: {{report.id}},
	        	    	    	     file_name: '../gsk_prj4_t5/corr/corr.3c.6h.diversity_sorted.csv'
	        	    	            },        	      
	        	    
	        	},"deferRender": true });
	        }
	    });
		///////////////////////////////////////////  3c 24h
		
		$.ajax({
	        type: 'GET',
	        dataType: 'json',
	        url: '/report-portal/report-ajaxtablecolumns/',
	        data:{
	   	         reportID: {{report.id}},
	   	         file_name: '../gsk_prj4_t5/corr/corr.3c.24h.cytotox_tr.csv'
	           },
	        success: function(d) {	        	
	        	let thead = '<tr>';	        	
	        	for (let col of d.columns) {	        		
	        		thead+='<th>'+col+'</th>';
	        	}
	        	thead+='</tr>';
	        	
	        	$("#tbl-3c_24h_cytotox").find("thead").html(thead);
	        	
	        	var tbl_sim = $("#tbl-3c_24h_cytotox").DataTable( {
	        		"paging":   true,
	        	    "iDisplayLength": 20,
	        	    "ordering": true,
	        	    "order": [],
	        	    "info":     false,
	        	    "dom": 'Bfrtip',
	        	    "scrollX": false,
	        	    "bAutoWidth": false,
	        	    "buttons": [
	        	               {extend: 'csv', title: 'tbl-3c_24h_cytotox'},
	        	               {extend: 'print', title: 'tbl-3c_24h_cytotox'}
	        	               
	        	          ],
	        	    
	        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
	        	    	     'type': 'GET',
	        	    	     'data':{
	        	    	    	     reportID: {{report.id}},
	        	    	    	     file_name: '../gsk_prj4_t5/corr/corr.3c.24h.cytotox_tr.csv'
	        	    	            },        	      
	        	    
	        	},"deferRender": true });
	        }
	    });
		
		/////////////////////////////////////
		$.ajax({
	        type: 'GET',
	        dataType: 'json',
	        url: '/report-portal/report-ajaxtablecolumns/',
	        data:{
	   	         reportID: {{report.id}},
	   	         file_name: '../gsk_prj4_t5/corr/corr.3c.24h.diversity_sorted.csv'
	           },
	        success: function(d) {	        	
	        	let thead = '<tr>';	        	
	        	for (let col of d.columns) {	        		
	        		thead+='<th>'+col+'</th>';
	        	}
	        	thead+='</tr>';
	        	
	        	$("#tbl-3c_24h_diversity").find("thead").html(thead);
	        	
	        	var tbl_sim = $("#tbl-3c_24h_diversity").DataTable( {
	        		"paging":   true,
	        	    "iDisplayLength": 20,
	        	    "ordering": true,
	        	    "order": [],
	        	    "info":     false,
	        	    "dom": 'Bfrtip',
	        	    "scrollX": false,
	        	    "bAutoWidth": false,
	        	    "buttons": [
	        	               {extend: 'csv', title: 'tbl-3c_24h_diversity'},
	        	               {extend: 'print', title: 'tbl-3c_24h_diversity'}
	        	               
	        	          ],
	        	    
	        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
	        	    	     'type': 'GET',
	        	    	     'data':{
	        	    	    	     reportID: {{report.id}},
	        	    	    	     file_name: '../gsk_prj4_t5/corr/corr.3c.24h.diversity_sorted.csv'
	        	    	            },        	      
	        	    
	        	},"deferRender": true });
	        }
	    });
		///////////////////////////////////////////
		///////////////////////////////////////////  sag 6h
		
		$.ajax({
	        type: 'GET',
	        dataType: 'json',
	        url: '/report-portal/report-ajaxtablecolumns/',
	        data:{
	   	         reportID: {{report.id}},
	   	         file_name: '../gsk_prj4_t5/corr/corr.sag.6h.cytotox_tr.csv'
	           },
	        success: function(d) {	        	
	        	let thead = '<tr>';	        	
	        	for (let col of d.columns) {	        		
	        		thead+='<th>'+col+'</th>';
	        	}
	        	thead+='</tr>';
	        	
	        	$("#tbl-sag_6h_cytotox").find("thead").html(thead);
	        	
	        	var tbl_sim = $("#tbl-sag_6h_cytotox").DataTable( {
	        		"paging":   true,
	        	    "iDisplayLength": 20,
	        	    "ordering": true,
	        	    "order": [],
	        	    "info":     false,
	        	    "dom": 'Bfrtip',
	        	    "scrollX": false,
	        	    "bAutoWidth": false,
	        	    "buttons": [
	        	               {extend: 'csv', title: 'tbl-sag_6h_cytotox'},
	        	               {extend: 'print', title: 'tbl-sag_6h_cytotox'}
	        	               
	        	          ],
	        	    
	        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
	        	    	     'type': 'GET',
	        	    	     'data':{
	        	    	    	     reportID: {{report.id}},
	        	    	    	     file_name: '../gsk_prj4_t5/corr/corr.sag.6h.cytotox_tr.csv'
	        	    	            },        	      
	        	    
	        	},"deferRender": true });
	        }
	    });
		
		/////////////////////////////////////
		$.ajax({
	        type: 'GET',
	        dataType: 'json',
	        url: '/report-portal/report-ajaxtablecolumns/',
	        data:{
	   	         reportID: {{report.id}},
	   	         file_name: '../gsk_prj4_t5/corr/corr.sag.6h.diversity_sorted.csv'
	           },
	        success: function(d) {	        	
	        	let thead = '<tr>';	        	
	        	for (let col of d.columns) {	        		
	        		thead+='<th>'+col+'</th>';
	        	}
	        	thead+='</tr>';
	        	
	        	$("#tbl-sag_6h_diversity").find("thead").html(thead);
	        	
	        	var tbl_sim = $("#tbl-sag_6h_diversity").DataTable( {
	        		"paging":   true,
	        	    "iDisplayLength": 20,
	        	    "ordering": true,
	        	    "order": [],
	        	    "info":     false,
	        	    "dom": 'Bfrtip',
	        	    "scrollX": false,
	        	    "bAutoWidth": false,
	        	    "buttons": [
	        	               {extend: 'csv', title: 'tbl-sag_6h_diversity'},
	        	               {extend: 'print', title: 'tbl-sag_6h_diversity'}
	        	               
	        	          ],
	        	    
	        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
	        	    	     'type': 'GET',
	        	    	     'data':{
	        	    	    	     reportID: {{report.id}},
	        	    	    	     file_name: '../gsk_prj4_t5/corr/corr.sag.6h.diversity_sorted.csv'
	        	    	            },        	      
	        	    
	        	},"deferRender": true });
	        }
	    });
		///////////////////////////////////////////
		///////////////////////////////////////////  sag 24h
		
		$.ajax({
	        type: 'GET',
	        dataType: 'json',
	        url: '/report-portal/report-ajaxtablecolumns/',
	        data:{
	   	         reportID: {{report.id}},
	   	         file_name: '../gsk_prj4_t5/corr/corr.sag.24h.cytotox_tr.csv'
	           },
	        success: function(d) {	        	
	        	let thead = '<tr>';	        	
	        	for (let col of d.columns) {	        		
	        		thead+='<th>'+col+'</th>';
	        	}
	        	thead+='</tr>';
	        	
	        	$("#tbl-sag_24h_cytotox").find("thead").html(thead);
	        	
	        	var tbl_sim = $("#tbl-sag_24h_cytotox").DataTable( {
	        		"paging":   true,
	        	    "iDisplayLength": 20,
	        	    "ordering": true,
	        	    "order": [],
	        	    "info":     false,
	        	    "dom": 'Bfrtip',
	        	    "scrollX": false,
	        	    "bAutoWidth": false,
	        	    "buttons": [
	        	               {extend: 'csv', title: 'tbl-sag_24h_cytotox'},
	        	               {extend: 'print', title: 'tbl-sag_24h_cytotox'}
	        	               
	        	          ],
	        	    
	        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
	        	    	     'type': 'GET',
	        	    	     'data':{
	        	    	    	     reportID: {{report.id}},
	        	    	    	     file_name: '../gsk_prj4_t5/corr/corr.sag.24h.cytotox_tr.csv'
	        	    	            },        	      
	        	    
	        	},"deferRender": true });
	        }
	    });
		
		/////////////////////////////////////
		$.ajax({
	        type: 'GET',
	        dataType: 'json',
	        url: '/report-portal/report-ajaxtablecolumns/',
	        data:{
	   	         reportID: {{report.id}},
	   	         file_name: '../gsk_prj4_t5/corr/corr.sag.24h.diversity_sorted.csv'
	           },
	        success: function(d) {	        	
	        	let thead = '<tr>';	        	
	        	for (let col of d.columns) {	        		
	        		thead+='<th>'+col+'</th>';
	        	}
	        	thead+='</tr>';
	        	
	        	$("#tbl-sag_24h_diversity").find("thead").html(thead);
	        	
	        	var tbl_sim = $("#tbl-sag_24h_diversity").DataTable( {
	        		"paging":   true,
	        	    "iDisplayLength": 20,
	        	    "ordering": true,
	        	    "order": [],
	        	    "info":     false,
	        	    "dom": 'Bfrtip',
	        	    "scrollX": false,
	        	    "bAutoWidth": false,
	        	    "buttons": [
	        	               {extend: 'csv', title: 'tbl-sag_24h_diversity'},
	        	               {extend: 'print', title: 'tbl-sag_24h_diversity'}
	        	               
	        	          ],
	        	    
	        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
	        	    	     'type': 'GET',
	        	    	     'data':{
	        	    	    	     reportID: {{report.id}},
	        	    	    	     file_name: '../gsk_prj4_t5/corr/corr.sag.24h.diversity_sorted.csv'
	        	    	            },        	      
	        	    
	        	},"deferRender": true });
	        }
	    });
		///////////////////////////////////////////
		
		
	});
	
});

</script>

<!-- ---------- end of L1000-BioMAP correlation -->

<!-- ---------------------------------Hit qualification--------------------------------------------- -->

 <hr/> 
<h2 id='hit'>Hit qualification</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_hit'>Show Data</button>
</center>

<div id='hide_hit' style='display:none;'> <!-- Div to hide and load on button click -->



Here we utilised our already established methodology to estimate the probability of failure in clinical trials 
(<a href='http://biorxiv.org/content/biorxiv/early/2016/12/29/095653.full.pdf' target='_blank'>Artemov et al., 2016</a>). 
On Figures 1 and 2 you can see the overall statistic for provided L1000 dataset. In "Hit qualification table" you can see the actual score of drug perturbations analysed in this report.
<br/>
<br/>
<center>
<img src='/media/report-portal/gsk_prj4_t1/hit1.png' />
<p>Figure 1. Probability of success for drug perturbations from provided L1000 dataset.
</p>
<hr style='width:50%' />

<img src='/media/report-portal/gsk_prj4_t1/hit2.png' />
<p>Figure 2. Probability of failure for drug perturbations from provided L1000 dataset.
</p>
<hr style='width:50%' />
</center>
<h4>Table <span class='tbl_num'></span>Hit qualification </h4>
       <table id="tbl-hit" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Name</th>
           <th>Failed</th>
           <th>Passed</th>

          </tr>
        </thead>  
      </table>
      <br/><br/>

</div>
<script>

$(function() {
	
	
	$("#btn_show_hit").click(function(){
		
		$(this).hide();
		$("#hide_hit").slideDown('slow');
	});
	
	
	var tbl_hit = $("#tbl-hit").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl-hit'},
	               {extend: 'print', title: 'tbl-hit'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'clinical_trial/predCT.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
});

</script>
<!-- -----------------end of Hit qualification -->

<!-- ---------------------------------Tox groups analysis--------------------------------------------- -->

 <hr/> 
<h2 id='tox'>Tox groups analysis</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_tox'>Show Data</button>
</center>

<div id='hide_tox' style='display:none;'> <!-- Div to hide and load on button click -->

<br/>
<p>
Here we investigated the common and differential features among 5 Tox groups highlighted in the dataset
 annotation table. All samples corresponding to drugs from a particular group (case) were aggregated 
 and compared to "Rat clean" group (reference) that corresponds to perturbations with compounds that 
 didn't demonstrate toxicity. Resulting lists of differentially expressed genes were compared across all 
 5 Tox groups and obrained overlaps are depicted below and listed in the table.
</p>
<br/>
<center>
<img src='/media/report-portal/gsk_prj4_t5/img_tox.png' />
<p>
Differentially expressed genes distribution across 5 distinct Tox groups
</p>
<hr style='width:50%' />

</center>
<h4>Table <span class='tbl_num'></span>Tox groups analysis </h4>
       <table id="tbl-tox" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Gene</th>
           <th>Mitotox</th>
           <th>MRGX2 AG</th>
           <th>Structural Cardiotox-Positive</th>
           <th>Vasc. Tox Rat Positive</th>
           <th>SAPAD CS deg.nec</th>

          </tr>
        </thead>  
      </table>
      <br/><br/>

<div>

<span id="docs-internal-guid-9a8d3d3a-bce0-68ec-2d14-af59abfa0647">
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;text-align: center;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; text-decoration-line: underline; vertical-align: baseline; white-space: pre-wrap;">Cardiotoxicity</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">HMOX1 (Heme Oxygenase 1) (upregulated) &nbsp;</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">is </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">&nbsp;</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">also &nbsp;called</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;"> </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">HO-1 and its expression was found to be increased in cytoplasmic extracts in a dose-dependant manner in a study about Rosiglitazone inducing cardiotoxicity by accelerated apoptosis </span><a href="https://paperpile.com/c/hBQxRz/yZ8x" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Mishra et al. 2014)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">BMP4 (Bone Morphogenetic Protein 4) (upregulated)</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> &nbsp;BMPR2 is known to be critical in cardiogenesis; knockdown results in embryonic lethality via abnormal homeobox expression and heart formation </span><a href="https://paperpile.com/c/hBQxRz/8EDS" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Shi et al. 2000)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">, particularly involving positioning of the outflow tracts, remodeling of the arterioventricular cushion, and development of the valve leaflets </span><a href="https://paperpile.com/c/hBQxRz/p9Ds" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Beppu et al. 2009)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">GADD45A (Growth Arrest And DNA Damage Inducible Alpha) (upregulated)</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> It was shown to &nbsp;be a differentially expressed gene related to &nbsp;various pathways affected upon doxorubicin exposure leading to doxorubicin-induced toxicity in human cardiomyocytes </span><a href="https://paperpile.com/c/hBQxRz/e8EC" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Holmgren et al. 2015)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">ANKRD1 (Ankyrin Repeat Domain 1) (upregulated)</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> &nbsp;Cardiac ankyrin repeat protein (CARP, ANKRD1) is a transcriptional regulatory protein that is extremely susceptible to doxorubicin. CARP expression is regulated in part by GATA4 </span><a href="https://paperpile.com/c/hBQxRz/uKbi" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Chen et al. 2012)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">. </span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">MPO (Myeloperoxidase) (upregulated)</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> &nbsp;&nbsp;MPO was identified as one of two predictors of cardiotoxicity in breast cancer patients treated with ACs and Herceptin, from a panel of potential biomarkers including CRP, NT-proBNP, growth differentiation factor (GDF)-15, placenta growth factor (PlGF), soluble fms-like tyrosine kinase receptor (sFlt)-1, and galectin (gal)-3 </span><a href="https://paperpile.com/c/hBQxRz/Ax3q" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Ky et al. 2014)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">MDH2 (Malate Dehydrogenase 2) (upregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">mitochondrial malate dehydrogenase (MDH2) was recently discussed &nbsp;as a novel target for reducing DOX cardiotoxicity </span><a href="https://paperpile.com/c/hBQxRz/XowW" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Liu et al. 2014)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> &nbsp;It was demonstrated that treatment with structurally diverse inhibitors of MDH2 (i.e., mebendazole, thyroxine, iodine, malate) &nbsp;prevented DOX cardiotoxicity </span><a href="https://paperpile.com/c/hBQxRz/Qwqm" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Xi 2016)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">. </span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">TNNT1 (Troponin T1, Slow Skeletal Type) (upregulated)</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> it is a specific biomarkers of drug-induced skeletal muscle injury </span><a href="https://paperpile.com/c/hBQxRz/Tae2" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Campion et al. 2013)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">CDC42EP3 (CDC42 Effector Protein 3) (upregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene was found differentially expressed in a study about comparative phenotypic assessment of cardiac pathology </span><a href="https://paperpile.com/c/hBQxRz/FzcU" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Auerbach et al. 2010)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">. </span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">AKAP13 (A-Kinase Anchoring Protein 13) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Study showed that AKAP13-Mediated PKD1 signaling regulates the transcriptional response to cardiac hypertrophy </span><a href="https://paperpile.com/c/hBQxRz/dtdR" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Johnson et al. 2015)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">IL3RA (Interleukin 3 Receptor Subunit Alpha) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Its role was discussed in a study about genetic contributions to cardiac pathology induced by beta-adrenergic stimulation </span><a href="https://paperpile.com/c/hBQxRz/CPUE" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Rau et al. 2015)</span></a>
</p><br><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;text-align: center;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; text-decoration-line: underline; vertical-align: baseline; white-space: pre-wrap;">Mitotoxicity</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">SPARC (Secreted Protein Acidic And Cysteine Rich) (upregulated) &nbsp;</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">overexpression of SPARC suppresses radiation-induced HSP27 expression in neuroblastoma cells and the combination of SPARC and radiation induces the expression of protein 21, but suppresses neuroblastoma tumor density. SPARC also induces mitochondrial Δψ collapse in SK-N-BE and NB1691 neuroblastoma cells </span><a href="https://paperpile.com/c/hBQxRz/3CBu" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Tanpure et al. 2017)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> </span><a href="https://paperpile.com/c/hBQxRz/djwF" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Pinto et al. 2015)</span></a>
</p><br><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">EREG (Epiregulin) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">epiregulin significantly increased the proportion of J aggregates (from JC-1) in oocyte mitochondria and this coupled with FAD and REDOX measures indicates greater mitochondrial activity </span><a href="https://paperpile.com/c/hBQxRz/80hJ" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Richani et al. 2014)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> </span><a href="https://paperpile.com/c/hBQxRz/Cedh" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Campos-Chillon et al. 2015)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">. &nbsp;</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">MMP3 (Matrix Metallopeptidase 3) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Homocysteine stimulate intracellular ROS production and mitochondria dysfunction, then decrease the cell viability and bone formation activity, the mechanism may depend on increasing MMPs, ROS production, and also dysfunction of mitochondria. results showed that an endopeptidase, matrix metalloproteinase-3 (MMP3), was induced and activated in dopaminergic (DA) cells upon stress conditions</span><a href="https://paperpile.com/c/hBQxRz/l2uk" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Choi et al. 2011)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> . </span>
</p><br><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">PAK2 (P21 (RAC1) Activated Kinase 2) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Paks regulate cell survival and death in response to cellular stimuli. Full-length Pak1, Pak2, Pak4 and Pak5 have been reported to have anti-apoptotic roles. Pak2 can be activated via cleavage by caspase proteases to release a constitutively active C-terminal fragment (Pak2p34). Caspase-activated Pak2p34 has been observed after cells are exposed to death stimuli such as Fas ligand, TNFα, heat shock or UV radiation </span><a href="https://paperpile.com/c/hBQxRz/1b8a" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Luo and Rubinsztein 2009)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> </span><a href="https://paperpile.com/c/hBQxRz/E9H1" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Chan, Yu, and Yang 1999)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">DAD1 (Defender Against Cell Death 1) (upregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">the mammalian defender against cell death-1 is involved in apoptosis regulation </span><a href="https://paperpile.com/c/hBQxRz/uBNK" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Wnęk et al. 2016)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> </span><a href="https://paperpile.com/c/hBQxRz/xJNu" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Carmona-Gutierrez et al. 2010)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">. </span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">LPIN1 (Lipin 1) (upregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">The family of three lipin proteins act as phosphatidate phosphatase (PAP) enzymes required for glycerolipid biosynthesis, and also as transcriptional coactivators that regulate expression of lipid metabolism genes. Its main functions was reviewed </span><a href="https://paperpile.com/c/hBQxRz/Sx0j" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Reue 2009)</span></a>
</p><br><br><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;text-align: center;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; text-decoration-line: underline; vertical-align: baseline; white-space: pre-wrap;">Sapad genes</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">DCAF6 (DDB1 And CUL4 Associated Factor 6) (downregulated).</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">The protein encoded by this gene is a ligand-dependent coactivator of nuclear receptors, including nuclear receptor subfamily 3 group C member 1 (NR3C1), glucocorticoid receptor (GR), and androgen receptor (AR). &nbsp;Diseases associated with DCAF6 include Male Reproductive Organ Cancer</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">NSF (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Also &nbsp;called N-Ethylmaleimide Sensitive Factor, Vesicle Fusing ATPase. Diseases associated with NSF include Tetanus and Neuronal Intranuclear Inclusion Disease</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">TCEB2 (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Also &nbsp;called Elongin B. It is a subunit of the transcription factor B (SIII) complex. The SIII complex is composed of elongins A/A2, B and C. It activates elongation by RNA polymerase II by suppressing transient pausing of the polymerase at many sites within transcription units. Diseases associated with ELOB include Renal Cell Carcinoma and Renal Clear Cell Carcinoma</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">COX4I1 (downregulated)</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> Cytochrome C Oxidase Subunit 4I1. It &nbsp;is the terminal enzyme of the mitochondrial respiratory chain. Diseases associated with COX4I1 include Calvarial Hyperostosis and Hyperostosis</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">RPS14P3 (downregulated)</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> it is ribosomal protein S14 pseudogene 3.</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">NUP155 (Nucleoporin 155) (upregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Its role was discussed in a study about the evaluation of drug-induced neurotoxicity </span><a href="https://paperpile.com/c/hBQxRz/wZWB" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Schultz et al. 2015)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">EFHC1 (EF-Hand Domain Containing 1) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene is listed in a patent application about the compositions and methods for diagnosis or treatment of epilepsy disease with EFHC1, EFHC1 agonists, or EFHC1 analogs (patent application number US9220697 B2)</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">TSPAN6 (Tetraspanin 6) (downregulated)</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> Its involvement in the Molecular Pathomechanisms of Classical Galactosemia was discussed </span><a href="https://paperpile.com/c/hBQxRz/TEyK" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Staubach et al. 2017)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">RRP1 (Ribosomal RNA Processing 1) (upregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene was listed in a patent application on the use of markers in the identification of cardiotoxic agents and in the diagnosis and monitoring of cardiomyopathy and cardiovascular disease (patent application US20140100128 A1)</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">TAGLN (Transgelin) (upregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">this gene was shown to &nbsp;be highly perturbed &nbsp;in a comparative study of lung toxicity in rats induced by three types of nanomaterials </span><a href="https://paperpile.com/c/hBQxRz/X2lm" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Lin et al. 2013)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">. &nbsp;This gene is also used as vascular-organ-specific biomarkers </span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">TGDS (TDP-Glucose 4,6-Dehydratase) (upregulated). </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">&nbsp;Mutations in this gene have been associated with Catel-Manzke syndrome, which is characterized by Pierre Robin sequence, and radial deviation of the index finger due to the presence of an accessory bone between the index finger and its proximal phalanx. Other diseases associated with TGDS include Catel-Manzke Syndrome and Hyperphalangy</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">RNF10 (Ring Finger Protein 10) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene is listed in a patent (US publication EP1585972 B1) application about the use of gene expression markers for assessing cardiac allograft rejection </span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">PWP2 (PWP2, UTPB SSU Processome Subunit) (upregulated) &nbsp;</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">In </span><a href="https://paperpile.com/c/hBQxRz/v84m" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(James et al. 2014)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> depletion of Pwp2 was discussed to &nbsp;cause an accumulation of unbudded cells. In contrast, even after an extended </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Pwp2</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> depletion, the </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">whi5</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> null cultures accumulated far fewer unbudded cells, indicating </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Whi5</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> plays a role in regulating the G1/S checkpoint upon depletion of a ribosome biogenesis factor</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">EEF1D (Eukaryotic Translation Elongation Factor 1 Delta) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene appears in a Signaling networks (Gene expression, tissue morphology, amino acid metabolism) of components affected by nitrite therapy &nbsp;which &nbsp;protects against kidney injury induced by brain death </span><a href="https://paperpile.com/c/hBQxRz/aKw3" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Kelpke et al. 2012)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">CNTRL (Centriolin) (upregulated)</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> This gene encodes a centrosomal protein required for the centrosome to function as a microtubule organizing center. Diseases associated with CNTRL include 8P11 Myeloproliferative Syndrome and Microcephalic Osteodysplastic Primordial Dwarfism, Type Ii.</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">PVRIG (Poliovirus Receptor Related Immunoglobulin Domain Containing) (upregulated)</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> &nbsp;Cell surface receptor for NECTIN2. May act as a coinhibitory receptor that suppresses T-cell receptor-mediated signals. Following interaction with NECTIN2, inhibits T-cell proliferation. Competes with CD226 for NECTIN2-binding.</span>
</p><br><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">GMCL1 (Germ Cell-Less, Spermatogenesis Associated 1) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">The gene is described in a study about the effect of Valporoic acid on pathologic endothelial &nbsp;cell gene expression profile associated with ischemia </span><a href="https://paperpile.com/c/hBQxRz/kGKt" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Causey et al. 2012)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">. </span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">MCAM (Melanoma Cell Adhesion Molecule) (upregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Also &nbsp;CD146. It was investigated as a potential molecular target for diagnosis and targeted therapy of glioblastoma multiforme </span><a href="https://paperpile.com/c/hBQxRz/SQuB" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Y. Yang et al. 2015)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">SSH1 (Slingshot Protein Phosphatase 1) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">The etiology of disseminated superficial actinic porokeratosis (DSAP) involves the focal clonal expansion of atypical keratinocytes, which give rise to the shape of the cornoid lamella </span><a href="https://paperpile.com/c/hBQxRz/pIXA" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Bhatia et al. 2014)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">. Phenotypic variants include a simultaneous expression of closely linked genes (gene studies map the loci responsible on chromosomes 12q, 15q, and 18q) </span><a href="https://paperpile.com/c/hBQxRz/vTXo" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Wu et al. 2004)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">.Several mutations have been identified in the SSH1 gene on chromosome 12 </span><a href="https://paperpile.com/c/hBQxRz/FHxy" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Zhang et al. 2004)</span></a>
</p><br><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">EPHX2 (Epoxide Hydrolase 2) (upregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene is involved in acute Doxorubicin Toxicity </span><a href="https://paperpile.com/c/hBQxRz/lfKx" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Zordoky et al. 2011)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;text-align: center;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; text-decoration-line: underline; vertical-align: baseline; white-space: pre-wrap;">Vasctox</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">BPTF (Bromodomain PHD Finger Transcription Factor) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene (acting on H3K4me) is instrumental in turning on a genetic cardiac program </span><a href="https://paperpile.com/c/hBQxRz/E9H1+9ncs" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Chan, Yu, and Yang 1999; Van Vliet et al. 2012)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">.</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">ZNF423 (Zinc Finger Protein 423) (downregulated)</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">WBP11 (WW Domain Binding Protein 11) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene was shown to &nbsp;be differentially expressed in an analysis describing the distinct cardiac transcriptomic profile defining pregnancy and exercise </span><a href="https://paperpile.com/c/hBQxRz/lLcK" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Chung, Heimiller, and Leinwand 2012)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">. </span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">TBCE (Tubulin Folding Cofactor E) (upregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">There are tubulin-active agents which &nbsp;are used as chemotherapeutics and elicit various kinds of cardiac toxicities &nbsp;</span><a href="https://paperpile.com/c/hBQxRz/jeVO" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Menna, Salvatorelli, and Minotti 2008)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;"> MZT2A (Mitotic Spindle Organizing Protein 2A) (downregulated)</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">NDUFA6 (NADH:Ubiquinone Oxidoreductase Subunit A6) (upregulated)</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;"> SMYD2 (SET And MYND Domain Containing 2) (upregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">The proteins SMYD2 and SMYD3 are two of five members of a unique family of lysine methyltransferases defined by a catalytic SET domain that is split into two</span>
</p>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">segments by a MYND protein interaction domain, followed by a cysteine-rich post-SET domain. The SMYD family members have been shown to be essential for cellular</span>
</p>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">development, cell cycle progression, and when dysregulated, tumorigenesis. SMYD have</span>
</p>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">been widely studied as a pivotal component of cardiac and skeletal muscle development.Smyd2 was also &nbsp;patented as a target gene for cancer therapy and diagnosis (US patent US 20140308678 A1, PCT/JP2012/004232)</span>
</p><br><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">ABCB7 (ATP Binding Cassette Subfamily B Member 7) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene was found to be involved with heme in cardiovascular physiology and disease </span><a href="https://paperpile.com/c/hBQxRz/FEnS" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Sawicki, Chang, and Ardehali 2015)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">CNPPD1 (Cyclin Pas1/PHO80 Domain Containing 1) (downregulated)</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">EDNRA (Endothelin Receptor Type A) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">&nbsp;endothelin receptor type A. It was shown that cardiac metabolic deregulation induced by the Tyrosine Kinase Receptor Inhibitor Sunitinib is rescued by Endothelin Receptor Antagonism </span><a href="https://paperpile.com/c/hBQxRz/3nBT" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Sourdon et al. 2017)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">BCAS3 (BCAS3, Microtubule Associated Cell Migration Factor) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">The role of this &nbsp;gene was discussed with respect to &nbsp;the epigenetic foundation of the cardiovascular disease </span><a href="https://paperpile.com/c/hBQxRz/rcpF" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Aslibekyan, Claas, and Arnett 2015)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">KDM3B (Lysine Demethylase 3B) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">this gene </span><a href="https://paperpile.com/c/hBQxRz/yrCi" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Tough et al. 2014)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> &nbsp;was found to &nbsp;be involved in the epigenetic regulation of endothelial-cell-mediated vascular repair </span><a href="https://paperpile.com/c/hBQxRz/CeJC" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Fraineau et al. 2015)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">.</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">GLIPR1 (GLI Pathogenesis Related 1) (upregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene was cited in a patent application related to methods and compositions for the treatment of diseases including cardiovascular ones (US publication US20090163434 A1)</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">HEXB (Hexosaminidase Subunit Beta) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene was involved in cardiac anomalies in -glucuronidase-null mice </span><a href="https://paperpile.com/c/hBQxRz/sEjA" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Schuldt et al. 2004)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">ATXN2 (Ataxin 2) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">ATXN2 was described as a cytoplasmic protein,</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;"> </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">the physiological role of ATXN2 subtly modifies the abundance of cellular translation factors as well as global translation </span><a href="https://paperpile.com/c/hBQxRz/jNPA" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Fittschen et al. 2015)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">. </span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">PLCB4 (Phospholipase C Beta 4) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene is included in an analysis of gene networks associated with chronic cardiovascular disease </span><a href="https://paperpile.com/c/hBQxRz/C0zJ" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Yu et al. 2015)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> It also &nbsp;played a role in cardiac metabolic deregulation Induced by the tyrosine kinase receptor &nbsp;</span><a href="https://paperpile.com/c/hBQxRz/3nBT" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Sourdon et al. 2017)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">PFDN2 (Prefoldin Subunit 2) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">this gene has direct interaction with MAP3K3 which is an essential actor for early embryonic cardiovascular development </span><a href="https://paperpile.com/c/hBQxRz/drqW" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(J. Yang et al. 2000)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">PSMA1 (Proteasome Subunit Alpha 1) (upregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene was included in a patent application for Protein markers for cardiovascular events (patent publication EP2019318 A1) </span>
</p>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><a href="https://paperpile.com/c/hBQxRz/4tOs" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Fuchs et al. 2014)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">RBM4 (RNA Binding Motif Protein 4) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene appears in a study about the identification of RNA binding motif proteins essential for cardiovascular development </span><a href="https://paperpile.com/c/hBQxRz/Waup" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Maragh et al. 2011)</span></a>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">PTPRM (Protein Tyrosine Phosphatase, Receptor Type M) (downregulated) </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">This gene was shown to be involved in various aspects of cardiovascular autophagy </span><a href="https://paperpile.com/c/hBQxRz/iXHc" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">(Salabei and Conklin 2013)</span></a><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">. </span>
</p>
<div><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"><br></span>
</div></span>


</div>


</div>
<script>

$(function() {
	
	
	$("#btn_show_tox").click(function(){
		
		$(this).hide();
		$("#hide_tox").slideDown('slow', function(){
			
			var tbl_tox = $("#tbl-tox").DataTable( {
				"paging":   true,
			    "iDisplayLength": 20,
			    "ordering": true,
			    "order": [],
			    "info":     false,
			    "dom": 'Bfrtip',
			    "scrollX": false,
			    "bAutoWidth": false,
			    "buttons": [
			               {extend: 'csv', title: 'tbl-tox'},
			               {extend: 'print', title: 'tbl-tox'}
			               
			          ],
			    
			    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
			    	     'type': 'GET',
			    	     'data':{
			    	    	     reportID: {{report.id}},
			    	    	     file_name: 'gsk_prj4_t5/de_genes_binary.csv'
			    	            },        	      
			    
			},"deferRender": true });
		});
	});
	
	
	
	
});

</script>
<!-- -----------------end of Tox groups analysis -->

<!-- ---------------------------------Toxcast analysis--------------------------------------------- -->
<div id="hide_section_tox_analysis" >
 <hr/> 
<h2 id='tox_analysis'>Toxcast analysis</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_tox_analysis'>Show Data</button>
</center>

<div id='hide_tox_analysis' style='display:none;'> <!-- Div to hide and load on button click -->

<br/>
<p>
<span id="docs-internal-guid-6b4a8ad0-bfc2-e431-5957-c9307582ad8c">
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:8pt;background-color:#ffffff;"><span style="font-size: 10.5pt; font-family: Arial; background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">We collected ToxCast data for SAg and 3C systems from EPA. For SAg we found 10 common biomarkers that are present in investigated dataset, these include: CD38, CD40, CD69, CD62E/E-Selectin, CXCL8/IL-8, CCL2/MCP-1, CXCL9/MIG, PBMC Cytotoxicity, Proliferation, SRB. For 3C we found 12 common biomarkers: CD62E/E-Selectin, CXCL8/IL-8, CCL2/MCP-1, CXCL9/MIG, PBMC Cytotoxicity, Proliferation, SRB, CD54/ICAM-1, HLA-DR, CD141/Thrombomodulin, CD142/Tissue Factor, CD106/VCAM-1, CD87/uPAR. For each drug perturbation from BioMAP we correlated log-ratio/conf.envelope to downward curve-fit analysis readout from ToxCast. Results can be found in two following tables.</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">Input:</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 
Biomap datatable (<a href='/media/report-portal/gsk_prj4_t5/diversity_plus.biomarker_scores.renamed_features.csv' >Download file</a>): 
biomarkers as rows, perturbations as columns; ToxCast tables with readouts 
for 3C (<a href='/media/report-portal/gsk_prj4_t5/toxcast.3C_down.csv' >Download file</a>) 
and SAg(<a href='/media/report-portal/gsk_prj4_t5/toxcast.SAg_down.csv' >Download file</a>): 
biomarkers as columns, perturbations as rows.</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">Processing:</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> Within each of the two systems (3C, SAg) for each compound perturbation pair (GSKvsToxCast) rows (biomarkers) from BioMAP datatable were matched to columns (biomarkers) of ToxCast table. For each compound pair (GSKvsToxCast) this resulted in two vectors of length 10 (in case of SAg system) and 12 (in case of 3C system). Pearson correlation was calculated for all possible compound perturbation pairs (GSKvsToxCast).</span>
</p><br>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-weight: 700; vertical-align: baseline; white-space: pre-wrap;">Output:</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> A table (with ToxCast perturbations as rows and BioMAP perturbations as columns) populated with Pearson correlation values. As the calculations for 3C and SAg were done independently, two separate tables were generated.</span>
</p>
<div><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"><br></span>
</div></span>
</p>
<br/>

<h4>Table <span class='tbl_num'></span>3C results </h4>
  <div style="overflow-x:auto;">
       <table id="tbl-diversity_toxcast_3C" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          
        </thead>  
      </table>
  </div>
      <br/><br/>
      
      <hr style="width:50%" />
  <h4>Table <span class='tbl_num'></span>SAg results </h4>
  <div style="overflow-x:auto;">
       <table id="tbl-diversity_toxcast_SAg" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          
        </thead>  
      </table>
  </div>
      <br/><br/>

</div>

</div> <!-- end of hide section div -->
<script>

$(function() {
	
	// Hide Section if files are missing
	$.ajax({
        type: 'GET',
        dataType: 'json',
        url: '/report-portal/report-ajaxtablecolumns/',
        data:{
   	         reportID: {{report.id}},
   	         file_name: 'granular_toxcast_3C.txt'
           },
        success: function(d) { },
        error: function(e) {
        	                $("#hide_section_tox_analysis").hide();
                            }
	});
	
	
	$("#btn_show_tox_analysis").click(function(){
		
		$(this).hide();
		$("#hide_tox_analysis").slideDown('slow', function(){
			
			
			/////////////////////////// 3C
			$.ajax({
		        type: 'GET',
		        dataType: 'json',
		        url: '/report-portal/report-ajaxtablecolumns/',
		        data:{
		   	         reportID: {{report.id}},
		   	         file_name: 'granular_toxcast_3C.txt'
		           },
		        success: function(d) {
		        	
		        	let thead = '<tr>';
		        	
		        	for (let col of d.columns) {
		        		
		        		thead+='<th>'+col+'</th>';
		        	}
		        	
		        	thead+='</tr>';
		        	
		        	$("#tbl-diversity_toxcast_3C").find("thead").html(thead);
		        	
		        	var tbl_diversity_toxcast_3C = $("#tbl-diversity_toxcast_3C").DataTable( {
		        		"paging":   true,
		        	    "iDisplayLength": 20,
		        	    "ordering": true,
		        	    "order": [],
		        	    "info":     false,
		        	    "dom": 'Bfrtip',
		        	    "scrollX": false,
		        	    "bAutoWidth": false,
		        	    "buttons": [
		        	               {extend: 'csv', title: 'tbl-diversity_toxcast_3C'},
		        	               {extend: 'print', title: 'tbl-diversity_toxcast_3C'}
		        	               
		        	          ],
		        	    
		        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
		        	    	     'type': 'GET',
		        	    	     'data':{
		        	    	    	     reportID: {{report.id}},
		        	    	    	     file_name: 'granular_toxcast_3C.txt'
		        	    	            },        	      
		        	    
		        	},"deferRender": true });
		        }
		    });
			
			////////////////////////// SAg
			
			$.ajax({
		        type: 'GET',
		        dataType: 'json',
		        url: '/report-portal/report-ajaxtablecolumns/',
		        data:{
		   	         reportID: {{report.id}},
		   	         file_name: 'granular_toxcast_SAg.txt'
		           },
		        success: function(d) {
		        	
		        	let thead = '<tr>';
		        	
		        	for (let col of d.columns) {
		        		
		        		thead+='<th>'+col+'</th>';
		        	}
		        	
		        	thead+='</tr>';
		        	
		        	$("#tbl-diversity_toxcast_SAg").find("thead").html(thead);
		        	
		        	var tbl_diversity_toxcast_3C = $("#tbl-diversity_toxcast_SAg").DataTable( {
		        		"paging":   true,
		        	    "iDisplayLength": 20,
		        	    "ordering": true,
		        	    "order": [],
		        	    "info":     false,
		        	    "dom": 'Bfrtip',
		        	    "scrollX": false,
		        	    "bAutoWidth": false,
		        	    "buttons": [
		        	               {extend: 'csv', title: 'tbl-diversity_toxcast_SAg'},
		        	               {extend: 'print', title: 'tbl-diversity_toxcast_SAg'}
		        	               
		        	          ],
		        	    
		        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
		        	    	     'type': 'GET',
		        	    	     'data':{
		        	    	    	     reportID: {{report.id}},
		        	    	    	     file_name: 'granular_toxcast_SAg.txt'
		        	    	            },        	      
		        	    
		        	},"deferRender": true });
		        }
		    });
			
			
		});
		
	});
			
			
	
	
	
	
});

</script>
<!-- -----------------end of Toxcast analysis analysis -->


<!-- ---------------------------------Similarity--------------------------------------------- -->

 <hr/> 
<h2 id='sim_analysis'>Similarity to LINCS dataset</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_sim'>Show Data</button>
</center>

<div id='hide_sim' style='display:none; overflow-x: auto;'> <!-- Div to hide and load on button click -->
<br/>
<p>
In this section we calculated the similarity of investigated perturbations to LINCS dataset. For each sample we performed the 
correlation analysis of its gene expression values and 1.3+ million samples from LINCS on the probe level. Then for each sample 
we picked top-200 LINCS samples by correlation coefficient and combined these lists in the resulting table. Within each perturbation 
we averaged correlation coefficients between two replicates. In LINCS dataset pert_ids with "BRD-", "TRCN-" and "BRDN-" prefixes 
designate small molecule, knock-down and ORF overexpression perturbations, respectively. In pert_desc column you can find 
perturbation name (name of the molecule or name of the gene in case of knock-downs and ORF overexpression).
</p>
<br/><br/>
<h4>Table <span class='tbl_num'></span>Similarity scoring of GSK L1000 data in comparison to LINCS dataset </h4>
       <table id="tbl-sim" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
      </table>
      <br/><br/>





</div>

<script>
$(function() {
	
	
	$("#btn_show_sim").click(function(){
		
		$(this).hide();
		$("#hide_sim").slideDown('slow');
	});
	
	
	$.ajax({
        type: 'GET',
        dataType: 'json',
        url: '/report-portal/report-ajaxtablecolumns/',
        data:{
   	         reportID: {{report.id}},
   	         file_name: 'q2norm/correlation_top3.csv'
           },
        success: function(d) {
        	
        	let thead = '<tr>';
        	
        	for (let col of d.columns) {
        		
        		thead+='<th>'+col+'</th>';
        	}
        	
        	thead+='</tr>';
        	
        	$("#tbl-sim").find("thead").html(thead);
        	
        	var tbl_sim = $("#tbl-sim").DataTable( {
        		"paging":   true,
        	    "iDisplayLength": 20,
        	    "ordering": true,
        	    "order": [],
        	    "info":     false,
        	    "dom": 'Bfrtip',
        	    "scrollX": false,
        	    "bAutoWidth": false,
        	    "buttons": [
        	               {extend: 'csv', title: 'tbl-sim'},
        	               {extend: 'print', title: 'tbl-sim'}
        	               
        	          ],
        	    
        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
        	    	     'type': 'GET',
        	    	     'data':{
        	    	    	     reportID: {{report.id}},
        	    	    	     file_name: 'q2norm/correlation_top3.csv'
        	    	            },        	      
        	    
        	},"deferRender": true });
        }
    });
	/*
	
	*/
});


</script> 

<!-- -----------------end of Similarity -->

<!-- ---------------------------------Barcodes--------------------------------------------- -->

 <hr/> 
<h2 id='barcodes'>Barcodes for L1000 data</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_barcodes'>Show Data</button>
</center>

<div id='hide_barcodes' style='display:none; overflow-x: auto;'> <!-- Div to hide and load on button click -->
<br/>
<p>
Here we utilised an improved version of methodology from 
the <a href='http://journals.plos.org/ploscompbiol/article/authors?id=10.1371/journal.pcbi.1005335' target='_blank' >original paper</a>. 
We used landmark gene expression values to represent each sample through 100 distinct features.
</p>
<br/><br/>
<h4>Table <span class='tbl_num'></span>Barcodes for L1000 data </h4>
       <table id="tbl-barcodes" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
      </table>
      <br/><br/>





</div>

<script>
$(function() {
	
	
	$("#btn_show_barcodes").click(function(){
		
		$(this).hide();
		$("#hide_barcodes").slideDown('slow');
	});
	
	
	$.ajax({
        type: 'GET',
        dataType: 'json',
        url: '/report-portal/report-ajaxtablecolumns/',
        data:{
   	         reportID: {{report.id}},
   	         file_name: 'barcodes.csv'
           },
        success: function(d) {
        	
        	let thead = '<tr>';
        	
        	for (let col of d.columns) {
        		
        		thead+='<th>'+col+'</th>';
        	}
        	
        	thead+='</tr>';
        	
        	$("#tbl-barcodes").find("thead").html(thead);
        	
        	var tbl_barcodes = $("#tbl-barcodes").DataTable( {
        		"paging":   true,
        	    "iDisplayLength": 20,
        	    "ordering": true,
        	    "order": [],
        	    "info":     false,
        	    "dom": 'Bfrtip',
        	    "scrollX": false,
        	    "bAutoWidth": false,
        	    "buttons": [
        	               {extend: 'csv', title: 'tbl-barcodes'},
        	               {extend: 'print', title: 'tbl-barcodes'}
        	               
        	          ],
        	    
        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
        	    	     'type': 'GET',
        	    	     'data':{
        	    	    	     reportID: {{report.id}},
        	    	    	     file_name: 'barcodes.csv'
        	    	            },        	      
        	    
        	},"deferRender": true });
        }
    });
	/*
	
	*/
});


</script> 

<!-- -----------------end of Barcodes for L1000 data -->

<hr/>
  
  {%if report.conclusion_text%}
          <h2 id="conclusions" >Discussion</h2>
          <hr/>
          
          {{report.conclusion_text|safe}}
          
          <hr/>
  {%endif%}
          
  
          <h2 id="references" >References</h2>
          <hr/>
          
          <span id="docs-internal-guid-9a8d3d3a-bce2-45e9-f9bd-f7b5a8a1adbf">
<p dir="ltr" style="line-height:1.2;margin-top:12pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/rcpF" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Aslibekyan, Stella, Steven A. Claas, and Donna K. Arnett. 2015. “Clinical Applications of Epigenetics in Cardiovascular Disease: The Long Road Ahead.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Translational Research: The Journal of Laboratory and Clinical Medicine</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 165 (1): 143–53.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/FzcU" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Auerbach, Scott S., Reuben Thomas, Ruchir Shah, Hong Xu, Molly K. Vallant, Abraham Nyska, and June K. Dunnick. 2010. “Comparative Phenotypic Assessment of Cardiac Pathology, Physiology, and Gene Expression in C3H/HeJ, C57BL/6J, and B6C3F1/J Mice.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Toxicologic Pathology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 38 (6): 923–42.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/p9Ds" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Beppu, Hideyuki, Rajeev Malhotra, Yuko Beppu, John J. Lepore, Michael S. Parmacek, and Kenneth D. Bloch. 2009. “BMP Type II Receptor Regulates Positioning of Outflow Tract and Remodeling of Atrioventricular Cushion during Cardiogenesis.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Developmental Biology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 331 (2): 167–75.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/pIXA" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Bhatia, Neal, Andrew Blauvelt, Marc Brown, Whitney High, Craig T. Leonardi, Ted Rosen, Linda Stein Gold, et al. 2014. “Updates on Psoriasis and Cutaneous Oncology: Proceedings from the 2014 MauiDerm Meeting.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">The Journal of Clinical and Aesthetic Dermatology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 7 (7 Suppl SC2S5-S22): S5–22.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/Tae2" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Campion, Sarah, Jiri Aubrecht, Kim Boekelheide, David W. Brewster, Vishal S. Vaidya, Linnea Anderson, Deborah Burt, et al. 2013. “The Current Status of Biomarkers for Predicting Toxicity.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Expert Opinion on Drug Metabolism & Toxicology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 9 (11): 1391–1408.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/Cedh" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Campos-Chillon, Fernando, Todd A. Farmerie, Gerrit J. Bouma, Colin M. Clay, and Elaine M. Carnevale. 2015. “Effects of Aging on Gene Expression and Mitochondrial DNA in the Equine Oocyte and Follicle Cells.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Reproduction, Fertility, and Development</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 27 (6): 925–33.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/xJNu" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Carmona-Gutierrez, D., T. Eisenberg, S. Büttner, C. Meisinger, G. Kroemer, and F. Madeo. 2010. “Apoptosis in Yeast: Triggers, Pathways, Subroutines.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Cell Death and Differentiation</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 17 (5): 763–73.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/kGKt" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Causey, Marlin Wayne, Shashikumar Salgar, Niten Singh, Matthew Martin, and Jonathan D. Stallings. 2012. “Valproic Acid Reversed Pathologic Endothelial Cell Gene Expression Profile Associated with Ischemia-Reperfusion Injury in a Swine Hemorrhagic Shock Model.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Journal of Vascular Surgery</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 55 (4): 1096–1103.e51.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/E9H1" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Chan, W. H., J. S. Yu, and S. D. Yang. 1999. “PAK2 Is Cleaved and Activated during Hyperosmotic Shock-Induced Apoptosis via a Caspase-Dependent Mechanism: Evidence for the Involvement of Oxidative Stress.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Journal of Cellular Physiology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 178 (3): 397–408.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/uKbi" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Chen, Billy, Lin Zhong, Sarah F. Roush, Laura Pentassuglia, Xuyang Peng, Susan Samaras, Jeffrey M. Davidson, Douglas B. Sawyer, and Chee Chew Lim. 2012. “Disruption of a GATA4/Ankrd1 Signaling Axis in Cardiomyocytes Leads to Sarcomere Disarray: Implications for Anthracycline Cardiomyopathy.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">PloS One</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 7 (4): e35743.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/l2uk" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Choi, Dong-Hee, Youn-Jung Kim, Young-Gun Kim, Tong H. Joh, M. Flint Beal, and Yoon-Seong Kim. 2011. “Role of Matrix Metalloproteinase 3-Mediated Alpha-Synuclein Cleavage in Dopaminergic Cell Death.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">The Journal of Biological Chemistry</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 286 (16): 14168–77.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/lLcK" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Chung, Eunhee, Joseph Heimiller, and Leslie A. Leinwand. 2012. “Distinct Cardiac Transcriptional Profiles Defining Pregnancy and Exercise.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">PloS One</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 7 (7): e42297.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/jNPA" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Fittschen, M., I. Lastres-Becker, M. V. Halbach, E. Damrath, S. Gispert, M. Azizov, M. Walter, S. Müller, and G. Auburger. 2015. “Genetic Ablation of Ataxin-2 Increases Several Global Translation Factors in Their Transcript Abundance but Decreases Translation Rate.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Neurogenetics</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 16 (3): 181–92.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/CeJC" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Fraineau, Sylvain, Carmen G. Palii, David S. Allan, and Marjorie Brand. 2015. “Epigenetic Regulation of Endothelial-Cell-Mediated Vascular Repair.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">The FEBS Journal</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 282 (9): 1605–29.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/4tOs" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Fuchs, T. C., A. Mally, A. Wool, M. Beiman, and P. Hewitt. 2014. “An Exploratory Evaluation of the Utility of Transcriptional and Urinary Kidney Injury Biomarkers for the Prediction of Aristolochic Acid-Induced Renal Injury in Male Rats.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Veterinary Pathology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 51 (3): 680–94.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/e8EC" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Holmgren, Gustav, Jane Synnergren, Yalda Bogestål, Caroline Améen, Karolina Åkesson, Sandra Holmgren, Anders Lindahl, and Peter Sartipy. 2015. “Identification of Novel Biomarkers for Doxorubicin-Induced Toxicity in Human Cardiomyocytes Derived from Pluripotent Stem Cells.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Toxicology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 328 (February): 102–11.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/v84m" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">James, Allison, Yubo Wang, Himanshu Raje, Raphyel Rosby, and Patrick DiMario. 2014. “Nucleolar Stress with and without p53.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Nucleus </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">&nbsp;5 (5): 402–26.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/dtdR" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Johnson, Keven R., Jessie Nicodemus-Johnson, Mathew J. Spindler, and Graeme K. Carnegie. 2015. “Genome-Wide Gene Expression Analysis Shows AKAP13-Mediated PKD1 Signaling Regulates the Transcriptional Response to Cardiac Hypertrophy.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">PloS One</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 10 (7): e0132474.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/aKw3" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Kelpke, Stacey S., Bo Chen, Kelley M. Bradley, Xinjun Teng, Phillip Chumley, Angela Brandon, Brett Yancey, et al. 2012. “Sodium Nitrite Protects against Kidney Injury Induced by Brain Death and Improves Post-Transplant Function.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Kidney International</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 82 (3): 304–13.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/Ax3q" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Ky, Bonnie, Mary Putt, Heloisa Sawaya, Benjamin French, James L. Januzzi Jr, Igal A. Sebag, Juan Carlos Plana, et al. 2014. “Early Increases in Multiple Biomarkers Predict Subsequent Cardiotoxicity in Patients with Breast Cancer Treated with Doxorubicin, Taxanes, and Trastuzumab.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Journal of the American College of Cardiology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 63 (8): 809–16.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/X2lm" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Lin, Zhiqing, Li Ma, Zhu-Ge X, Huashan Zhang, and Bencheng Lin. 2013. “A Comparative Study of Lung Toxicity in Rats Induced by Three Types of Nanomaterials.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Nanoscale Research Letters</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 8 (1): 521.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/XowW" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Liu, Yan, Aarti Asnani, Lin Zou, Victoria L. Bentley, Min Yu, You Wang, Graham Dellaire, et al. 2014. “Visnagin Protects against Doxorubicin-Induced Cardiomyopathy through Modulation of Mitochondrial Malate Dehydrogenase.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Science Translational Medicine</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 6 (266): 266ra170.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/1b8a" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Luo, Shouqing, and David C. Rubinsztein. 2009. “Huntingtin Promotes Cell Survival by Preventing Pak2 Cleavage.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Journal of Cell Science</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 122 (Pt 6): 875–85.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/Waup" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Maragh, Samantha, Ronald A. Miller, Seneca L. Bessling, David M. McGaughey, Marja W. Wessels, Bianca de Graaf, Eric A. Stone, et al. 2011. “Identification of RNA Binding Motif Proteins Essential for Cardiovascular Development.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">BMC Developmental Biology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 11 (October): 62.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/jeVO" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Menna, Pierantonio, Emanuela Salvatorelli, and Giorgio Minotti. 2008. “Cardiotoxicity of Antitumor Drugs.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Chemical Research in Toxicology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 21 (5): 978–89.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/yZ8x" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Mishra, Pratibha, Sarvendra Vikram Singh, Ajeet Kumar Verma, Pallavi Srivastava, Sarwat Sultana, and Srikanta Kumar Rath. 2014. “Rosiglitazone Induces Cardiotoxicity by Accelerated Apoptosis.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Cardiovascular Toxicology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 14 (2): 99–119.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/djwF" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Pinto, Navin R., Mark A. Applebaum, Samuel L. Volchenboum, Katherine K. Matthay, Wendy B. London, Peter F. Ambros, Akira Nakagawara, et al. 2015. “Advances in Risk Classification and Treatment Strategies for Neuroblastoma.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Journal of Clinical Oncology: Official Journal of the American Society of Clinical Oncology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 33 (27): 3008–17.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/CPUE" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Rau, Christoph D., Jessica Wang, Rozeta Avetisyan, Milagros C. Romay, Lisa Martin, Shuxun Ren, Yibin Wang, and Aldons J. Lusis. 2015. “Mapping Genetic Contributions to Cardiac Pathology Induced by Beta-Adrenergic Stimulation in Mice.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Circulation. Cardiovascular Genetics</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 8 (1): 40–49.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/Sx0j" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Reue, Karen. 2009. “The Lipin Family: Mutations and Metabolism.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Current Opinion in Lipidology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 20 (3): 165–70.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/80hJ" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Richani, Dulama, Melanie L. Sutton-McDowall, Laura A. Frank, Robert B. Gilchrist, and Jeremy G. Thompson. 2014. “Effect of Epidermal Growth Factor-like Peptides on the Metabolism of in Vitro- Matured Mouse Oocytes and Cumulus Cells.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Biology of Reproduction</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 90 (3): 49.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/iXHc" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Salabei, Joshua K., and Daniel J. Conklin. 2013. “Cardiovascular Autophagy: Crossroads of Pathology, Pharmacology and Toxicology.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Cardiovascular Toxicology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 13 (3): 220–29.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/FEnS" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Sawicki, Konrad Teodor, Hsiang-Chun Chang, and Hossein Ardehali. 2015. “Role of Heme in Cardiovascular Physiology and Disease.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Journal of the American Heart Association</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 4 (1): e001138.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/sEjA" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Schuldt, A. J. T., T. J. Hampton, V. Chu, C. A. Vogler, N. Galvin, M. D. Lessard, and J. E. Barker. 2004. “Electrocardiographic and Other Cardiac Anomalies in Beta-Glucuronidase-Null Mice Corrected by Nonablative Neonatal Marrow Transplantation.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Proceedings of the National Academy of Sciences of the United States of America</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 101 (2): 603–8.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/wZWB" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Schultz, Luise, Marie-Gabrielle Zurich, Maxime Culot, Anaelle da Costa, Christophe Landry, Patricia Bellwon, Theresa Kristl, et al. 2015. “Evaluation of Drug-Induced Neurotoxicity Based on Metabolomics, Proteomics and Electrical Activity Measurements in Complementary CNS in Vitro Models.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Toxicology in Vitro: An International Journal Published in Association with BIBRA</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 30 (1 Pt A): 138–65.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/8EDS" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Shi, Y., S. Katsev, C. Cai, and S. Evans. 2000. “BMP Signaling Is Required for Heart Formation in Vertebrates.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Developmental Biology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 224 (2): 226–37.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/3nBT" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Sourdon, Joevin, Franck Lager, Thomas Viel, Daniel Balvay, Rebecca Moorhouse, Evangeline Bennana, Gilles Renault, Pierre-Louis Tharaux, Neeraj Dhaun, and Bertrand Tavitian. 2017. “Cardiac Metabolic Deregulation Induced by the Tyrosine Kinase Receptor Inhibitor Sunitinib Is Rescued by Endothelin Receptor Antagonism.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Theranostics</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 7 (11): 2757–74.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/TEyK" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Staubach, Simon, Stefan Müller, Murat Pekmez, and Franz-Georg Hanisch. 2017. “Classical Galactosemia: Insight into Molecular Pathomechanisms by Differential Membrane Proteomics of Fibroblasts under Galactose Stress.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Journal of Proteome Research</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 16 (2): 516–27.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/3CBu" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Tanpure, Smita, Jerusha Boyineini, Manu Gnanamony, Reuben Antony, Karen S. Fernández, Jaime Libes, Julian Lin, David Pinson, Pushpa A. Joseph, and Christopher S. Gondi. 2017. “SPARC Overexpression Suppresses Radiation-Induced HSP27 and Induces the Collapse of Mitochondrial Δψ in Neuroblastoma Cells.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Oncology Letters</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 13 (6): 4602–10.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/yrCi" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Tough, David F., Huw D. Lewis, Inmaculada Rioja, Matthew J. Lindon, and Rab K. Prinjha. 2014. “Epigenetic Pathway Targets for the Treatment of Disease: Accelerating Progress in the Development of Pharmacological Tools: IUPHAR Review 11.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">British Journal of Pharmacology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 171 (22): 4981–5010.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/9ncs" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Van Vliet, Patrick, Sean M. Wu, Stéphane Zaffran, and Michel Pucéat. 2012. “Early Cardiac Development: A View from Stem Cells to Embryos.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Cardiovascular Research</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 96 (3): 352–62.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/uBNK" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Wnęk, Małgorzata, Lorenzo Ressel, Emanuele Ricci, Carmen Rodriguez-Martinez, Julio Cesar Villalvazo Guerrero, Zarini Ismail, Colin Smith, et al. 2016. “Herpes Simplex Encephalitis Is Linked with Selective Mitochondrial Damage; a Post-Mortem and in Vitro Study.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Acta Neuropathologica</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 132 (3): 433–51.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/vTXo" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Wu, L. Q., Y. F. Yang, D. Zheng, H. Deng, Q. Pan, T. L. Zhao, F. Cai, et al. 2004. “Confirmation and Refinement of a Genetic Locus for Disseminated Superficial Actinic Porokeratosis (DSAP1) at 12q23.2-24.1.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">The British Journal of Dermatology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 150 (5): 999–1004.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/Qwqm" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Xi, Lei. 2016. “Visnagin-a New Protectant against Doxorubicin Cardiotoxicity? Inhibition of Mitochondrial Malate Dehydrogenase 2 (MDH2) and beyond.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Annals of Translational Medicine</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 4 (4): 65.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/drqW" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Yang, J., M. Boerm, M. McCarty, C. Bucana, I. J. Fidler, Y. Zhuang, and B. Su. 2000. “Mekk3 Is Essential for Early Embryonic Cardiovascular Development.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Nature Genetics</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 24 (3): 309–13.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/SQuB" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Yang, Yunan, Reinier Hernandez, Jun Rao, Li Yin, Yazhuo Qu, Jinrong Wu, Christopher G. England, et al. 2015. “Targeting CD146 with a 64Cu-Labeled Antibody Enables in Vivo immunoPET Imaging of High-Grade Gliomas.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Proceedings of the National Academy of Sciences of the United States of America</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 112 (47): E6525–34.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/C0zJ" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Yu, Yu, Jing Qin, Di Chen, Hui Wang, Junwen Wang, and Ying Yu. 2015. “Chronic Cardiovascular Disease-Associated Gene Network Analysis in Human Umbilical Vein Endothelial Cells Exposed to 2,3,7,8-Tetrachlorodibenzo-P-Dioxin.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Cardiovascular Toxicology</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 15 (2): 157–71.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/FHxy" style="text-decoration-line: none;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Zhang, Zhenghua, Zhenmin Niu, Wentao Yuan, Weida Liu, Leihong Xiang, Jing Zhang, Xun Chu, et al. 2004. “Fine Mapping and Identification of a Candidate Gene SSH1 in Disseminated Superficial Actinic Porokeratosis.” </span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Human Mutation</span><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 24 (5): 438.</span></a>
</p>
<p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:12pt;text-indent: -24pt;padding:0pt 0pt 0pt 24pt;"><a href="http://paperpile.com/b/hBQxRz/lfKx" style="text-decoration-line: none;"><span style="text-decoration-line: none; font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Zordoky, Beshay N. M., Anwar Anwar-Mohamed, Mona E. Aboutabl, and Ayman O. S. El-Kadi. 2011. “Acute Doxorubicin Toxicity Differentially Alters Cytochrome P450 Expression and Arachidonic Acid Metabolism in Rat Kidney and Liver.” </span><span style="text-decoration-line: none; font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; font-style: italic; vertical-align: baseline; white-space: pre-wrap;">Drug Metabolism and Disposition: The Biological Fate of Chemicals</span><span style="text-decoration-line: none; font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"> 39 (8): 1440–50.</span></a>
</p>
<p><br>
</p></span>
          
          <hr/>
 

</div>  


<div style='min-height:500px'></div>
            
          </div>
          
          
          
        </div>
      </div>
      
    </div>
    <div class="bar" style="heigh:100%">
      <div class="inner h100" style="heigh:100%">
        <div class="top"><img src="/static/images/logo.svg"></div>
        <div class="menu">
          <ul class="list-unstyled" style='margin-top: 0;'>
            <li class="title">Table of contents: </li>
            <!-- 
            {%if report.input_data_text%}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#input_data">Input data</a></li>
            {%endif%}
            {%if report.methods_text%}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#methods">Methods</a></li>
            {%endif%}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#gene_analysis">Gene level analysis</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#path_analysis">Pathway analysis</a></li>
            {%if report.conclusion_text%}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#conclusions">Conclusions</a></li>
            {%endif%}
            {%if report.references_text%}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#references">References</a></li>
            {%endif%}
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#input_data">Input data</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#dl_analysis">Deep Learning</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#kinetic_analysis">Kinetic pathway modeling</a></li>
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#sim_analysis">Similarity</a></li>
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#cor_analysis">Public domain screening</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#pt_analysis">Scoring of potential targets</a></li>
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#conclusions">Discussion</a></li>
             -->
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#gene_analysis">Gene level analysis</a></li>
            <!-- <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tf_analysis">Transcription Factor Analysis</a></li>
            -->
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#path_analysis">Pathway Analysis</a></li>
            
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#dl_analysis">Drug Repurposing</a></li>
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#biomap">BioMAP inference</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#biomap_l1000">L1000-BioMAP correlation</a></li>            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#hit">Hit qualification</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tox">Tox groups analysis</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tox_analysis">Toxcast analysis</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#sim_analysis">Similarity to LINCS dataset </a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#barcodes">Barcodes for L1000 data</a></li>
            {%if 'gsk' in report.slug %}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#conclusions">Discussion</a></li>
            {%endif%}
          </ul>
          <br/><br/>
        </div>
        <div class="bottom">
          <div class="inner"> Insilico Medicine Inc., Johns Hopkins <br/>University Eastern, ETC B301, 1101 East <br/>33rd Street, Baltimore, MD 21218 </div>
        </div>
      </div>
    </div>
    
      
    <script src="https://cdn.datatables.net/1.10.10/js/jquery.dataTables.min.js" type="text/javascript"></script>
    <script src="https://cdn.datatables.net/1.10.10/js/dataTables.bootstrap.min.js" type="text/javascript"></script>
    <script src="https://cdn.datatables.net/buttons/1.1.0/js/dataTables.buttons.min.js" type="text/javascript"></script>
    <script src="https://cdn.datatables.net/buttons/1.1.0/js/buttons.bootstrap.min.js" type="text/javascript"></script>
    <script src="//cdn.datatables.net/buttons/1.1.0/js/buttons.flash.min.js" type="text/javascript"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/jszip/2.5.0/jszip.min.js" type="text/javascript"></script>
    <script src="//cdn.rawgit.com/bpampuch/pdfmake/0.1.18/build/pdfmake.min.js" type="text/javascript"></script>
    <script src="//cdn.rawgit.com/bpampuch/pdfmake/0.1.18/build/vfs_fonts.js" type="text/javascript"></script>
    <script src="//cdn.datatables.net/buttons/1.1.0/js/buttons.html5.min.js" type="text/javascript"></script>
    <script src="//cdn.datatables.net/buttons/1.1.0/js/buttons.print.min.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/venn.js/venn.js" type="text/javascript"></script>
    
    
    
    <script src="/static/report/portal/js/opentip.js" charset="utf-8"></script>
    <script src="/static/report/bt/js/adapter-native.js" charset="utf-8"></script>
    <script src="/static/report/portal/js/volcano.js" type="text/javascript"></script>biomap
    <script src="/static/report/portal/js/scatter.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/tables.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/boxplot.js" type="text/javascript"></script>    
    <script src="/static/report/portal/js/venn-diagram.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/tf.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/deeplearning.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/correlation.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/ds.js" type="text/javascript"></script>
    
    
    
    <!-- MODAL FOR PATHS  -->
    <div id="pathmodal" class="modal fade large" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true" >
 <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;  </button>
        <h4 class="modal-title" id="myModalLabel">Modal title</h4>
      </div>
      <div class="modal-body" id='modalBody'>
        
      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
      </div>
    </div>
  </div>
</div>
    
<script>

$(function() {
	
	
	$("#btn_show_genes").click(function(){
		
		$(this).hide();
		$("#hide_genes").slideDown('slow');
		 
		{%for group in report.genegroup_set.all%}
		  {%if forloop.counter = 1%}
		
		     drawVolcanoPlot('{{group.get_slug}}_volcano', {{report.id}}, '{{group.doc_logfc}}');
	         drawGeneTable({{report.id}}, 'tbl-{{group.get_slug}}', '{{group.doc_logfc}}', categories='{{group.get_slug}}'); 
	     
	       {%endif%}
	     {%endfor%}
	     
	     drawGeneTable({{report.id}}, 'tbl-gene_all', 'all', categories='{{report.get_all_gene_groups_norm_as_list}}');
	 	 
	     
	   //GENES
	 	{%for perm in report.get_gene_permutations%}
	 	
	 	$("#venn_boot_gene_{{perm.strip}}").click(function(){
	 		drawVenn({{report.id}},
	 				'venn_gene-{{perm.strip}}-up',
	 				'{{perm.strip}}',	 
	 				false, 'up', 'venn_gene_dinamic_{{perm.strip}}', 'genes', categories='{{report.get_all_gene_groups_norm_as_list}}');
	 		drawVenn({{report.id}},
	 				'venn_gene-{{perm.strip}}-down',
	 				'{{perm.strip}}',			
	 				false, 'down', 'venn_gene_dinamic_{{perm.strip}}', 'genes', categories='{{report.get_all_gene_groups_norm_as_list}}');
	 		
	 	});
	 	
	 	{%endfor%}
	 	
	 	
	 	{%for gene in report.get_notable_genes_list%}
		  drawGeneBoxplot(reportID={{report.id}}, name='{{gene}}', renderTo='{{gene}}', gene='{{gene}}', categories='{{report.get_all_gene_groups_norm_as_list}}');
		{%endfor%}
		
	});
	
	{%if report.gene_plot == 'vulcano'%}
	  {%for group in report.genegroup_set.all%}
	  
	  {%if forloop.counter = 1%}
	     
	      
	  {%else%}
	  $('#vol_boot_{{group.get_slug}}').click(function(){
			
			if($("#{{group.get_slug}}_volcano span").html() == 'Loading plot...'){
				drawVolcanoPlot('{{group.get_slug}}_volcano', {{report.id}}, '{{group.doc_logfc}}');
			}
			if ( !$.fn.dataTable.isDataTable( '#tbl-{{group.get_slug}}' ) ) {
				 drawGeneTable({{report.id}}, 'tbl-{{group.get_slug}}', '{{group.doc_logfc}}', categories='{{group.get_slug}}'); 
				}
		   });
	  
	  {%endif%}
	  
	  
	  {%endfor%}
	{%endif%}
	{%if  report.gene_plot == 'scatter'%}
	{%for group in report.genegroup_set.all%}
	  drawGeneScatter(name='{{group.name}} VS {{report.norm_name}}',
			          renderTo='{{group.get_slug}}_volcano',
			          xname='log2({{group.name}} Expression level)',
			          yname='log2({{report.norm_name}} Expression level)',
			          file_name='{{group.document}}',
			          reportID={{report.id}});
	  drawGeneTableScatter({{report.id}}, 'tbl-{{group.get_slug}}', '{{group.doc_logfc}}', categories='{{report.get_all_gene_groups_norm_as_list}}'); 
	{%endfor%}
	{%endif%}
	
	
	
	
	
	
	
///////////// SIGNALING PATHs

$("#btn_show_path").click(function(){
	
	$(this).hide();
	$("#hide_path").slideDown('slow');
	
	drawPathwayTable({{report.id}},
			'tbl-path_all', 
	         'all', 
	         'all', false, '{{report.organism}}');
	
	
	drawPathwayTable({{report.id}},
			'tbl-path_tox', 
	         'tox', 
	         'tox', false, '{{report.organism}}');
	
});
	

	
	{%for perm in report.get_path_permutations%}
	
	$("#venn_boot_path_{{perm.strip}}").click(function(){
	
	drawVenn({{report.id}},
		'venn_path-{{perm.strip}}-up',
		'{{perm.strip}}',
		false, 'up', 'venn_path_dinamic_{{perm.strip}}', 'pathways', '{{report.organism}}');
	drawVenn({{report.id}},
			'venn_path-{{perm.strip}}-down',
			'{{perm.strip}}',
			false, 'down', 'venn_path_dinamic_{{perm.strip}}', 'pathways', '{{report.organism}}');
	});
    {%endfor%}
    
/////////////////// Transcription Factor
$("#btn_show_tf").click(function(){
	$(this).hide();
	$("#hide_tf").slideDown('slow');
	
	{%for group in report.tfgroup_set.all%}
    
      {%if forloop.counter = 1%}   
        drawTFTable({{report.id}}, 'tbl-tf-{{group.get_slug}}', '{{group.document}}', categories='{{group.name}}'); 
        drawTFTrrustTable({{report.id}}, 'tbl-tftrrust-{{group.get_slug}}', '{{group.trrust}}', categories='{{group.name}}'); 
      {%endif%}
    {%endfor%}
	
});

    {%for group in report.tfgroup_set.all%}
    
        {%if forloop.counter = 1%}        
          

        {%else%}
        
        $('#tf_boot_{{group.get_slug}}').click(function(){
        	
        	if ( !$.fn.dataTable.isDataTable( '#tbl-tf-{{group.get_slug}}' ) ) {
        		drawTFTable({{report.id}}, 'tbl-tf-{{group.get_slug}}', '{{group.document}}', categories='{{group.name}}'); 
				}
        	
        	
        });
        
        $('#tftrrust_boot_{{group.get_slug}}').click(function(){
        	if ( !$.fn.dataTable.isDataTable( '#tbl-tftrrust-{{group.get_slug}}' ) ) {
        		drawTFTrrustTable({{report.id}}, 'tbl-tftrrust-{{group.get_slug}}', '{{group.trrust}}', categories='{{group.name}}'); 
				}
        	
        });
        
        
        {%endif%}
    
    
    {%endfor%}

/////////////////// Drug Scoring
    $("#btn_show_ds").click(function(){
    	$(this).hide();
    	$("#hide_ds").slideDown('slow');
    	
    	{%for group in report.drugscoregroup_set.all%}
        
          {%if forloop.counter = 1%}   
            drawDSTable({{report.id}}, 'tbl-ds-{{group.get_slug}}', '{{group.document}}', categories='{{group.name}}'); 
            drawDSBarplot({{report.id}}, '{{group.get_slug}}_dsbarmolecule_plot', 
            		'{{group.document}}', 
            		categories='{{group.name}}',
            		pert_type='molecule');
            drawDSBarplot({{report.id}}, '{{group.get_slug}}_dsbargene_plot', 
            		'{{group.document}}', 
            		categories='{{group.name}}',
            		pert_type='gene');
          {%endif%}
        {%endfor%}
    	
    });

        {%for group in report.drugscoregroup_set.all%}
        
            {%if forloop.counter = 1%}        
              

            {%else%}
            
            $('#ds_boot_{{group.get_slug}}').click(function(){
            	
            	if ( !$.fn.dataTable.isDataTable( '#tbl-ds-{{group.get_slug}}' ) ) {
            		drawDSTable({{report.id}}, 'tbl-ds-{{group.get_slug}}', '{{group.document}}', categories='{{group.name}}'); 
    				}
            	
            	
            });
            
            $('#dsbarmolecule_boot_{{group.get_slug}}').click(function(){
    			
    			if($("#{{group.get_slug}}_dsbarmolecule_plot span").html() == 'Loading Bar Plot...'){
    				drawDSBarplot({{report.id}}, '{{group.get_slug}}_dsbarmolecule_plot', 
    	            		'{{group.document}}', 
    	            		categories='{{group.name}}',
    	            		pert_type='molecule');
    			}
    			});
            
           $('#dsbargene_boot_{{group.get_slug}}').click(function(){
    			
    			if($("#{{group.get_slug}}_dsbargene_plot span").html() == 'Loading Bar Plot...'){
    				drawDSBarplot({{report.id}}, '{{group.get_slug}}_dsbargene_plot', 
    	            		'{{group.document}}', 
    	            		categories='{{group.name}}',
    	            		pert_type='gene');
    			}
    			});
            
            //$('#tftrrust_boot_{{group.get_slug}}').click(function(){
            //	if ( !$.fn.dataTable.isDataTable( '#tbl-tftrrust-{{group.get_slug}}' ) ) {
            //		drawTFTrrustTable({{report.id}}, 'tbl-tftrrust-{{group.get_slug}}', '{{group.trrust}}', categories='{{group.name}}'); 
    		//		}
            //	
            //});
            
            
            {%endif%}
        
        
        {%endfor%}





    

    
 /* METABOLIC PATHs
  drawPathwayTable({{report.id}},
		  'tbl-meta_all', 
	         'all', 
	         'all', true, '{{report.organism}}');
 
	{%for perm in report.get_path_permutations%}
	drawVenn({{report.id}},
		'venn_meta-{{perm.strip}}-up',
		'{{perm.strip}}',
		true, 'up', 'venn_meta_dinamic_{{perm.strip}}', 'pathways', '{{report.organism}}');
	drawVenn({{report.id}},
			'venn_meta-{{perm.strip}}-down',
			'{{perm.strip}}',
			true, 'down', 'venn_meta_dinamic_{{perm.strip}}', 'pathways', '{{report.organism}}');
    {%endfor%}
*/
});

</script>
  </body>
</html>