<!DOCTYPE html>
<html lang="en">
  <head>
    <title> {{report.title_short}} | Insilico Medicine</title>
    <link href="/static/bower_components/bootstrap-stylus/dist/bootstrap.min.css" rel="stylesheet" type="text/css">
    <script src="//code.jquery.com/jquery-2.1.4.min.js" type="text/javascript"></script>
    <script src="/static/js/oncofinder2.js" type="text/javascript"></script>
    <script src="/static/bower_components/bootstrap-stylus/dist/bootstrap.min.js" type="text/javascript"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore.js" type="text/javascript"></script>
    <script src="/static/js/d3.v3.min.js" type="text/javascript"></script>
    <script src="https://cdn.datatables.net/1.10.10/js/jquery.dataTables.min.js" type="text/javascript"></script>
    <script src="http://code.highcharts.com/highcharts.js" type="text/javascript"></script>
    <script src="https://code.highcharts.com/highcharts-more.js"  type="text/javascript"></script>
    <script src="https://code.highcharts.com/highcharts-3d.js"  type="text/javascript"></script>
    <script src="http://code.highcharts.com/modules/exporting.js"  type="text/javascript"></script>
    <link href="/static/css/report_style.css" rel="stylesheet" type="text/css">
    <link href="https://cdn.datatables.net/1.10.10/css/dataTables.bootstrap.min.css" rel="stylesheet" type="text/css">
    <link href="https://cdn.datatables.net/buttons/1.0.3/css/buttons.dataTables.min.css" rel="stylesheet">
    
    <link rel="shortcut icon" href="/static/images/favicon.ico" type="image/x-icon">
    <link rel="icon" href="/static/images/favicon.ico" type="image/x-icon">
    
    <link href="/static/report/bt/js/opentip.css" rel="stylesheet" type="text/css">
    <link href="/static/report/bt/js/volcano.css" rel="stylesheet" type="text/css">
    
  </head>
  <body style='font-size: 14px;'>
    <div class="content h100">
      <div class="main">
        <div class="inner">
         <div>
         <span class='pull-left' style='font-size:11pt; margin:15px;'>
         <img src='/static/images/insilico.png' style="width:40px;"/>
         Insilico Medicine - {{report.title_short}}</span>
         <span class='pull-right' style='font-size:11pt; margin:15px;'>Access Level: Z-II, <br/>Confidential </span>
         </div><br style="clear:both;"/>
          <div class='text-center'>
            <h1>{{report.title}}</h1>
          </div>
          
          
          <div style="margin: 50px 0 0 50px; padding: 0 50px 0 0;">
            <h4 style='font-weight:bold; font-size:12pt;'>Organization: </h4>
            <span style="font-size:13pt">{{report.organization}}</span>
            <br/><br/>
            <!-- <h4 style='font-weight:bold; font-size:12pt;'>Data Quality Assessment: </h4>-->
 
            
            
            <ul class="list-unstyled" style='font-size: 11pt; margin: 50px 0 0 0;' >
            <li class="title">Table of contents: </li>

            
            <!--<li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#input_data">Input data</a></li>
              <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#dl_analysis">Deep Learning</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#kinetic_analysis">Kinetic pathway modeling</a></li>-->
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#gene_analysis">Gene level analysis</a></li>
            <!--  <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tf_analysis">Transcription Factor Analysis</a></li>-->
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#path_analysis">Pathway Analysis</a></li>
            {%if 'above-and-beyond' in report.slug %}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#ds_analysis">Drug Scoring</a></li>
            {%endif%}
            {%if 'l1k' in report.slug %}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#dl_analysis">Drug Repurposing</a></li>
            {%endif%}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#biomap">BioMAP inference</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#hit">Hit qualification</a></li>  
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tox">Tox groups analysis</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#sim_analysis">Similarity to LINCS dataset</a></li>
            {%if 'gsk' in report.slug %}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#conclusions">Discussion</a></li>
            {%endif%}
            <!--  
            
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#cor_analysis">Public domain screening</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#pt_analysis">Scoring of potential targets</a></li>
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#conclusions">Discussion</a></li>
            -->
          </ul>
          
          {%if report.input_data_text%}
          <h2 id="input_data">Experimental design and RNA-seq data analysis</h2>
          <hr/>
          {{report.input_data_text|safe}}

          {%endif%}
          

{%if 'dmso' in report.slug %}
<center>
<img src='/media/report-portal/gsk_prj3_dmso/prj3_dmso.png' style="width:800px"/>
</center>
{%endif%}
{%if 'prj3_l1k' in report.slug %}
<center>
<img src='/media/report-portal/gsk_prj3_l1k/prj3_l1k.png' style="width:800px"/>
</center>
<h4>PCA gene weights</h4>
<table id="tbl-top_l1k" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr >
           <th>Probeset</th>
           <th>Symbol</th>
           <th>Comp1</th>
           <th>Comp2</th>
           <th>Comp3</th>
          </tr>
        </thead>  
      </table>



{%endif%}
{%if 'prj3_mm' in report.slug %}
<center>
<img src='/media/report-portal/gsk_prj3_mm/prj3_mm.png' style="width:800px"/>
</center>
{%endif%}
{%if 'nocmp' in report.slug %}
<center>
<img src='/media/report-portal/gsk_prj3_nocmp/prj3_nocmp.png' style="width:800px"/>
</center>
{%endif%}

<!-------------------------------------  GENE level  -->          
          <hr/>
          <h2 id='gene_analysis'>Gene level analysis</h2>
          <hr/>
          {%if report.gene_text%}
          {{report.gene_text|safe}}
          <br/><br/>
          {%endif%}


<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_genes'>Show Data</button>
</center>

<div id='hide_genes' style='display:none;'> <!-- Div to hide and load on button click -->
{%if 'above-and-beyond' in report.slug %}
<center>
<img src='/media/report-portal/above-and-beyond/gene1.png' />
<h4>PCA plot with outlier</h4>
<br/>
<img src='/media/report-portal/above-and-beyond/gene2.png' />
<h4>PCA plot without outlier</h4>
</center>
{%endif%}

           
   <!-- GENE Nav tabs -->      
   <ul class="nav nav-tabs" role="tablist">
     {%for group in report.genegroup_set.all%}
       <li role="presentation" {%if forloop.counter == 1 %}class="active"{%endif%}>
         <a href="#{{group.get_slug}}_tab" aria-controls="home1" role="tab" data-toggle="tab" id='vol_boot_{{group.get_slug}}'>
           {{group.name}} vs {{report.norm_name}}
         </a>
       </li>
     {%endfor%}
   </ul>
         
   <div class="tab-content">
    {%for group in report.genegroup_set.all%}
     {%if forloop.counter == 1 %}
     <div role="tabpanel" class="tab-pane fade in active" id="{{group.get_slug}}_tab">
     {%else%}
     <div role="tabpanel" class="tab-pane fade " id="{{group.get_slug}}_tab">
     {%endif%}
       <div id="{{group.get_slug}}_volcano" 
            style="min-width: 800px; height: 400px; max-width: 800px; margin: 5px auto">
            <span>Loading plot...</span>
       </div> <!-- Volcano plot div -->
       
       
       <h4>Table <span class='tbl_num'></span>. {{group.name}} vs {{report.norm_name}}</h4>
       <table id="tbl-{{group.get_slug}}" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Gene</th>
           <th>log2FoldChange</th>
           {%if 'nova' in report.slug %}
           <th>p-value</th>
           {%else%}
           <th>adj. p-value</th>
           {%endif%}
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
    {%endfor%}
   
   </div> <!--  end of tab-content div -->
   
<!-- -------------------- GENE VENN  -->
<br/>
<h3>Gene level comparison</h3>
<br/>

<!-- Gene level comparison Nav tabs -->
<ul class="nav nav-tabs" role="tablist">
         <li role="presentation" class="active"><a href="#gene_all_tab" aria-controls="settings" role="tab" data-toggle="tab">ALL</a></li>
         {%for perm in report.get_gene_permutations%}
         <li role="presentation" ><a href="#gene_{{perm.strip}}_tab" aria-controls="settings" role="tab" data-toggle="tab" id='venn_boot_gene_{{perm.strip}}'>{{perm}}</a></li>
         {%endfor%}
</ul>

<!-- GEne VEnn Tab panes -->
  <div class="tab-content">
  
  <div role="tabpanel" class="tab-pane fade in active " id="gene_all_tab">     <br/> 
    <h4>Table . All genes</h4>
    <table id="tbl-gene_all" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Gene</th>
           {%for group in report.get_all_gene_groups_as_list%}
            <!--  {% if forloop.counter > 1 %}<th>{{group}}</th> {%endif%}-->
            <th>{{group}}</th>
           {%endfor%}
          </tr>
        </thead>  
      </table>       
   <br/>
  
  </div>
  
  {%for perm in report.get_gene_permutations%}
  <div role="tabpanel" class="tab-pane fade " id="gene_{{perm.strip}}_tab">     <br/> 
    <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_gene-{{perm.strip}}-up" style="min-width: 450px; height: 400px; max-width: 450px; margin: 0 auto">Loading diagram...</div> 
       Significantly upregulated genes
       </td>
       <td>
       <div id="venn_gene-{{perm.strip}}-down" style="min-width: 450px; height: 400px; max-width: 450px; margin: 0 auto">Loading diagram...</div> 
       Significantly downregulated genes
       </td>
      </tr>
    </table>        
   <br/>
  <div id="venn_gene_dinamic_{{perm.strip}}" class="" style="max-width:1000px;"></div> <!--  div to place dinamic tables from Venn clicks -->
  </div>
  {%endfor%}
  
  </div>
<hr/> 
{%if report.notable_genes%}
   <h3>Notable genes</h3>
<hr/>


 <!-- Notable GENE Nav tabs -->
         <ul class="nav nav-tabs" role="tablist">
           {%for gene in report.get_notable_genes_list%}
           <li role="presentation" {%if forloop.counter == 1 %}class="active"{%endif%}><a href="#{{gene}}_tab" aria-controls="home1" role="tab" data-toggle="tab">{{gene}}</a></li>
           {%endfor%}
         </ul>

   <div class="tab-content">
    {%for gene in report.get_notable_genes_list%}
     {%if forloop.counter == 1 %}
     <div role="tabpanel" class="tab-pane fade in active" id="{{gene}}_tab">
     {%else%}
     <div role="tabpanel" class="tab-pane fade " id="{{gene}}_tab">
     {%endif%}
     <div id="{{gene}}" style="min-width: 800px; height: 400px; width: 1100px; margin: 0 auto">Loading box plot...</div>
   
     </div>
    {%endfor%}
   
   </div> <!--  end of tab panel -->
{%endif%}

</div> <!--  end of hide div -->


<!-- ---------------------------------Transcription Factor--------------------------------------------- 

 
 <hr/> 
<h2 id='tf_analysis'>Transcription Factor Analysis</h2>
<hr/>

{%if report.path_text%}
          {{report.tf_text|safe}}
          <br/><br/>
{%endif%}

<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_tf'>Show Data</button>
</center>

<div id='hide_tf' style='display:none;'> <!-- Div to hide and load on button click 

   <!-- TF Nav tabs       
   <ul class="nav nav-tabs" role="tablist">
     {%for group in report.genegroup_set.all%}
       <li role="presentation" {%if forloop.counter == 1 %}class="active"{%endif%}>
         <a href="#{{group.get_slug}}_tf_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_{{group.get_slug}}'>
           {{group.name}}
         </a>
       </li>
     {%endfor%}
   </ul>
         
   <div class="tab-content">
    {%for group in report.genegroup_set.all%}
     {%if forloop.counter == 1 %}
     <div role="tabpanel" class="tab-pane fade in active" id="{{group.get_slug}}_tf_tab">
     {%else%}
     <div role="tabpanel" class="tab-pane fade " id="{{group.get_slug}}_tf_tab">
     {%endif%}
             
       <h4>Table <span class='tbl_num'></span>Transcription Factor (FANTOM5 database) for {{group.name}} </h4>
       <table id="tbl-tf-{{group.get_slug}}" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Factor</th>
           <th>underTF_DEG</th>
           <th>underTF_nonDEG</th>
           <th>notunderTF_DEG</th>
           <th>notunderTF_nonDEG</th>
           <th>p-value</th>
           <th>OR</th>
           <th>adj. p-value</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div 
    {%endfor%}
   
   </div> <!--  end of tab-content div 
   
   
   <!-- TF Nav tabs       
   <ul class="nav nav-tabs" role="tablist">
     {%for group in report.genegroup_set.all%}
       <li role="presentation" {%if forloop.counter == 1 %}class="active"{%endif%}>
         <a href="#{{group.get_slug}}_tftrrust_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tftrrust_boot_{{group.get_slug}}'>
           {{group.name}}
         </a>
       </li>
     {%endfor%}
   </ul>
         
   <div class="tab-content">
    {%for group in report.genegroup_set.all%}
     {%if forloop.counter == 1 %}
     <div role="tabpanel" class="tab-pane fade in active" id="{{group.get_slug}}_tftrrust_tab">
     {%else%}
     <div role="tabpanel" class="tab-pane fade " id="{{group.get_slug}}_tftrrust_tab">
     {%endif%}
             
       <h4>Table <span class='tbl_num'></span>Transcription Factor Analysis (TRRUST database) for {{group.name}} </h4>
       <table id="tbl-tftrrust-{{group.get_slug}}" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Factor</th>
           <th>underTF_DEG</th>
           <th>underTF_nonDEG</th>
           <th>notunderTF_DEG</th>
           <th>notunderTF_nonDEG</th>
           <th>p-value</th>
           <th>OR</th>
           <th>adj. p-value</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div 
    {%endfor%}
   
   </div> <!--  end of tab-content div 

</div> <!--  end of hide div 
<!-- End of TF -->

<!-- ---------------------------------PATHWAY--------------------------------------------- -->
 <hr/> 
<h2 id='path_analysis'>Pathway Analysis</h2>
<hr/>
{%if report.path_text%}
          {{report.path_text|safe}}
          <br/><br/>
          {%endif%}

<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_path'>Show Data</button>
</center>

<div id='hide_path' style='display:none;'> <!-- Div to hide and load on button click -->
         
<h3>Signaling Pathways</h3>
<hr/> 

<!-- Signaling Pathways level comparison Nav tabs -->
<ul class="nav nav-tabs" role="tablist">
         <li role="presentation" class="active"><a href="#path_all_tab" aria-controls="settings" role="tab" data-toggle="tab">ALL</a></li>
         {%for perm in report.get_path_permutations%}
         <li role="presentation" ><a href="#path_{{perm.strip}}_tab" aria-controls="settings" role="tab" data-toggle="tab" id="venn_boot_path_{{perm.strip}}">{{perm}}</a></li>
         {%endfor%}
</ul>

<!-- PAth VEnn Tab panes -->
  <div class="tab-content">
  
  <div role="tabpanel" class="tab-pane fade in active " id="path_all_tab" style='overflow-x: auto;'>     <br/> 
    <h4>Table . All signaling pathways</h4>
    <table id="tbl-path_all" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th id='pathname_th'>Pathway</th>
           {%for group in report.get_all_gene_groups_as_list%}
            <!--  {% if forloop.counter > 1 %}<th>{{group}}</th> {%endif%}-->
            <th>{{group}}</th>
           {%endfor%}
          </tr>
        </thead>  
      </table>       
   <br/>
  
  </div>
  
  {%for perm in report.get_path_permutations%}
  <div role="tabpanel" class="tab-pane fade " id="path_{{perm.strip}}_tab">     <br/> 
    <table style='text-align:center;'>
      <tr>
       <td>
       <div id="venn_path-{{perm.strip}}-up" style="min-width: 450px; height: 400px; max-width: 450px; margin: 0 auto">Loading diagram...</div> 
       Upregulated pathways
       </td>
       <td>
       <div id="venn_path-{{perm.strip}}-down" style="min-width: 450px; height: 400px; max-width: 450px; margin: 0 auto">Loading diagram...</div> 
       Downregulated pathways
       </td>
      </tr>
    </table>        
   <br/>
  <div id="venn_path_dinamic_{{perm.strip}}" class="" style="max-width:1000px;"></div> <!--  div to place dinamic tables from Venn clicks -->
  </div>
  {%endfor%}
  
  </div>
          
</div>  <!-- end of hide div -->
<!------------- End of Pathway Analysis -->


<!-- ---------------------------------Deep Learning--------------------------------------------- -->


 <hr/> 
<h2 id='dl_analysis'>Drug Repurposing</h2>

<hr/>
<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_dl'>Show Data</button>
</center>

<div id='hide_dl' style='display:none;'> <!-- Div to hide and load on button click -->

<h3>Therapeutic use</h3>
<hr/>
   <!-- DL Nav tabs -->      
   <ul class="nav nav-tabs" role="tablist">
     <!--  <li role="presentation" class="active">
         <a href="#overall_dlfc_tab" aria-controls="home1" role="tab" data-toggle="tab" id='dlfc_boot_overall'>
           Overall
         </a>
       </li>-->
     {%for group in report.genegroup_set.all%}
       <li role="presentation" >
         <a href="#{{group.get_slug}}_dlfc_tab" aria-controls="home1" role="tab" data-toggle="tab" id='dlfc_boot_{{group.get_slug}}'>
           {{group.name}}
         </a>
       </li>
     {%endfor%}
   </ul>
         
   <div class="tab-content">
     <div role="tabpanel" class="tab-pane fade in active" id="overall_dlfc_tab">
     <!--  
      <ul class="nav nav-tabs" role="tablist">
        <li role="presentation" class="active">
          <a href="#overall_dlfc_tab10"  role="tab" data-toggle="tab" id='dlfc_boot_overall10'>
          Drug 10</a>
          </li>
          <li role="presentation" >
          <a href="#overall_dlfc_tab12"  role="tab" data-toggle="tab" id='dlfc_boot_overall12'>
          Drug 12</a>
          </li>
          <li role="presentation" >
          <a href="#overall_dlfc_tab15"  role="tab" data-toggle="tab" id='dlfc_boot_overall15'>
          Drug 15</a>
          </li>
          <li role="presentation" >
          <a href="#overall_dlfc_tab29"  role="tab" data-toggle="tab" id='dlfc_boot_overall29'>
          Drug 29</a>
          </li>
      </ul>-->
      <!--  
      <div class="tab-content">
        <div role="tabpanel" class="tab-pane fade in active" id="overall_dlfc_tab10">
          <div id="dlfc_overall10" style="min-width: 800px; height: 500px; max-width: 800px; margin: 0 auto">
            <span>Loading bar plot...</span>
          </div>
        </div>
        <div role="tabpanel" class="tab-pane fade " id="overall_dlfc_tab12">
          <div id="dlfc_overall12" style="min-width: 800px; height: 500px; max-width: 800px; margin: 0 auto">
            <span>Loading bar plot...</span>
          </div>
        </div>
        <div role="tabpanel" class="tab-pane fade " id="overall_dlfc_tab15">
          <div id="dlfc_overall15" style="min-width: 800px; height: 500px; max-width: 800px; margin: 0 auto">
            <span>Loading bar plot...</span>
          </div>
        </div>
        <div role="tabpanel" class="tab-pane fade " id="overall_dlfc_tab29">
          <div id="dlfc_overall29" style="min-width: 800px; height: 500px; max-width: 800px; margin: 0 auto">
            <span>Loading bar plot...</span>
          </div>
        </div>
      
      
      </div> --><!-- end of inner tab content -->
           
                  
     
     </div>
   
    {%for group in report.genegroup_set.all%}
     
     <div role="tabpanel" class="tab-pane fade " id="{{group.get_slug}}_dlfc_tab">     
             
       <div id="dlfc_{{group.get_slug}}" style="min-width: 800px; height: 500px; max-width: 800px; margin: 0 auto">
         <span>Loading bar plot...</span>
       </div>     
     
     </div> <!-- end of tabpanel div -->
    {%endfor%}
   
   </div> <!--  end of tab-content div -->


</div>

<script>
$(function(){
	
    $("#btn_show_dl").click(function(){
		
		$(this).hide();
		$("#hide_dl").slideDown('slow');
	});
    
  //Group tabs
    {%for group in report.genegroup_set.all%}              
        
        
            	
        	
        		drawDLColumn({{report.id}}, 'dlfc_{{group.get_slug}}',
      		                 '{{group.get_slug}}', 'fc');
    		       
          
    {%endfor%}
	
	
	
});

</script>


<!-- ---------------------------------Drug Scoring--------------------------------------------- -->
{% if report.count_ds_groups > 0 %} 
 <hr/> 
<h2 id='ds_analysis'>Drug Scoring</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_ds'>Show Data</button>
</center>

<div id='hide_ds' style='display:none;'> <!-- Div to hide and load on button click -->

   <!-- DS Nav tabs -->      
   <ul class="nav nav-tabs" role="tablist">
     {%for group in report.drugscoregroup_set.all%}
       <li role="presentation" {%if forloop.counter == 1 %}class="active"{%endif%}>
         <a href="#{{group.get_slug}}_ds_tab" aria-controls="home1" role="tab" data-toggle="tab" id='ds_boot_{{group.get_slug}}'>
           {{group.name}}
         </a>
       </li>
     {%endfor%}
   </ul>
         
   <div class="tab-content">
    {%for group in report.drugscoregroup_set.all%}
     {%if forloop.counter == 1 %}
     <div role="tabpanel" class="tab-pane fade in active" id="{{group.get_slug}}_ds_tab">
     {%else%}
     <div role="tabpanel" class="tab-pane fade " id="{{group.get_slug}}_ds_tab" style='overflow-x: auto;'>
     {%endif%}
      
        
       <h4>Table <span class='tbl_num'></span>Individual perturbation scoring for {{group.name}} </h4>
       <table id="tbl-ds-{{group.get_slug}}" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr style="min-width: 900px; width: 100%;">
           <th>Perturbation</th>
           <th>Perturbation_id</th>
           <th>Time</th>
           <th>Concentration</th>
           <th>Cell line</th>
           <!--  <th>Condition</th>-->
           <th>p-val</th>
           <th>adj. p-val</th>
           <th>Effect</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
    {%endfor%}
   
   </div> <!--  end of tab-content div -->
   
   <h3>Top-10 small molecule hits</h3>
   <!-- DS Boxplot Nav tabs -->      
   <ul class="nav nav-tabs" role="tablist">
     {%for group in report.drugscoregroup_set.all%}
       <li role="presentation" {%if forloop.counter == 1 %}class="active"{%endif%}>
         <a href="#{{group.get_slug}}_dsbarmolecule_tab" aria-controls="home1" role="tab" data-toggle="tab" id='dsbarmolecule_boot_{{group.get_slug}}'>
           {{group.name}}
         </a>
       </li>
     {%endfor%}
   </ul>
         
   <div class="tab-content">
   
     {%for group in report.drugscoregroup_set.all%} 
     
     {%if forloop.counter == 1 %}
     <div role="tabpanel" class="tab-pane fade in active" id="{{group.get_slug}}_dsbarmolecule_tab">
     {%else%}
     <div role="tabpanel" class="tab-pane fade " id="{{group.get_slug}}_dsbarmolecule_tab">
     {%endif%}
     <h4>Top-10 hits for {{group.name}} </h4>
     <div id="{{group.get_slug}}_dsbarmolecule_plot" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">
     <span>Loading Bar Plot...</span>
     </div>
   
     </div>
    {%endfor%}
   
   </div> <!--  end of tab panel -->
   
<!--  DS GENE -->
<h3>Top-10 gene knockdowns</h3>
   <!-- DS Boxplot Nav tabs -->      
   <ul class="nav nav-tabs" role="tablist">
     {%for group in report.drugscoregroup_set.all%}
       <li role="presentation" {%if forloop.counter == 1 %}class="active"{%endif%}>
         <a href="#{{group.get_slug}}_dsbargene_tab" aria-controls="home1" role="tab" data-toggle="tab" id='dsbargene_boot_{{group.get_slug}}'>
           {{group.name}}
         </a>
       </li>
     {%endfor%}
   </ul>
         
   <div class="tab-content">
   
     {%for group in report.drugscoregroup_set.all%} 
     
     {%if forloop.counter == 1 %}
     <div role="tabpanel" class="tab-pane fade in active" id="{{group.get_slug}}_dsbargene_tab">
     {%else%}
     <div role="tabpanel" class="tab-pane fade " id="{{group.get_slug}}_dsbargene_tab">
     {%endif%}
     <h4>Top-10 hits for {{group.name}} </h4>
     <div id="{{group.get_slug}}_dsbargene_plot" style="min-width: 800px; height: 400px; max-width: 800px; margin: 0 auto">
     <span>Loading Bar Plot...</span>
     </div>
   
     </div>
    {%endfor%}
   
   </div> <!--  end of tab panel -->
   

</div> <!--  end of hide div -->
{%endif%}
<!-- End of DS -->

<!-- ---------------------------------Deep---------------------------------------------> 

{%if report.slug == 'gsk_prj3_l1k'%} 
 <hr/> 
<h2 id='dl_analysis'>Deep Learning</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_tf'>Show Data</button>
</center>

<div id='hide_tf' style='display:none;'> <!-- Div to hide and load on button click -->

{%if report.tf_text%}
          {{report.tf_text|safe}}
          <br/><br/>
{%endif%}

<!-- DL Nav tabs  -->     
   <ul class="nav nav-tabs" role="tablist">
     
       <li role="presentation" class="active">
         <a href="#all_models_tab" aria-controls="home1" role="tab" data-toggle="tab" >
           All Models
         </a>
       </li>
       <li role="presentation" >
         <a href="#top_featur_tab" aria-controls="home1" role="tab" data-toggle="tab" >
           Top Model Features
         </a>
        </li>
         {%if 'l1k' in report.slug %} 
         <li role="presentation">
              <a href="#top_featurfilter_tab" aria-controls="home1" role="tab" data-toggle="tab" >
               Top Model Features Filtered
             </a>
            </li>
            <li role="presentation">
              <a href="#all_inotrope_tab" aria-controls="home1" role="tab" data-toggle="tab" >
               All Models Inotrope
             </a>
            </li>
            <li role="presentation">
              <a href="#top_inotrope_tab" aria-controls="home1" role="tab" data-toggle="tab" >
               Top Model Features Inotrope
             </a>
            </li>
            <li role="presentation">
              <a href="#all_pos_inotrope_tab" aria-controls="home1" role="tab" data-toggle="tab" >
               All Models Pos Inotrope
             </a>
            </li>
            <li role="presentation">
              <a href="#top_pos_inotrope_tab" aria-controls="home1" role="tab" data-toggle="tab" >
               Top Model Features Pos Inotrope
             </a>
            </li>
            <li role="presentation">
              <a href="#all_neg_inotrope_tab" aria-controls="home1" role="tab" data-toggle="tab" >
               All Models Neg Inotrope
             </a>
            </li>
            <li role="presentation">
              <a href="#top_neg_inotrope_tab" aria-controls="home1" role="tab" data-toggle="tab" >
               Top Model Features Neg Inotrope
             </a>
            </li>
         {%endif%}
    </ul>
    
    <div class="tab-content">
    
     
     <div role="tabpanel" class="tab-pane fade in active" id="all_models_tab" style='overflow-x: auto;'>
     <h4>Table <span class='tbl_num'></span>All Models </h4>
       <table id="tbl-all_models" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr >
           <th>base_model</th>
           <th>f1</th>
           <th>matthews_corrcoef</th>
           <th>num_features</th>
           <th>precision</th>
           <th>recall</th>
           <th>roc_auc</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="top_featur_tab">
     <h4>Table <span class='tbl_num'></span>Top Model Features </h4>
       <table id="tbl-top_featur" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr >
           
           <th>name</th>
           <th>test</th>
          </tr>
        </thead>  
      </table>
     </div> <!-- end of tabpanel div -->
     
     {%if 'l1k' in report.slug %}
     
     <div role="tabpanel" class="tab-pane fade " id="top_featurfilter_tab">
     <h4>Table <span class='tbl_num'></span>Top Model Features Filtered </h4>
       <table id="tbl_featurfilter_tab" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr >
           <th>Probe names</th>
           <th>Importance-score-DNN</th>
           <th>Gene symbol</th>
           
          </tr>
        </thead>  
      </table>
     </div> <!-- end of tabpanel div --> 
     
     <div role="tabpanel" class="tab-pane fade " id="all_inotrope_tab">
     <h4>Table <span class='tbl_num'></span>All Models Inotrope </h4>
       <table id="tbl-all_inotrope" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr >
           <th>base_model</th>
           <th>f1</th>
           <th>matthews_corrcoef</th>
           <th>num_features</th>
           <th>precision</th>
           <th>recall</th>
           <th>roc_auc</th>
           
          </tr>
        </thead>  
      </table>
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="top_inotrope_tab">
     <h4>Table <span class='tbl_num'></span>Top Model Features Inotrope </h4>
       <table id="tbl-top_inotrope" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr >
           
           <th>name</th>
           <th>test</th>
          </tr>
        </thead>  
      </table>
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="all_pos_inotrope_tab">
     <h4>Table <span class='tbl_num'></span>All Models Pos Inotrope </h4>
       <table id="tbl-all_pos_inotrope" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr >
           <th>base_model</th>
           <th>f1</th>
           <th>matthews_corrcoef</th>
           <th>num_features</th>
           <th>precision</th>
           <th>recall</th>
           <th>roc_auc</th>           
          </tr>
        </thead>  
      </table>  
      </table>
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="top_pos_inotrope_tab">
     <h4>Table <span class='tbl_num'></span>Top Model Features Pos Inotrope </h4>
       <table id="tbl-top_pos_inotrope" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr >
           
           <th>name</th>
           <th>test</th>
          </tr>
        </thead>  
      </table>
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="all_neg_inotrope_tab">
     <h4>Table <span class='tbl_num'></span>All Models Pos Inotrope </h4>
       <table id="tbl-all_neg_inotrope" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr >
           <th>base_model</th>
           <th>f1</th>
           <th>matthews_corrcoef</th>
           <th>num_features</th>
           <th>precision</th>
           <th>recall</th>
           <th>roc_auc</th>           
          </tr>
        </thead>  
      </table>  
      </table>
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="top_neg_inotrope_tab">
     <h4>Table <span class='tbl_num'></span>Top Model Features Pos Inotrope </h4>
       <table id="tbl-top_neg_inotrope" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr >
           
           <th>name</th>
           <th>test</th>
          </tr>
        </thead>  
      </table>
     </div> <!-- end of tabpanel div -->
     
     {%endif%}
     
   </div><!-- end of tab-content div -->

</div>

<script>

$(function() {
	
});
</script>

{%endif%}



<!-- ---------------------------------BioMAP inference--------------------------------------------- -->

 <hr/> 
<h2 id='biomap'>BioMAP inference</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_biomap'>Show Data</button>
</center>

<div id='hide_biomap' style='display:none;'> <!-- Div to hide and load on button click -->


<span id="docs-internal-guid-cf75ab3e-f99b-284f-fc64-f1995dd70398">
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">In this section we present the results of the experiment aimed at inferring BioMAP cytotox and diversity biomarkers independently from L1000 dataset using drug perturbation response data. We utilised 280 different model architectures that vary by feature selection method (S_523, f_score, fisher_score, reliefF), number of input features (200, 300, 500, 978) and algorithm (ElasticNet, Linear regression, KNN, RF, ExtraTrees, SVR, MLP). As an input for a models we utilised robust z-scores for landmark genes from L1000 dataset and as an output the ratio ‘Log-Ratio/Confidence interval’ for BioMAP cytotox and diversity data. Robust z-scores were averaged for perturbation replicates. Input L1000 samples and output BioMAP samples were matched by drug and concentration. Also for L1000 dataset we experimented with addition of provided molecular properties (# of aromatic rings, clogp, molecular weight, TPSA) as extra features.</span>
</p><br/>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Since the number of common with L1000 drugs profiled in cytotox and diversity BioMAP sets is different we used two separate output sets: cytotox and diversity. We also used 4 different training set designs to stratify the L1000 set: </span>
</p><br/>
<ol style="margin-top:0pt;margin-bottom:0pt;">
	<li dir="ltr" style="list-style-type: decimal; font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline;"><span style="font-size: 11pt; background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Data set separated by time point (6 and 24 hours) with robust z-scores for landmark genes (Landmarks_by_time).</span></li>
	<li dir="ltr" style="list-style-type: decimal; font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline;"><span style="font-size: 11pt; background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Data set separated by time point (6 and 24 hours) with robust z-scores for landmark genes and values for 4 additional molecular properties (Landmarks_and_molprop_by_time).</span></li>
	<li dir="ltr" style="list-style-type: decimal; font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline;"><span style="font-size: 11pt; background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Aggregated data set with robust z-scores for landmark genes (Landmarks).</span></li>
	<li dir="ltr" style="list-style-type: decimal; font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline;"><span style="font-size: 11pt; background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">Aggregated data set with robust z-scores for landmark genes and values for 4 additional molecular properties (Landmarks_and_molprop).</span></li>
</ol><br/>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">For each of the training sets feature selection methods were applied to L1000 landmark genes followed by building ML and DL models to predict the BioMAP feature. Each BioMAP biomarker was inferred independently. For each ML and DL model optimization of its hyperparameters was applied using grid search. Each model underwent 10-fold cross-validation and the resulting performance was averaged across folds. </span>
</p><br/>
<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;">In this report we present the tables with performance (mean R-squared across all biomarkers in either diversity or cytotox sets) of the models for each training set design. After each performance table we also put in separate tabs the R-squared metrics for each biomarker of top model for cytotox and top model for diversity inference. Due to curse of dimensionality and relatively small number of matching samples DNNs lost to a more conventional methods like ElasticNet and tree-based methods.</span>
</p><br/>
<div><span style="font-size: 11pt; font-family: Arial; color: rgb(0, 0, 0); background-color: transparent; vertical-align: baseline; white-space: pre-wrap;"><br></span>
</div></span>


<!-- BIO Nav tabs  -->     
   <ul class="nav nav-tabs" role="tablist">
     
       <li role="presentation" class="active">
         <a href="#lbt_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_lbt'>
           Landmarks_by_time
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmc1_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmc1'>
           Top model cytotox Landmarks_by_time
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmd1_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmd1'>
           Top model diversity Landmarks_by_time
         </a>
       </li>
       <li role="presentation" >
         <a href="#lambt_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_lambt'>
           Landmarks_and_molprop_by_time
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmc2_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmc2'>
           Top model cytotox Landmarks_and_molprop_by_time
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmd2_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmd2'>
           Top model diversity Landmarks_and_molprop_by_time
         </a>
       </li>
       <li role="presentation" >
         <a href="#land_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_land'>
           Landmarks
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmc3_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmc3'>
           Top model cytotox Landmarks
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmd3_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmd3'>
           Top model diversity Landmarks
         </a>
       </li>
       <li role="presentation" >
         <a href="#lam_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_lam'>
           Landmarks_and_molprop
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmc4_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmc4'>
           Top model cytotox Landmarks_and_molprop
         </a>
       </li>
       <li role="presentation" >
         <a href="#tmd4_tab" aria-controls="home1" role="tab" data-toggle="tab" id='tf_boot_tmd4'>
           Top model diversity Landmarks_and_molprop
         </a>
       </li>
    
   </ul>
         
   <div class="tab-content">
    
     <div role="tabpanel" class="tab-pane fade in active" id="lbt_tab">  
             
       <h4>Table <span class='tbl_num'></span>Landmarks_by_time </h4>
       <table id="tbl-bio-lbt" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>count</th>
           <th>key</th>
           <th>mean_r2</th>
           <th>method</th>
           <th>model</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     <div role="tabpanel" class="tab-pane fade " id="tmc1_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model cytotox Landmarks_by_time</h4>
       <table id="tbl-bio-tmc1" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
       
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     <div role="tabpanel" class="tab-pane fade " id="tmd1_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model diversity Landmarks_by_time</h4>
       <table id="tbl-bio-tmd1" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="lambt_tab">  
             
       <h4>Table <span class='tbl_num'></span>Landmarks_and_molprop_by_time </h4>
       <table id="tbl-bio-lambt" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>count</th>
           <th>key</th>
           <th>mean_r2</th>
           <th>method</th>
           <th>model</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="tmc2_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model cytotox Landmarks_and_molprop_by_time</h4>
       <table id="tbl-bio-tmc2" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
       
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     <div role="tabpanel" class="tab-pane fade " id="tmd2_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model diversity Landmarks_and_molprop_by_time</h4>
       <table id="tbl-bio-tmd2" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="land_tab">  
             
       <h4>Table <span class='tbl_num'></span>Landmarks </h4>
       <table id="tbl-bio-land" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>count</th>
           <th>key</th>
           <th>mean_r2</th>
           <th>method</th>
           <th>model</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="tmc3_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model cytotox Landmarks</h4>
       <table id="tbl-bio-tmc3" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
       
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     <div role="tabpanel" class="tab-pane fade " id="tmd3_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model diversity Landmarks</h4>
       <table id="tbl-bio-tmd3" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="lam_tab">  
             
       <h4>Table <span class='tbl_num'></span>Landmarks_and_molprop </h4>
       <table id="tbl-bio-lam" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>count</th>
           <th>key</th>
           <th>mean_r2</th>
           <th>method</th>
           <th>model</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     
     <div role="tabpanel" class="tab-pane fade " id="tmc4_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model cytotox Landmarks_and_molprop</h4>
       <table id="tbl-bio-tmc4" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
       
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
     <div role="tabpanel" class="tab-pane fade " id="tmd4_tab">  
             
       <h4>Table <span class='tbl_num'></span>Top model diversity Landmarks_and_molprop</h4>
       <table id="tbl-bio-tmd4" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Biomarker_System</th>
           <th>ExtraTreesRegressor</th>
          </tr>
        </thead>  
      </table>
      <br/><br/>
     
     
     </div> <!-- end of tabpanel div -->
    
   
   </div> <!--  end of tab-content div -->

</div>

<script>

$(function() {
	
	
	$("#btn_show_biomap").click(function(){
		
		$(this).hide();
		$("#hide_biomap").slideDown('slow');
	});
	
	
	var Landmarks_by_time = $("#tbl-bio-lbt").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl-bio-lbt'},
	               {extend: 'print', title: 'tbl-bio-lbt'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/timewise_report_land.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var Landmarks_and_molprop_by_time = $("#tbl-bio-lambt").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'Landmarks_and_molprop_by_time'},
	               {extend: 'print', title: 'Landmarks_and_molprop_by_time'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/timewise_report_land_mol.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var Landmarks = $("#tbl-bio-land").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'Landmarks'},
	               {extend: 'print', title: 'Landmarks'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/report_land.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var Landmarks_and_molprop = $("#tbl-bio-lam").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'Landmarks_and_molprop'},
	               {extend: 'print', title: 'Landmarks_and_molprop'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/report_land_mol.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	
	var tbl_bio_tmc1 = $("#tbl-bio-tmc1").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmc1'},
	               {extend: 'print', title: 'tbl_bio_tmc1'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/Landmarks_by_time_cytotoxtop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var tbl_bio_tmd1 = $("#tbl-bio-tmd1").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmd1'},
	               {extend: 'print', title: 'tbl_bio_tmd1'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/Landmarks_by_time_diversitytop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	////////
	var tbl_bio_tmc2 = $("#tbl-bio-tmc2").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmc2'},
	               {extend: 'print', title: 'tbl_bio_tmc2'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/LandmarksMol_by_time_cytotoxtop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var tbl_bio_tmd2 = $("#tbl-bio-tmd2").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmd2'},
	               {extend: 'print', title: 'tbl_bio_tmd2'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/LandmarksMol_by_time_diversitytop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	/////////////
	
	////////
	var tbl_bio_tmc3 = $("#tbl-bio-tmc3").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmc3'},
	               {extend: 'print', title: 'tbl_bio_tmc3'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/Landmarks_cytotoxtop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var tbl_bio_tmd3 = $("#tbl-bio-tmd3").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmd3'},
	               {extend: 'print', title: 'tbl_bio_tmd3'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/Landmarks_diversitytop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	/////////////
	
	////////
	var tbl_bio_tmc4 = $("#tbl-bio-tmc4").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmc4'},
	               {extend: 'print', title: 'tbl_bio_tmc4'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/LandmarksMol_cytotoxtop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
	var tbl_bio_tmd4 = $("#tbl-bio-tmd4").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl_bio_tmd4'},
	               {extend: 'print', title: 'tbl_bio_tmd4'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'gsk_prj4_t1/LandmarksMol_diversitytop_model.csv'
	    	            },        	      
	    
	},"deferRender": true });
	/////////////
	
});


</script>

<!-- ---------- end of BioMAP inference -->

<!-- ---------------------------------Hit qualification--------------------------------------------- -->

 <hr/> 
<h2 id='hit'>Hit qualification</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_hit'>Show Data</button>
</center>

<div id='hide_hit' style='display:none;'> <!-- Div to hide and load on button click -->



Here we utilised our already established methodology to estimate the probability of failure in clinical trials 
(<a href='http://biorxiv.org/content/biorxiv/early/2016/12/29/095653.full.pdf' target='_blank'>Artemov et al., 2016</a>). 
On Figures 1 and 2 you can see the overall statistic for provided L1000 dataset. In "Hit qualification table" you can see the actual score of drug perturbations analysed in this report.
<br/>
<br/>
<center>
<img src='/media/report-portal/gsk_prj4_t1/hit1.png' />
<p>Figure 1. Probability of success for drug perturbations from provided L1000 dataset.
</p>
<hr style='width:50%' />

<img src='/media/report-portal/gsk_prj4_t1/hit2.png' />
<p>Figure 2. Probability of failure for drug perturbations from provided L1000 dataset.
</p>
<hr style='width:50%' />
</center>
<h4>Table <span class='tbl_num'></span>Hit qualification </h4>
       <table id="tbl-hit" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Name</th>
           <th>Failed</th>
           <th>Passed</th>

          </tr>
        </thead>  
      </table>
      <br/><br/>

</div>
<script>

$(function() {
	
	
	$("#btn_show_hit").click(function(){
		
		$(this).hide();
		$("#hide_hit").slideDown('slow');
	});
	
	
	var tbl_hit = $("#tbl-hit").DataTable( {
		"paging":   true,
	    "iDisplayLength": 20,
	    "ordering": true,
	    "order": [],
	    "info":     false,
	    "dom": 'Bfrtip',
	    "scrollX": false,
	    "bAutoWidth": false,
	    "buttons": [
	               {extend: 'csv', title: 'tbl-hit'},
	               {extend: 'print', title: 'tbl-hit'}
	               
	          ],
	    
	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
	    	     'type': 'GET',
	    	     'data':{
	    	    	     reportID: {{report.id}},
	    	    	     file_name: 'clinical_trial/predCT.csv'
	    	            },        	      
	    
	},"deferRender": true });
	
});

</script>
<!-- -----------------end of Hit qualification -->

<!-- ---------------------------------Tox groups analysis--------------------------------------------- -->

 <hr/> 
<h2 id='tox'>Tox groups analysis</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_tox'>Show Data</button>
</center>

<div id='hide_tox' style='display:none;'> <!-- Div to hide and load on button click -->

<center>
<img src='/media/report-portal/gsk_prj4_t5/img_tox.png' />
<p>
Differentially expressed genes distribution across 5 distinct Tox groups
</p>
<hr style='width:50%' />

</center>
<h4>Table <span class='tbl_num'></span>Tox groups analysis </h4>
       <table id="tbl-tox" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>
          <tr>
           <th>Gene</th>
           <th>Mitotox</th>
           <th>MRGX2 AG</th>
           <th>Structural Cardiotox-Positive</th>
           <th>Vasc. Tox Rat Positive</th>
           <th>SAPAD CS deg.nec</th>

          </tr>
        </thead>  
      </table>
      <br/><br/>

</div>
<script>

$(function() {
	
	
	$("#btn_show_tox").click(function(){
		
		$(this).hide();
		$("#hide_tox").slideDown('slow', function(){
			
			var tbl_tox = $("#tbl-tox").DataTable( {
				"paging":   true,
			    "iDisplayLength": 20,
			    "ordering": true,
			    "order": [],
			    "info":     false,
			    "dom": 'Bfrtip',
			    "scrollX": false,
			    "bAutoWidth": false,
			    "buttons": [
			               {extend: 'csv', title: 'tbl-tox'},
			               {extend: 'print', title: 'tbl-tox'}
			               
			          ],
			    
			    "ajax": {'url':'/report-portal/report-ajaxtablejson/',
			    	     'type': 'GET',
			    	     'data':{
			    	    	     reportID: {{report.id}},
			    	    	     file_name: 'gsk_prj4_t5/de_genes_binary.csv'
			    	            },        	      
			    
			},"deferRender": true });
		});
	});
	
	
	
	
});

</script>
<!-- -----------------end of Tox groups analysis -->


<!-- ---------------------------------Similarity--------------------------------------------- -->

 <hr/> 
<h2 id='sim_analysis'>Similarity to LINCS dataset</h2>
<hr/>



<center>
<button type="button"  class="btn btn-success btn-lg center" id='btn_show_sim'>Show Data</button>
</center>

<div id='hide_sim' style='display:none; overflow-x: auto;'> <!-- Div to hide and load on button click -->
<br/>
<p>
In this section we calculated the similarity of investigated perturbations to LINCS dataset. For each sample we performed the
 correlation analysis of its gene expression values and 1.3+ million samples from LINCS on the probe level. 
 Then for each sample we picked top-200 LINCS samples by correlation coefficient and combined these lists 
 in the resulting table. Within each perturbation we averaged correlation coefficients between two replicates
</p>
<br/><br/>
<h4>Table <span class='tbl_num'></span>Similarity scoring of GSK L1000 data in comparison to LINCS dataset </h4>
       <table id="tbl-sim" class="path table table-striped table-bordered" cellspacing="0" width="100%">
        <thead>

        </thead>  
      </table>
      <br/><br/>





</div>

<script>
$(function() {
	
	
	$("#btn_show_sim").click(function(){
		
		$(this).hide();
		$("#hide_sim").slideDown('slow');
	});
	
	
	$.ajax({
        type: 'GET',
        dataType: 'json',
        url: '/report-portal/report-ajaxtablecolumns/',
        data:{
   	         reportID: {{report.id}},
   	         file_name: 'q2norm/correlation_top3.csv'
           },
        success: function(d) {
        	
        	let thead = '<tr>';
        	
        	for (let col of d.columns) {
        		
        		thead+='<th>'+col+'</th>';
        	}
        	
        	thead+='</tr>';
        	
        	$("#tbl-sim").find("thead").html(thead);
        	
        	var tbl_sim = $("#tbl-sim").DataTable( {
        		"paging":   true,
        	    "iDisplayLength": 20,
        	    "ordering": true,
        	    "order": [],
        	    "info":     false,
        	    "dom": 'Bfrtip',
        	    "scrollX": false,
        	    "bAutoWidth": false,
        	    "buttons": [
        	               {extend: 'csv', title: 'tbl-sim'},
        	               {extend: 'print', title: 'tbl-sim'}
        	               
        	          ],
        	    
        	    "ajax": {'url':'/report-portal/report-deepgsktablejson/',
        	    	     'type': 'GET',
        	    	     'data':{
        	    	    	     reportID: {{report.id}},
        	    	    	     file_name: 'q2norm/correlation_top3.csv'
        	    	            },        	      
        	    
        	},"deferRender": true });
        }
    });
	/*
	
	*/
});


</script> 

<!-- -----------------end of Similarity -->

<hr/>
  
  {%if report.conclusion_text%}
          <h2 id="conclusions" >Discussion</h2>
          <hr/>
          
          {{report.conclusion_text|safe}}
          
          <hr/>
  {%endif%}
          
  {%if report.references_text%}
          <h2 id="references" >References</h2>
          <hr/>
          
          {{report.references_text|safe}}
          
          <hr/>
 {%endif%}

</div>  


<div style='min-height:500px'></div>
            
          </div>
          
          
          
        </div>
      </div>
      
    </div>
    <div class="bar" style="heigh:100%">
      <div class="inner h100" style="heigh:100%">
        <div class="top"><img src="/static/images/logo.svg"></div>
        <div class="menu">
          <ul class="list-unstyled" style='margin-top: 0;'>
            <li class="title">Table of contents: </li>
            <!-- 
            {%if report.input_data_text%}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#input_data">Input data</a></li>
            {%endif%}
            {%if report.methods_text%}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#methods">Methods</a></li>
            {%endif%}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#gene_analysis">Gene level analysis</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#path_analysis">Pathway analysis</a></li>
            {%if report.conclusion_text%}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#conclusions">Conclusions</a></li>
            {%endif%}
            {%if report.references_text%}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#references">References</a></li>
            {%endif%}
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#input_data">Input data</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#dl_analysis">Deep Learning</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#kinetic_analysis">Kinetic pathway modeling</a></li>
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#sim_analysis">Similarity</a></li>
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#cor_analysis">Public domain screening</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#pt_analysis">Scoring of potential targets</a></li>
            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#conclusions">Discussion</a></li>
             -->
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#gene_analysis">Gene level analysis</a></li>
            <!-- <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tf_analysis">Transcription Factor Analysis</a></li>
            -->
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#path_analysis">Pathway Analysis</a></li>
            {%if 'above-and-beyond' in report.slug %}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#ds_analysis">Drug Scoring</a></li>
            {%endif%}
            {%if 'l1k' in report.slug %}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#dl_analysis">Deep Learning</a></li>
            {%endif%}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#biomap">BioMAP inference</a></li>            
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#hit">Hit qualification</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tox">Tox groups analysis</a></li>
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#sim_analysis">Similarity to LINCS dataset </a></li>
            {%if 'gsk' in report.slug %}
            <li>&nbsp;&nbsp;&nbsp;&nbsp;<a href="#conclusions">Discussion</a></li>
            {%endif%}
          </ul>
          <br/><br/>
        </div>
        <div class="bottom">
          <div class="inner"> Insilico Medicine Inc., Johns Hopkins <br/>University Eastern, ETC B301, 1101 East <br/>33rd Street, Baltimore, MD 21218 </div>
        </div>
      </div>
    </div>
    
      
    <script src="https://cdn.datatables.net/1.10.10/js/jquery.dataTables.min.js" type="text/javascript"></script>
    <script src="https://cdn.datatables.net/1.10.10/js/dataTables.bootstrap.min.js" type="text/javascript"></script>
    <script src="https://cdn.datatables.net/buttons/1.1.0/js/dataTables.buttons.min.js" type="text/javascript"></script>
    <script src="https://cdn.datatables.net/buttons/1.1.0/js/buttons.bootstrap.min.js" type="text/javascript"></script>
    <script src="//cdn.datatables.net/buttons/1.1.0/js/buttons.flash.min.js" type="text/javascript"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/jszip/2.5.0/jszip.min.js" type="text/javascript"></script>
    <script src="//cdn.rawgit.com/bpampuch/pdfmake/0.1.18/build/pdfmake.min.js" type="text/javascript"></script>
    <script src="//cdn.rawgit.com/bpampuch/pdfmake/0.1.18/build/vfs_fonts.js" type="text/javascript"></script>
    <script src="//cdn.datatables.net/buttons/1.1.0/js/buttons.html5.min.js" type="text/javascript"></script>
    <script src="//cdn.datatables.net/buttons/1.1.0/js/buttons.print.min.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/venn.js/venn.js" type="text/javascript"></script>
    
    
    
    <script src="/static/report/portal/js/opentip.js" charset="utf-8"></script>
    <script src="/static/report/bt/js/adapter-native.js" charset="utf-8"></script>
    <script src="/static/report/portal/js/volcano.js" type="text/javascript"></script>biomap
    <script src="/static/report/portal/js/scatter.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/tables.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/boxplot.js" type="text/javascript"></script>    
    <script src="/static/report/portal/js/venn-diagram.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/tf.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/deeplearning.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/correlation.js" type="text/javascript"></script>
    <script src="/static/report/portal/js/ds.js" type="text/javascript"></script>
    
    
    
    <!-- MODAL FOR PATHS  -->
    <div id="pathmodal" class="modal fade large" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true" >
 <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;  </button>
        <h4 class="modal-title" id="myModalLabel">Modal title</h4>
      </div>
      <div class="modal-body" id='modalBody'>
        
      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
      </div>
    </div>
  </div>
</div>
    
<script>

$(function() {
	
	
	$("#btn_show_genes").click(function(){
		
		$(this).hide();
		$("#hide_genes").slideDown('slow');
		 
		{%for group in report.genegroup_set.all%}
		  {%if forloop.counter = 1%}
		
		     drawVolcanoPlot('{{group.get_slug}}_volcano', {{report.id}}, '{{group.doc_logfc}}');
	         drawGeneTable({{report.id}}, 'tbl-{{group.get_slug}}', '{{group.doc_logfc}}', categories='{{group.get_slug}}'); 
	     
	       {%endif%}
	     {%endfor%}
	     
	     drawGeneTable({{report.id}}, 'tbl-gene_all', 'all', categories='{{report.get_all_gene_groups_norm_as_list}}');
	 	 
	     
	   //GENES
	 	{%for perm in report.get_gene_permutations%}
	 	
	 	$("#venn_boot_gene_{{perm.strip}}").click(function(){
	 		drawVenn({{report.id}},
	 				'venn_gene-{{perm.strip}}-up',
	 				'{{perm.strip}}',	 
	 				false, 'up', 'venn_gene_dinamic_{{perm.strip}}', 'genes', categories='{{report.get_all_gene_groups_norm_as_list}}');
	 		drawVenn({{report.id}},
	 				'venn_gene-{{perm.strip}}-down',
	 				'{{perm.strip}}',			
	 				false, 'down', 'venn_gene_dinamic_{{perm.strip}}', 'genes', categories='{{report.get_all_gene_groups_norm_as_list}}');
	 		
	 	});
	 	
	 	{%endfor%}
	 	
	 	
	 	{%for gene in report.get_notable_genes_list%}
		  drawGeneBoxplot(reportID={{report.id}}, name='{{gene}}', renderTo='{{gene}}', gene='{{gene}}', categories='{{report.get_all_gene_groups_norm_as_list}}');
		{%endfor%}
		
	});
	
	{%if report.gene_plot == 'vulcano'%}
	  {%for group in report.genegroup_set.all%}
	  
	  {%if forloop.counter = 1%}
	     
	      
	  {%else%}
	  $('#vol_boot_{{group.get_slug}}').click(function(){
			
			if($("#{{group.get_slug}}_volcano span").html() == 'Loading plot...'){
				drawVolcanoPlot('{{group.get_slug}}_volcano', {{report.id}}, '{{group.doc_logfc}}');
			}
			if ( !$.fn.dataTable.isDataTable( '#tbl-{{group.get_slug}}' ) ) {
				 drawGeneTable({{report.id}}, 'tbl-{{group.get_slug}}', '{{group.doc_logfc}}', categories='{{group.get_slug}}'); 
				}
		   });
	  
	  {%endif%}
	  
	  
	  {%endfor%}
	{%endif%}
	{%if  report.gene_plot == 'scatter'%}
	{%for group in report.genegroup_set.all%}
	  drawGeneScatter(name='{{group.name}} VS {{report.norm_name}}',
			          renderTo='{{group.get_slug}}_volcano',
			          xname='log2({{group.name}} Expression level)',
			          yname='log2({{report.norm_name}} Expression level)',
			          file_name='{{group.document}}',
			          reportID={{report.id}});
	  drawGeneTableScatter({{report.id}}, 'tbl-{{group.get_slug}}', '{{group.doc_logfc}}', categories='{{report.get_all_gene_groups_norm_as_list}}'); 
	{%endfor%}
	{%endif%}
	
	
	
	
	
	
	
///////////// SIGNALING PATHs

$("#btn_show_path").click(function(){
	$(this).hide();
	$("#hide_path").slideDown('slow');
	
	drawPathwayTable({{report.id}},
			'tbl-path_all', 
	         'all', 
	         'all', false, '{{report.organism}}');
	
});
	

	
	{%for perm in report.get_path_permutations%}
	
	$("#venn_boot_path_{{perm.strip}}").click(function(){
	
	drawVenn({{report.id}},
		'venn_path-{{perm.strip}}-up',
		'{{perm.strip}}',
		false, 'up', 'venn_path_dinamic_{{perm.strip}}', 'pathways', '{{report.organism}}');
	drawVenn({{report.id}},
			'venn_path-{{perm.strip}}-down',
			'{{perm.strip}}',
			false, 'down', 'venn_path_dinamic_{{perm.strip}}', 'pathways', '{{report.organism}}');
	});
    {%endfor%}
    
/////////////////// Transcription Factor
$("#btn_show_tf").click(function(){
	$(this).hide();
	$("#hide_tf").slideDown('slow');
	
	{%for group in report.tfgroup_set.all%}
    
      {%if forloop.counter = 1%}   
        drawTFTable({{report.id}}, 'tbl-tf-{{group.get_slug}}', '{{group.document}}', categories='{{group.name}}'); 
        drawTFTrrustTable({{report.id}}, 'tbl-tftrrust-{{group.get_slug}}', '{{group.trrust}}', categories='{{group.name}}'); 
      {%endif%}
    {%endfor%}
	
});

    {%for group in report.tfgroup_set.all%}
    
        {%if forloop.counter = 1%}        
          

        {%else%}
        
        $('#tf_boot_{{group.get_slug}}').click(function(){
        	
        	if ( !$.fn.dataTable.isDataTable( '#tbl-tf-{{group.get_slug}}' ) ) {
        		drawTFTable({{report.id}}, 'tbl-tf-{{group.get_slug}}', '{{group.document}}', categories='{{group.name}}'); 
				}
        	
        	
        });
        
        $('#tftrrust_boot_{{group.get_slug}}').click(function(){
        	if ( !$.fn.dataTable.isDataTable( '#tbl-tftrrust-{{group.get_slug}}' ) ) {
        		drawTFTrrustTable({{report.id}}, 'tbl-tftrrust-{{group.get_slug}}', '{{group.trrust}}', categories='{{group.name}}'); 
				}
        	
        });
        
        
        {%endif%}
    
    
    {%endfor%}

/////////////////// Drug Scoring
    $("#btn_show_ds").click(function(){
    	$(this).hide();
    	$("#hide_ds").slideDown('slow');
    	
    	{%for group in report.drugscoregroup_set.all%}
        
          {%if forloop.counter = 1%}   
            drawDSTable({{report.id}}, 'tbl-ds-{{group.get_slug}}', '{{group.document}}', categories='{{group.name}}'); 
            drawDSBarplot({{report.id}}, '{{group.get_slug}}_dsbarmolecule_plot', 
            		'{{group.document}}', 
            		categories='{{group.name}}',
            		pert_type='molecule');
            drawDSBarplot({{report.id}}, '{{group.get_slug}}_dsbargene_plot', 
            		'{{group.document}}', 
            		categories='{{group.name}}',
            		pert_type='gene');
          {%endif%}
        {%endfor%}
    	
    });

        {%for group in report.drugscoregroup_set.all%}
        
            {%if forloop.counter = 1%}        
              

            {%else%}
            
            $('#ds_boot_{{group.get_slug}}').click(function(){
            	
            	if ( !$.fn.dataTable.isDataTable( '#tbl-ds-{{group.get_slug}}' ) ) {
            		drawDSTable({{report.id}}, 'tbl-ds-{{group.get_slug}}', '{{group.document}}', categories='{{group.name}}'); 
    				}
            	
            	
            });
            
            $('#dsbarmolecule_boot_{{group.get_slug}}').click(function(){
    			
    			if($("#{{group.get_slug}}_dsbarmolecule_plot span").html() == 'Loading Bar Plot...'){
    				drawDSBarplot({{report.id}}, '{{group.get_slug}}_dsbarmolecule_plot', 
    	            		'{{group.document}}', 
    	            		categories='{{group.name}}',
    	            		pert_type='molecule');
    			}
    			});
            
           $('#dsbargene_boot_{{group.get_slug}}').click(function(){
    			
    			if($("#{{group.get_slug}}_dsbargene_plot span").html() == 'Loading Bar Plot...'){
    				drawDSBarplot({{report.id}}, '{{group.get_slug}}_dsbargene_plot', 
    	            		'{{group.document}}', 
    	            		categories='{{group.name}}',
    	            		pert_type='gene');
    			}
    			});
            
            //$('#tftrrust_boot_{{group.get_slug}}').click(function(){
            //	if ( !$.fn.dataTable.isDataTable( '#tbl-tftrrust-{{group.get_slug}}' ) ) {
            //		drawTFTrrustTable({{report.id}}, 'tbl-tftrrust-{{group.get_slug}}', '{{group.trrust}}', categories='{{group.name}}'); 
    		//		}
            //	
            //});
            
            
            {%endif%}
        
        
        {%endfor%}





    

    
 /* METABOLIC PATHs
  drawPathwayTable({{report.id}},
		  'tbl-meta_all', 
	         'all', 
	         'all', true, '{{report.organism}}');
 
	{%for perm in report.get_path_permutations%}
	drawVenn({{report.id}},
		'venn_meta-{{perm.strip}}-up',
		'{{perm.strip}}',
		true, 'up', 'venn_meta_dinamic_{{perm.strip}}', 'pathways', '{{report.organism}}');
	drawVenn({{report.id}},
			'venn_meta-{{perm.strip}}-down',
			'{{perm.strip}}',
			true, 'down', 'venn_meta_dinamic_{{perm.strip}}', 'pathways', '{{report.organism}}');
    {%endfor%}
*/
});

</script>
  </body>
</html>